
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.e22e79d5a578.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.f012d399c229.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.4455260d4345.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Identification of genetic variants or genes that are associated with Homoharringtonine (HHT) response through a genome-wide association study in human lymphoblastoid cell lines (LCLs) - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939B9567B155015500002F74ECCDEC78.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.6.0.post2+24fead6" />

<meta name="ncbi_domain" content="frontgene" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC4292778/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4292778/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Frontiers in Genetics" /><meta name="citation_title" content="Identification of genetic variants or genes that are associated with Homoharringtonine (HHT) response through a genome-wide association study in human lymphoblastoid cell lines (LCLs)" /><meta name="citation_author" content="Yin Tong" /><meta name="citation_author_institution" content="1Department of Hematology, Shanghai General Hospital, Shanghai Jiao Tong University Affiliated First People's Hospital, Shanghai, China" /><meta name="citation_author" content="Nifang Niu" /><meta name="citation_author_institution" content="2Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN, USA" /><meta name="citation_author" content="Gregory Jenkins" /><meta name="citation_author_institution" content="3Division of Statistic and Bioinformatics, Department of Health Science Research, Mayo Clinic, Rochester, MN, USA" /><meta name="citation_author" content="Anthony Batzler" /><meta name="citation_author_institution" content="3Division of Statistic and Bioinformatics, Department of Health Science Research, Mayo Clinic, Rochester, MN, USA" /><meta name="citation_author" content="Liang Li" /><meta name="citation_author_institution" content="4Department of Oncology, Institute of Medicinal Biotechnology, Peking Union Medical College and Chinese Academy of Medical Sciences, Beijing, China" /><meta name="citation_author" content="Krishna R. Kalari" /><meta name="citation_author_institution" content="3Division of Statistic and Bioinformatics, Department of Health Science Research, Mayo Clinic, Rochester, MN, USA" /><meta name="citation_author" content="Liewei Wang" /><meta name="citation_author_institution" content="2Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN, USA" /><meta name="citation_publication_date" content="2014" /><meta name="citation_volume" content="5" /><meta name="citation_doi" content="10.3389/fgene.2014.00465" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4292778/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC4292778/" /><meta name="citation_pmid" content="25628645" /><meta name="DC.Title" content="Identification of genetic variants or genes that are associated with Homoharringtonine (HHT) response through a genome-wide association study in human lymphoblastoid cell lines (LCLs)" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Frontiers Media SA" /><meta name="DC.Contributor" content="Yin Tong" /><meta name="DC.Contributor" content="Nifang Niu" /><meta name="DC.Contributor" content="Gregory Jenkins" /><meta name="DC.Contributor" content="Anthony Batzler" /><meta name="DC.Contributor" content="Liang Li" /><meta name="DC.Contributor" content="Krishna R. Kalari" /><meta name="DC.Contributor" content="Liewei Wang" /><meta name="DC.Date" content="2014" /><meta name="DC.Identifier" content="10.3389/fgene.2014.00465" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Identification of genetic variants or genes that are associated with Homoharringtonine (HHT) response through a genome-wide association study in human lymphoblastoid cell lines (LCLs)" /><meta property="og:type" content="article" /><meta property="og:description" content="Homoharringtonine (HHT) has been widely used in China to treat patients with acute and chronic myeloid leukemia for decades. Since response to HHT varies among patients, our study aimed to identify biomarkers that might influence the response to HHT using ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4292778/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it�s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you�re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Genet%22[journal]" class="navlink">Front Genet</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC4292778
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC4292778/pdf/fgene-05-00465.pdf" class="int-view">PDF (4.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/4292778/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/4292778/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="4292778" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4292778%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="gatFRLvbEzFgB3Lt1Q1NEJIqEy9vMzy8GpJ8HTgaViD0gwdlUp7AUlyTtlGScut9">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4292778%2F&amp;text=Identification%20of%20genetic%20variants%20or%20genes%20that%20are%20associated%20with%20Homoharringtonine%20(HHT)%20response%20through%20a%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4292778%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4292778/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC4292778/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/25628645/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC4292778/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/25628645/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC4292778/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/4292778/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Genet%22[journal]" class="navlink">Front Genet</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC4292778
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-frontgene.gif" alt="Logo of frontgene" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.frontiersin.org/genetics" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Front Genet.</span> 2014; 5: 465. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2015 Jan 13. </span>  <span class="doi"><span>doi:�</span><a href="//doi.org/10.3389%2Ffgene.2014.00465" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3389/fgene.2014.00465</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4292778</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/25628645">25628645</a></div></div></div><h1 class="content-title">Identification of genetic variants or genes that are associated with Homoharringtonine (HHT) response through a genome-wide association study in human lymphoblastoid cell lines (LCLs)</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tong%20Y%5BAuthor%5D" class="affpopup" co-rid="_co_idm140326072374336" co-class="co-affbox">Yin Tong</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Niu%20N%5BAuthor%5D" class="affpopup" co-rid="_co_idm140326072372592" co-class="co-affbox">Nifang Niu</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jenkins%20G%5BAuthor%5D" class="affpopup" co-rid="_co_idm140326059525504" co-class="co-affbox">Gregory Jenkins</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Batzler%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140326069538208" co-class="co-affbox">Anthony Batzler</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Li%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140326069536208" co-class="co-affbox">Liang Li</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kalari%20KR%5BAuthor%5D" class="affpopup" co-rid="_co_idm140326066847776" co-class="co-affbox">Krishna R. Kalari</a>,<sup>3</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20L%5BAuthor%5D" class="affpopup" co-rid="_co_idm140326107345552" co-class="co-affbox">Liewei Wang</a><sup>2,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140326072374336"><h3 class="no_margin">Yin Tong</h3><p><sup>1</sup>Department of Hematology, Shanghai General Hospital, Shanghai Jiao Tong University Affiliated First People's Hospital, Shanghai, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tong%20Y%5BAuthor%5D">Yin Tong</a></div></div><div id="_co_idm140326072372592"><h3 class="no_margin">Nifang Niu</h3><p><sup>2</sup>Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Niu%20N%5BAuthor%5D">Nifang Niu</a></div></div><div id="_co_idm140326059525504"><h3 class="no_margin">Gregory Jenkins</h3><p><sup>3</sup>Division of Statistic and Bioinformatics, Department of Health Science Research, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jenkins%20G%5BAuthor%5D">Gregory Jenkins</a></div></div><div id="_co_idm140326069538208"><h3 class="no_margin">Anthony Batzler</h3><p><sup>3</sup>Division of Statistic and Bioinformatics, Department of Health Science Research, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Batzler%20A%5BAuthor%5D">Anthony Batzler</a></div></div><div id="_co_idm140326069536208"><h3 class="no_margin">Liang Li</h3><p><sup>4</sup>Department of Oncology, Institute of Medicinal Biotechnology, Peking Union Medical College and Chinese Academy of Medical Sciences, Beijing, China</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Li%20L%5BAuthor%5D">Liang Li</a></div></div><div id="_co_idm140326066847776"><h3 class="no_margin">Krishna R. Kalari</h3><p><sup>3</sup>Division of Statistic and Bioinformatics, Department of Health Science Research, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kalari%20KR%5BAuthor%5D">Krishna R. Kalari</a></div></div><div id="_co_idm140326107345552"><h3 class="no_margin">Liewei Wang</h3><p><sup>2</sup>Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20L%5BAuthor%5D">Liewei Wang</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140326106508736_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140326106508736_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140326106508736_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140326106508736_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>1</sup>Department of Hematology, Shanghai General Hospital, Shanghai Jiao Tong University Affiliated First People's Hospital, Shanghai, China</div><div class="fm-affl" id="aff2"><sup>2</sup>Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, Rochester, MN, USA</div><div class="fm-affl" id="aff3"><sup>3</sup>Division of Statistic and Bioinformatics, Department of Health Science Research, Mayo Clinic, Rochester, MN, USA</div><div class="fm-affl" id="aff4"><sup>4</sup>Department of Oncology, Institute of Medicinal Biotechnology, Peking Union Medical College and Chinese Academy of Medical Sciences, Beijing, China</div><div id="fn-a.q.b.i.a">Edited by: Jos&#x000e9; A. G. Ag&#x000fa;ndez, University of Extremadura, Spain</div><div id="fn-a.q.b.i.b">Reviewed by: Alessio Squassina, University of Cagliari, Italy; Pedro Ayuso Parejo, Hospital Infanta Leonor, Spain</div><div id="fn001">*Correspondence: Liewei Wang, Department of Molecular Pharmacology and Experimental Therapeutics, Mayo Clinic, 200 First Street SW, Rochester, MN 55905, USA e-mail: <a href="mailto:dev@null" data-email="ude.oyam@ieweiL.gnaW" class="oemail">ude.oyam@ieweiL.gnaW</a></div><div id="fn002">This article was submitted to Pharmacogenetics and Pharmacogenomics, a section of the journal Frontiers in Genetics.</div></div><div class="fm-article-notes hide half_rhythm" id="idm140326106508736_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Nov 21; Accepted 2014 Dec 18.</div></div><div class="permissions half_rhythm hide" id="idm140326106508736_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2015 Tong, Niu, Jenkins, Batzler, Li, Kalari and Wang.</div><div class="license half_rhythm">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC4292778/bin/Table1.PDF" data-ga-action="click_feat_suppl">Table1.PDF</a><span style="color:gray"> (28K)</span></div><div class="small guid">GUID:�A8C6CF28-481D-472D-9561-1262449F691E</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC4292778/bin/Table2.PDF" data-ga-action="click_feat_suppl">Table2.PDF</a><span style="color:gray"> (76K)</span></div><div class="small guid">GUID:�C722D89A-4168-42D4-BE39-938EDF9C14F7</div></div></dd></dl></div><div id="abstract-a.q.b.p" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.q.b.ptitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Homoharringtonine (HHT) has been widely used in China to treat patients with acute and chronic myeloid leukemia for decades. Since response to HHT varies among patients, our study aimed to identify biomarkers that might influence the response to HHT using a panel of various human lymphoblastoid cell lines (LCLs). Genome-wide association (GWA) analysis using single nucleotide polymorphism (SNP) and mRNA expression data was assessed for association with cytotoxicity to HHT in LCLs. Integrated analysis among SNPs, expression, AUC value was also performed to help select candidate genes for further functional characterization. Functional validation of candidate genes was performed using leukemia cell lines (U937, K562). Candidate genes were knocked down using specific siRNA and its response to HHT was assessed using MTS assay. We found that 15 expression probes were associated with HHT AUC with <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup>, and 96 individual probe sets with <em>P</em> &#x0003c; 10<sup>&#x02212;3</sup>. Eighteen SNPs were associated with HHT AUC with <em>P</em> &#x0003c; 10<sup>&#x02212;5</sup> and 281 SNPs with <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup>. The integrated analysis identified 4 unique SNPs that were associated with both expression and AUC. Functional validation using siRNA knockdown in leukemia cell lines showed that knocking down <em>CCDC88A, CTBP2, SOCS4</em> genes in U937 and K562 cells significantly altered HHT cytotoxicity. In summary, this study performed with LCLs can help to identify novel biomarker that might contribute to variation in response to HHT therapy.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genome-wide association study, biomarkers, Homoharringtonine (HHT), lymphoblastoid cell line system, leukemia</span></div></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title">Introduction</h2><p class="p p-first">Homoharringtonine (HHT) is a natural cephalotaxus alkaloid that is derived from the <em>Cephalotaxus</em> species found in China. It has been widely used in China for the treatment of acute myeloid leukemia (AML) and chronic myeloid leukemia (CML) for more than 3 decades. In the United States, the semi-synthesized HHT, known as Omacetaxine, has been approved by FDA in the setting of CML after failure of 2 or more tyrosine kinase inhibitors (TKIs) (Kantarjian et al., <a href="#B9" rid="B9" class=" bibr popnode">2013</a>). HHT exhibits its anti-leukemic effects in myeloid leukemia primarily through inhibition of protein synthesis, cell proliferation and by enhancing apoptosis of leukemic cells (Zhou et al., <a href="#B29" rid="B29" class=" bibr popnode">1995</a>; Yinjun et al., <a href="#B28" rid="B28" class=" bibr popnode">2004</a>; Tang et al., <a href="#B23" rid="B23" class=" bibr popnode">2006</a>).</p><p>Previous studies have shown that HHT has significant synergistic effects with cytarabine (Zhou et al., <a href="#B29" rid="B29" class=" bibr popnode">1995</a>). Based on these observations, investigators in China attempted to add HHT into the conventional induction therapy (anthracycline plus cytarabine regimen) in newly diagnosed patients with AML to improve therapeutic outcomes. A multicenter randomized controlled phase 3 trial was carried out to assess the efficacy and safety of HHT-based induction treatment. Results showed that HHT in combination with cytarabine and aclarubicin (HAA regimen) or daunorubicin (HAD regimen) improved both overall complete response (CR) rate and estimated 3-year overall survival in de novo AML patients under the age of 57 years. However, the addition of HHT also increased early death rate due to more severe marrow suppression (Jin et al., <a href="#B8" rid="B8" class=" bibr popnode">2013</a>). These findings raised a question of when HHT should be used in AML chemotherapy. If we can find biomarkers that can predict the response to HHT, then we can decide whether we should add HHT into conventional regimen. Selection of patients predicted to respond to HHT would maximize drug efficacy while minimizing drug-related toxicity.</p><p>Several clinical factors may influence drug response; however, the genetic polymorphism in germline genome can also play an important role in variation in the response to cancer therapy (Wang et al., <a href="#B25" rid="B25" class=" bibr popnode">2011</a>). In recent years, the application of high-throughput genomic techniques makes it possible to identify variation across genome that may help to find biomarkers responsible for drug sensitivity (Sachidanandam et al., <a href="#B20" rid="B20" class=" bibr popnode">2001</a>; Morley et al., <a href="#B13" rid="B13" class=" bibr popnode">2004</a>; Sabeti et al., <a href="#B19" rid="B19" class=" bibr popnode">2007</a>). In addition, expression quantitative trait locus (eQTL) studies performed using gene expression profiles have identified single nucleotide polymorphisms (SNPs) associated with drug response through their influence on gene expression. The integration of genotype, gene-expression data as well as drug-induced cytotoxicity or other pharmacologic phenotypes has been successfully applied in various previous studies using the same approach described in this study (Huang et al., <a href="#B6" rid="B6" class=" bibr popnode">2007</a>; Li et al., <a href="#B11" rid="B11" class=" bibr popnode">2009</a>).</p><p>This study was designed to study the role of genetic variation in variation in HHT response by using these well-established pharmacogenomic approaches that involve genome-wide basal gene expression profiles and genome-wide SNPs for 278 human lymphoblastoid cell lines (LCLs) to identify SNPs/genes that might contribute to variation in HHT drug response. This LCL model system has been successfully used to identify biomarkers for multiple chemotherapeutic agents in our laboratory (Li et al., <a href="#B10" rid="B10" class=" bibr popnode">2008</a>; Niu et al., <a href="#B14" rid="B14" class=" bibr popnode">2010</a>; Jiang et al., <a href="#B7" rid="B7" class=" bibr popnode">2013</a>). As described previously, we have obtained genomic information for these LCLs involving approximately 1.3 million SNPs per cell line, 54,613 mRNA expression probe sets. We also have performed genome-wide imputation using 1000 Genomes Project data in these LCLs. We next performed a HHT cytotoxicity assay with the same LCLs to obtain HHT area under the curve (AUC) as an <em>in vitro</em> HHT response phenotype. After genome-wide association (GWA) analysis involving single nucleotide polymorphisms (SNPs), mRNA data and also integrated analysis of SNPs, gene expression and AUC. The analysis resulted in 7 candidate genes that we further performed siRNA knockdown to determine their effect on HHT response or cytotoxicity. Three genes showed significant impact on HHT cytotoxicity after knocking down with specific siRNAs.</p><p class="p p-last">In summary, we have used genome-wide SNPs and expression data from a cell line-based model system to identified genes that were associated with HHT sensitivity. Candidate genes were then functionally validated by MTS assay. Our goals were to identify possible biomarkers that might contribute to HHT cytotoxicity.</p></div><div id="s2" class="tsec sec"><h2 class="head no_bottom_margin" id="s2title">Materials and methods</h2><div id="sec-a.r.b.b" class="sec sec-first"><h3 id="sec-a.r.b.btitle">Cell lines</h3><p class="p p-first-last">EBV-transformed LCLs from 93 African-American (AA), 91 Caucasian-American (CA), and 94 Han Chinese-American (HCA) unrelated healthy subjects (sample sets HD 100AA, HD 100 CAU, HD 100 CHI) were purchased from the Coriell Cell Repository (Camden, NJ, USA) as reported previously (Li et al., <a href="#B10" rid="B10" class=" bibr popnode">2008</a>, <a href="#B11" rid="B11" class=" bibr popnode">2009</a>; Niu et al., <a href="#B14" rid="B14" class=" bibr popnode">2010</a>; Jiang et al., <a href="#B7" rid="B7" class=" bibr popnode">2013</a>). All the samples had been anonymized by the National Institute of General Medical Sciences (NIGMS) before deposit, and all the subjects had provided written consent for their experimental use. Human leukemic monocyte cell line U937 and human erythromyeloblastoid leukemia cell line K562 were obtained from ATCC (U937, ATCC&#x000ae; CRL-1593.2&#x02122;; K562, ATCC&#x000ae; CCL-243&#x02122;). LCLs were cultured in RPMI 1640 medium (Mediatech, VA, USA) supplemented with 15% heat-inactivated Fetal Bovine Serum (FBS) (Atlanta Biologicals, GA, USA). K562 cells and U937 cells were grown in RPMI-1640 medium with 10% FBS.</p></div><div id="sec-a.r.b.c" class="sec"><h3 id="sec-a.r.b.ctitle">Drugs and cell proliferation assay</h3><p class="p p-first-last">HHT was purchased from Sigma-Aldrich (St. Louis, MO, USA). HHT cytotoxicity studies were performed to determine the range of variation in HHT AUC among the 278 LCL cell lines from three ethnic groups. Drugs were dissolved in DMSO and were frozen at &#x02212;20&#x000b0;C. Cell proliferation assay were performed in triplicate as described previously (Li et al., <a href="#B10" rid="B10" class=" bibr popnode">2008</a>). Briefly, 100 &#x003bc;l of LCL cells or U937 cells (5 &#x000d7; 10<sup>6</sup> cells/ml) were plated into 96-well plates (Corning) and were treated with HHT concentration at 0, 2.5, 5, 10, 20, 40, 80, 160, 320 nmol/L. After incubation for 72 h, 20 &#x003bc;l of CellTiter 96&#x000ae; Aqueous MTS Cell Proliferation Assay solution (Promega Corporation, Madison, WI, USA) was added to each well. Plates were read in a Safire 2 plate reader (Tecan, AG, USA). For K562 cells, 100 &#x003bc;l of K562 cells (4 &#x000d7; 10<sup>6</sup> cells/ml) were plated in 96-well plate and HHT concentration was adjusted to 0, 5, 10, 20, 40, 80, 160, 320, 640 nmol/L. The slight difference in HHT concentration chosen between K562 and U937 was to derive the best cytotoxicity curve for each cell line.</p></div><div id="sec-a.r.b.d" class="sec"><h3 id="sec-a.r.b.dtitle">Genome-wide SNP and expression data analysis</h3><p class="p p-first">The genotyping and expression array data were obtained for all 278 LCLs and were quality controlled as previously described (Li et al., <a href="#B10" rid="B10" class=" bibr popnode">2008</a>; Niu et al., <a href="#B14" rid="B14" class=" bibr popnode">2010</a>; Jiang et al., <a href="#B7" rid="B7" class=" bibr popnode">2013</a>). These data are publicly available from the NCBI Gene Expression Omnibus (<a href="http://www.ncbi.nlm.nih.gov/geo" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/geo</a>) under Super Series accession numbers <a id="__tag_421440253" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE24277" ref="/geo/query/acc.cgi?acc=GSE24277"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE24277","term_id":"24277"}}</span>GSE24277</a> and <a id="__tag_421440252" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE23120" ref="/geo/query/acc.cgi?acc=GSE23120"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE23120","term_id":"23120"}}</span>GSE23120</a>. Briefly, DNA from all the LCLs was genotyped using Illumina HumanHap 550K and 510S Beadchips, which assayed 561,303 and 487,407 SNPs, respectively. We also obtained publicly available Affymetrix SNP array 6.0 Chip SNP data involving 906,367 SNPs for the same cells to give us even better coverage. Quality control (QC) was performed for all of these SNPs prior to performing statistical analysis. Specifically, for data obtained with the Illumina 550K array, we removed 11,258 SNPs that had call rate &#x0003c;95%, 33,237 SNPs with minor allele frequencies (MAF) &#x0003c;5%, and 49 SNPs that deviated from Hardy-Weinberg equilibrium (HWE). Therefore, 516,759 Illumina 550K SNPs were used in the genome-wide SNP analysis. The same approach was used for the QC analysis of SNPs on the Illumina 510S platform which resulted in a total of 336,070 SNPs remained for analysis. QC analysis was also performed for the publicly available Affymetrix 6.0 SNPs. After removing 32,515 SNPs with call rate &#x0003c;95%, 133,553 SNPs with MAF &#x0003c;5%, and 324 SNPs that deviated from HWE, a total of 739,395 SNPs remained. After removing redundant SNPs genotyped on both Affy and Illumina platforms, 1,366,022 SNPs obtained from both platforms were available for the GWAS analysis. Genome-wide imputation was then performed in LCLs. Specifically, SNPs that were not genotyped but found in the multi-race (ALL) of the1000 genomes project (11/23/2010 released version) were imputed by race using BEAGLE v3.3.1 (Browning and Browning, <a href="#B2" rid="B2" class=" bibr popnode">2007</a>). SNPs in the 1000 genomes project with MAF &#x0003c;0.01 were excluded as well SNPs with BEAGLE imputation dosage R<sup>2</sup> quality measure &#x0003c;0.3 were excluded from the further analysis, which resulted in a total of 5,384,559 imputed markers. Therefore, a total of 6,750,581 SNPs including imputed and genotyped were used in all the association studies described below.</p><p class="p p-last">For the expression array assays, total RNA was extracted from each of the cell lines using the RNeasy Mini Kit (QIAGEN Inc., Valencia, CA, USA). Total mRNA was assayed after hybridization to Affymetrix U133 plus 2.0 GenChips as previously described (Li et al., <a href="#B10" rid="B10" class=" bibr popnode">2008</a>; Niu et al., <a href="#B14" rid="B14" class=" bibr popnode">2010</a>). A total of 54,613 probe sets were used in the analysis.</p></div><div id="sec-a.r.b.e" class="sec"><h3 id="sec-a.r.b.etitle">Transient transfection and RNA interference</h3><p class="p p-first-last">SiRNAs for the candidate genes and negative control siRNA were purchased from QIAGEN (QIAGEN Inc., Valencia, CA, USA). 4&#x02013;5 &#x000d7; 10<sup>6</sup> cells were transfected with 500 nM of siRNA using Amaxa&#x02122; Cell Line Nucleofector&#x02122; transfection kit (Lonza, Koeln, Germany). After incubation overnight, cells were plated in 96-well plate for further drug treatment and MTS assay.</p></div><div id="sec-a.r.b.f" class="sec"><h3 id="sec-a.r.b.ftitle">Real-time quantitative reverse transcription PCR</h3><p class="p p-first-last">Total RNA was isolated from cultured cells transfected with control or specific siRNA with the ZR RNA MiniPrep&#x02122; kit (Zymo Research, Irvine, CA, USA), followed by one-step qRT-PCR performed with SYBR&#x000ae; Green PCR Master Mix Kit (Applied Biosystem, Foster City, CA, USA). Specific primers for mRNA amplification were purchased from QIAGEN.</p></div><div id="sec-a.r.b.g" class="sec sec-last"><h3 id="sec-a.r.b.gtitle">Statistical analysis</h3><div id="sec-a.r.b.g.b" class="sec sec-first"><p></p><h4 id="sec-a.r.b.g.btitle" class="inline">Drug toxicity </h4><p class="p p-first-last">The detailed description of analysis methods for accessing the association of cytotoxicity with SNP and/or mRNA expression data in these LCL cells has been described previously (Li et al., <a href="#B10" rid="B10" class=" bibr popnode">2008</a>; Niu et al., <a href="#B14" rid="B14" class=" bibr popnode">2010</a>; Jiang et al., <a href="#B7" rid="B7" class=" bibr popnode">2013</a>). Cytotoxicity phenotypes were determined by the best fitting curve using the R package &#x0201c;drc&#x0201d; (dose response curve) (<a href="http://cran.r-project.org/web/packages/drc.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cran.r-project.org/web/packages/drc.pdf</a>) based on a logistic model. The AUC phenotype was determined using the best fitting curve by numerically determining the area under the estimated dose-response curve, from dose 0 to 320 nmol/L.</p></div><div id="sec-a.r.b.g.c" class="sec"><p></p><h4 id="sec-a.r.b.g.ctitle" class="inline">Association analysis </h4><p class="p p-first-last">Partial correlations between: AUC and gene-expression; AUC and SNPs; and gene expression and SNPs were calculated; adjusting for sex, race, and variables addressing possible population stratification (sub-race variables). Partial correlations were then tested using <em>F</em>-tests, with multiple testing being addressed by <em>q</em>-values which control the false discovery rate (Storey, <a href="#B22" rid="B22" class=" bibr popnode">2003</a>). To construct the partial correlations, an adjusted AUC was constructed as the standardized residuals from a linear regression of Van der Waerden transformed AUC values regressed on gender, race and sub-race variables. Gene expression was first normalized using GCRMA (Ballman et al., <a href="#B1" rid="B1" class=" bibr popnode">2004</a>; Wu et al., <a href="#B27" rid="B27" class=" bibr popnode">2004</a>); log2 transformed normalized expression was then likewise regressed on gender, race and sub-race variables. SNP genotype was modeled as count of rare alleles, then regressed on gender, race and sub-race variables. Variables addressing possible population stratification (sub-race variables) were created from linkage disequilibrium thinned genome-wide SNPs using principle components analysis within each race (Price et al., <a href="#B18" rid="B18" class=" bibr popnode">2006</a>). The top 5 principle components were used within each race, thus analyses were adjusted for labeled cell line race and sub-racial features within each race (Li et al., <a href="#B10" rid="B10" class=" bibr popnode">2008</a>; Niu et al., <a href="#B14" rid="B14" class=" bibr popnode">2010</a>). For siRNA transfection experiments, group mean value for AUC and gene expression was compared by using Student's <em>t</em>-test.</p></div><div id="sec-a.r.b.g.d" class="sec sec-last"><p></p><h4 id="sec-a.r.b.g.dtitle" class="inline">Pathway analysis </h4><p class="p p-first-last">For pathway analysis of top associated genes, we used Ingenuity Pathway Analysis (Ingenuity&#x000ae; Systems, <a href="http://www.ingenuity.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ingenuity.com</a>) for network analysis.</p></div></div></div><div id="s3" class="tsec sec"><h2 class="head no_bottom_margin" id="s3title">Results</h2><div id="sec-a.r.c.b" class="sec sec-first"><h3 id="sec-a.r.c.btitle">HHT cytotoxicity</h3><p class="p p-first">HHT cytotoxicity studies were performed to determine the range of variation in HHT AUC among the 278 individual LCL cell lines from three ethnic groups. Figure <a href="/pmc/articles/PMC4292778/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1A</span></span><span>1A</span></a> shows representative HHT cytotoxicity data for a set of cell lines. The frequency distribution of AUC for HHT was shown in Figure <a href="/pmc/articles/PMC4292778/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1B.</span></span><span>1B</span></a>. AUC values did not show a significant difference among the three racial groups studied (<em>P</em> = 0.096, Figure <a href="/pmc/articles/PMC4292778/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1C).</span></span><span>1C</span></a>). Gender did not have a significant effect on AUC (<em>P</em> = 0.334, Figure <a href="/pmc/articles/PMC4292778/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure1D</span></span><span>1D</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4292778/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140326109819824"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4292778_fgene-05-00465-g0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-05-00465-g0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4292778/bin/fgene-05-00465-g0001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140326109819824"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4292778/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p><strong>Cytotoxicity of Homoharringtonine (HHT)</strong>. <strong>(A)</strong> Representative cytotoxicity dose response curves for HHT. Two cell lines from each of the three ethnic groups (AA, Africa American; CA, Caucasian American and HC, Han Chinese American) were selected to illustrate a range of HHT cytotoxicity. The X-axis indicates the log transformed dosage (nM) and the y-axis indicates the cell viability normalized to control (without drug treatment). Symbols represent individual cell line from different ethnic groups. <strong>(B)</strong> Histograms of frequency distributions of AUC values for HHT. <strong>(C)</strong> Race effect on HHT cytotoxicity (AUC values). Y-axis represents normalized AUC values. <strong>(D)</strong> Gender effect on HHT cytotoxicity (AUC values). Y-axis represents normalized AUC values.</p></div></div></div></div><div id="sec-a.r.c.c" class="sec"><h3 id="sec-a.r.c.ctitle">Correlation between expression and AUC</h3><p class="p p-first">We next performed correlation analysis for the association of expression array and HHT AUC data to identify genes which might be associated with HHT cytotoxicity at the gene expression level (Figure <a href="/pmc/articles/PMC4292778/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2A).</span></span><span>2A</span></a>). The association analysis identified 15 expression probe represented 10 annotated genes that were associated with HHT AUC with <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup>, and 96 individual probe sets represented 79 annotated genes with <em>P</em> &#x0003c; 10<sup>&#x02212;3</sup> (Supplementary Table <a href="#SM1" rid="SM1" class=" supplementary-material">S1</a>). The most significant probe set for an annotated gene was <em>DST</em> (<em>P</em> = 3.2 &#x000d7; 10<sup>&#x02212;6</sup>). Two other probe sets of this gene were also associated with HHT AUC (215016-x-at, <em>P</em> = 5.2 &#x000d7; 10<sup>&#x02212;5</sup>; 212253-x-at, <em>P</em> = 6.19 &#x000d7; 10<sup>&#x02212;5</sup>). <em>CTBP2</em> was found to have three expression probe sets associated with AUC value (<em>P</em> &#x0003c; 10<sup>&#x02212;4</sup>) (201218-at, <em>P</em> = 6.24 &#x000d7; 10<sup>&#x02212;6</sup>; 210835-s-at, <em>P</em> = 1.39 &#x000d7; 10<sup>&#x02212;5</sup>; 201220-x-at, <em>P</em> = 5.92 &#x000d7; 10<sup>&#x02212;5</sup>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4292778/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140326070043552"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4292778_fgene-05-00465-g0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-05-00465-g0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4292778/bin/fgene-05-00465-g0002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140326070043552"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4292778/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p><strong>Genome-wide association of mRNA expression and SNPs with HHT cytotoxicity</strong>. <strong>(A)</strong> Association of basal gene expression with HHT AUC value for 278 LCL cell lines. The y-axis represents the &#x02013;log<sub>10</sub>(<em>P</em>-value) for the association of individual expression array probe sets with HHT AUC, and the x-axis represents chromosomal location of expression probe sets. <strong>(B)</strong> Genome-wide SNPs association with HHT for 278 LCL cell lines. The y-axis represents the &#x02013;log<sub>10</sub>(<em>P</em>-value) for the association of Genome-wide SNPs with HHT AUC, and the x-axis represents chromosomal location of SNPs. A <em>P</em>-value of 10<sup>&#x02212;4</sup> is highlighted with a red line.</p></div></div></div><p class="p">For the functional validation, we chose genes that had probe sets associated with AUC value with <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup>. Among these genes, we removed genes with low expression levels in the LCLs (&#x0003c;50 after GCRMA normalization). Therefore, three genes including <em>DST, CTBP2</em> and <em>CCDC88A</em> were selected for further functional studies (Table <a href="/pmc/articles/PMC4292778/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1A).</span></span><span>1A</span></a>). Furthermore, we subjected genes that were associated with AUC with <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup> to a pathway based analysis using Ingenuity Pathway analysis tools. 5 gene signature networks that were significantly associated with the HHT were obtained (Table <a href="/pmc/articles/PMC4292778/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). The most significant network consists of 36 molecules or genes in network and 17/36 genes are associated with the functions such as cellular growth and proliferation, cell death and survival (Table <a href="/pmc/articles/PMC4292778/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table2).</span></span><span>2</span></a>). The IPA network provides graphical representation of the biological connections between genes as shown in <strong>Figure 4</strong>.</p><!--table ft1--><!--table-wrap-group mode="anchored" t5--><div class="table-wrap-group anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p><strong>Candidate gene selected for siRNA screening based on GWA analysis</strong>.</p></div><!--table-wrap mode="anchored" t6--><div class="table-wrap anchored whole_rhythm" id="d35e633"><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" colspan="6" rowspan="1"><strong>A. mRNA Exp vs. AUC (PANEL 1)</strong></th></tr><tr><th align="left" rowspan="1" colspan="1"><strong>Gene symbol</strong></th><th align="center" rowspan="1" colspan="1"><strong>Chr</strong>.</th><th align="center" rowspan="1" colspan="1"><strong>Probe id</strong></th><th align="center" rowspan="1" colspan="1"><strong><em>P</em></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>R<sup>*</sup></em></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>Q</em><sup>**</sup></strong></th></tr></thead><tbody><tr><td align="center" rowspan="1" colspan="1">DST</td><td align="center" rowspan="1" colspan="1">6</td><td align="center" rowspan="1" colspan="1">212254-S-at</td><td align="center" rowspan="1" colspan="1">3.20E&#x02013;06</td><td align="center" rowspan="1" colspan="1">0.28</td><td align="center" rowspan="1" colspan="1">0.14</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">215016-x-at</td><td align="center" rowspan="1" colspan="1">5.20E&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.24</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">212253-x-at</td><td align="center" rowspan="1" colspan="1">6.19E&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.24</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td align="center" rowspan="1" colspan="1">CTBP2</td><td align="center" rowspan="1" colspan="1">10</td><td align="center" rowspan="1" colspan="1">201218-at</td><td align="center" rowspan="1" colspan="1">6.24E&#x02013;06</td><td align="center" rowspan="1" colspan="1">&#x02212;0.27</td><td align="center" rowspan="1" colspan="1">0.14</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">210835-s-at</td><td align="center" rowspan="1" colspan="1">1.39E&#x02013;05</td><td align="center" rowspan="1" colspan="1">&#x02212;0.26</td><td align="center" rowspan="1" colspan="1">0.16</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">201220-x-at</td><td align="center" rowspan="1" colspan="1">5.92E&#x02013;05</td><td align="center" rowspan="1" colspan="1">&#x02212;0.24</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td align="center" rowspan="1" colspan="1">DC88A</td><td align="center" rowspan="1" colspan="1">2</td><td align="center" rowspan="1" colspan="1">238759-at</td><td align="center" rowspan="1" colspan="1">3.71E&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.25</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">221078_s_at</td><td align="center" rowspan="1" colspan="1">1.76E&#x02013;04</td><td align="center" rowspan="1" colspan="1">0.29</td><td align="center" rowspan="1" colspan="1">0.23</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140326061184272"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/table/d35e633/?report=objectonly">Open in a separate window</a></div></div><!--table-wrap mode="anchored" t6--><div class="table-wrap anchored whole_rhythm" id="d35e764"><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" colspan="8" rowspan="1"><strong>B. SNP vs. AUC (PANEL 2)</strong></th></tr><tr><th align="left" rowspan="1" colspan="1"><strong>Gene symbol</strong></th><th align="center" rowspan="1" colspan="1"><strong>Chr</strong>.</th><th align="left" rowspan="1" colspan="1"><strong>SNP</strong></th><th align="center" rowspan="1" colspan="1"><strong>Position</strong></th><th align="center" rowspan="1" colspan="1"><strong>Location</strong></th><th align="center" rowspan="1" colspan="1"><strong>MAF<sup>***</sup></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>P</em></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>R</em></strong></th></tr></thead><tbody><tr><td align="center" rowspan="1" colspan="1">WDHD1/SOCS4</td><td align="center" rowspan="1" colspan="1">14</td><td align="left" rowspan="1" colspan="1">rs1187882</td><td align="left" rowspan="1" colspan="1">55488747</td><td align="left" rowspan="1" colspan="1">Intron of WDHD1 5&#x02032;-upstream of SOCS4</td><td align="center" rowspan="1" colspan="1">0.15</td><td align="center" rowspan="1" colspan="1">5.07e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.25</td></tr><tr><td align="center" rowspan="1" colspan="1">SOCS4/WDHD1</td><td align="center" rowspan="1" colspan="1">14</td><td align="left" rowspan="1" colspan="1">rs60805089</td><td align="left" rowspan="1" colspan="1">55503348</td><td align="left" rowspan="1" colspan="1">Intron of SOCS4 3&#x02032;-downstream of WDHD1</td><td align="center" rowspan="1" colspan="1">0.24</td><td align="center" rowspan="1" colspan="1">9.73e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.36</td></tr><tr><td align="center" rowspan="1" colspan="1">SOCS4</td><td align="center" rowspan="1" colspan="1">14</td><td align="left" rowspan="1" colspan="1">rs7150145</td><td align="left" rowspan="1" colspan="1">55508260</td><td align="left" rowspan="1" colspan="1">Intron</td><td align="center" rowspan="1" colspan="1">0.24</td><td align="center" rowspan="1" colspan="1">8.92e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.36</td></tr><tr><td align="center" rowspan="1" colspan="1">ZNF638</td><td align="center" rowspan="1" colspan="1">2</td><td align="left" rowspan="1" colspan="1">rs2058899</td><td align="left" rowspan="1" colspan="1">71580129</td><td align="left" rowspan="1" colspan="1">Intron</td><td align="center" rowspan="1" colspan="1">0.37</td><td align="center" rowspan="1" colspan="1">6.94e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">2</td><td align="left" rowspan="1" colspan="1">rs17749748</td><td align="left" rowspan="1" colspan="1">71605145</td><td align="left" rowspan="1" colspan="1">Intron</td><td align="center" rowspan="1" colspan="1">0.38</td><td align="center" rowspan="1" colspan="1">7.70e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">2</td><td align="left" rowspan="1" colspan="1">rs12474072</td><td align="left" rowspan="1" colspan="1">71636036</td><td align="left" rowspan="1" colspan="1">Intron</td><td align="center" rowspan="1" colspan="1">0.38</td><td align="center" rowspan="1" colspan="1">7.78e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">2</td><td align="left" rowspan="1" colspan="1">rs6714975</td><td align="left" rowspan="1" colspan="1">71633389</td><td align="left" rowspan="1" colspan="1">Intron</td><td align="center" rowspan="1" colspan="1">0.38</td><td align="center" rowspan="1" colspan="1">9.42e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">2</td><td align="left" rowspan="1" colspan="1">rs4852782</td><td align="left" rowspan="1" colspan="1">71636240</td><td align="left" rowspan="1" colspan="1">Intron</td><td align="center" rowspan="1" colspan="1">0.38</td><td align="center" rowspan="1" colspan="1">9.42e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.24</td></tr><tr><td rowspan="1" colspan="1"></td><td align="center" rowspan="1" colspan="1">2</td><td align="left" rowspan="1" colspan="1">rs6745907</td><td align="left" rowspan="1" colspan="1">71609073</td><td align="left" rowspan="1" colspan="1">Intron</td><td align="center" rowspan="1" colspan="1">0.38</td><td align="center" rowspan="1" colspan="1">9.66e&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.24</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140326073322240"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/table/d35e764/?report=objectonly">Open in a separate window</a></div></div><!--table-wrap mode="anchored" t6--><div class="table-wrap anchored whole_rhythm" id="d35e951"><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" colspan="14" rowspan="1"><strong>C. INTEGRATED ANALYSIs (PANEL 3)</strong></th></tr><tr><th align="center" colspan="5" rowspan="1"><strong>SNP</strong></th><th align="center" colspan="3" rowspan="1"><strong>Genes</strong></th><th align="center" colspan="2" rowspan="1"><strong>SNP vs. EXP</strong></th><th align="center" colspan="2" rowspan="1"><strong>SNP vs. AUC</strong></th><th align="center" colspan="2" rowspan="1"><strong>EXP vs. AUC</strong></th></tr><tr><th align="left" rowspan="1" colspan="1"><strong>SNP</strong></th><th align="left" rowspan="1" colspan="1"><strong>Chr</strong>.</th><th align="left" rowspan="1" colspan="1"><strong>Closest gene</strong></th><th align="left" rowspan="1" colspan="1"><strong>SNP location</strong></th><th align="center" rowspan="1" colspan="1"><strong>MAF</strong></th><th align="left" rowspan="1" colspan="1"><strong>Probe id</strong></th><th align="left" rowspan="1" colspan="1"><strong>Chr</strong>.</th><th align="left" rowspan="1" colspan="1"><strong>Gene symbol</strong></th><th align="center" rowspan="1" colspan="1"><strong><em>P</em></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>R<sup>*</sup></em></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>P</em></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>R<sup>*</sup></em></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>P</em></strong></th><th align="center" rowspan="1" colspan="1"><strong><em>R<sup>*</sup></em></strong></th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">rs10771093</td><td align="center" rowspan="1" colspan="1">12</td><td align="left" rowspan="1" colspan="1">SOX5</td><td align="left" rowspan="1" colspan="1">Inton</td><td align="center" rowspan="1" colspan="1">0.214</td><td align="left" rowspan="1" colspan="1">212253_x_at</td><td align="center" rowspan="1" colspan="1">6</td><td align="left" rowspan="1" colspan="1">DST</td><td align="center" rowspan="1" colspan="1">5.34E-05</td><td align="center" rowspan="1" colspan="1">0.242</td><td align="center" rowspan="1" colspan="1">2.37E&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.258</td><td align="center" rowspan="1" colspan="1">3.20E&#x02013;06</td><td align="center" rowspan="1" colspan="1">0.278</td></tr><tr><td align="left" rowspan="1" colspan="1">rs60342295</td><td align="center" rowspan="1" colspan="1">17</td><td align="left" rowspan="1" colspan="1">&#x02014;</td><td align="left" rowspan="1" colspan="1">&#x02014;</td><td align="center" rowspan="1" colspan="1">0.119</td><td align="left" rowspan="1" colspan="1">1566990_x_at</td><td align="center" rowspan="1" colspan="1">6</td><td align="left" rowspan="1" colspan="1">ARID1B</td><td align="center" rowspan="1" colspan="1">9.73E-05</td><td align="center" rowspan="1" colspan="1">&#x02212;0.234</td><td align="center" rowspan="1" colspan="1">3.28E&#x02013;05</td><td align="center" rowspan="1" colspan="1">&#x02212;0.254</td><td align="center" rowspan="1" colspan="1">0.000811</td><td align="center" rowspan="1" colspan="1">&#x02212;0.202</td></tr><tr><td align="left" rowspan="1" colspan="1">rs4531814</td><td align="center" rowspan="1" colspan="1">18</td><td align="left" rowspan="1" colspan="1">ZBTB7C</td><td align="left" rowspan="1" colspan="1">Inton</td><td align="center" rowspan="1" colspan="1">0.428</td><td align="left" rowspan="1" colspan="1">231325_at</td><td align="center" rowspan="1" colspan="1">8</td><td align="left" rowspan="1" colspan="1">UNC5D</td><td align="center" rowspan="1" colspan="1">0.00031</td><td align="center" rowspan="1" colspan="1">&#x02212;0.217</td><td align="center" rowspan="1" colspan="1">7.95E&#x02013;05</td><td align="center" rowspan="1" colspan="1">0.241</td><td align="center" rowspan="1" colspan="1">9.80E&#x02013;06</td><td align="center" rowspan="1" colspan="1">&#x02212;0.264</td></tr><tr><td align="left" rowspan="1" colspan="1">rs58707448</td><td align="center" rowspan="1" colspan="1">18</td><td align="left" rowspan="1" colspan="1">&#x02014;</td><td align="left" rowspan="1" colspan="1">&#x02014;</td><td align="center" rowspan="1" colspan="1">0.482</td><td align="left" rowspan="1" colspan="1">205352_at</td><td align="center" rowspan="1" colspan="1">3</td><td align="left" rowspan="1" colspan="1">SERPINI1</td><td align="center" rowspan="1" colspan="1">0.000138</td><td align="center" rowspan="1" colspan="1">0.229</td><td align="center" rowspan="1" colspan="1">4.60E&#x02013;05</td><td align="center" rowspan="1" colspan="1">&#x02212;0.249</td><td align="center" rowspan="1" colspan="1">6.96E&#x02013;05</td><td align="center" rowspan="1" colspan="1">&#x02212;0.238</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140326106028080"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/table/d35e951/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN1"><sup>*</sup><em>Represents the correlation coefficient R-value for associations</em>.</div><div id="TN2"><sup>**</sup><em>Represents the false discovery rate Q-value</em>.</div><div id="TN3"><sup>***</sup><em>Represents the minor allele frequency (MAF)</em>.</div></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p><strong>Top 5 networks associated with HHT response obtained by Ingenuity pathway analysis</strong>.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1"><strong>ID</strong></th><th align="left" rowspan="1" colspan="1"><strong>molecules in network</strong></th><th align="left" rowspan="1" colspan="1"><strong>Score</strong></th><th align="center" rowspan="1" colspan="1"><strong>Focus molecules</strong></th><th align="left" rowspan="1" colspan="1"><strong>Top diseases and functions</strong></th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">1</td><td align="left" rowspan="1" colspan="1">AICDA, AKR1B1, Akt, ANKRD1, BANK1, CASP2, CASP12, caspase, CBS/LOC102724560, CCDC88A, CD3, Ck2, EBF1, ERK1/2, FADD, IgG1, Igm, Immun oglobulin, Insulin, Interferon alpha, Jnk, NAP1L1, NFkB (complex), NRF1, P38 MAPK, PAWR, PPIA, PRDX3, PRKCE, SCD5, SER PINB9, SIRT4, SPRY4, STX6, TRAF5</td><td align="center" rowspan="1" colspan="1">36</td><td align="center" rowspan="1" colspan="1">17</td><td align="left" rowspan="1" colspan="1">Cellular growth and proliferation, cell death and Survival, renal necrosis/cell death</td></tr><tr><td align="left" rowspan="1" colspan="1">2</td><td align="left" rowspan="1" colspan="1">ALG1L2, ATG9A, BRD3, C14orf2, CALM1 (includes others), CEP68, CHD4, CRYAA/LOC102724652, CTBP1, FAM178A, FXR2, GID8, GIMAP1, GIMAP7, GJA3, GON4L, HAGHL, ITFG1, LCMT1, MAEA, MAP1LC3B, MKLN1, MRPL44, MTMR14, OPTN, PPP2R4, PPP2R2B, PTEN, RAB30, RMND5A, SCCPDH, SMAD9, UBC, WHSC1L1, ZNF143</td><td align="center" rowspan="1" colspan="1">33</td><td align="center" rowspan="1" colspan="1">16</td><td align="left" rowspan="1" colspan="1">Ophthalmic disease, hereditary disorder, cellular Assembly and organization</td></tr><tr><td align="left" rowspan="1" colspan="1">3</td><td align="left" rowspan="1" colspan="1">APP, ARID1B, BAZ2B, C15orf39, CASP12, CDK3, CLIP4, COL17A1, CTBP2, CTPS2, DOCK4, DST, ELAVL1, FAM117B, FAM178A, FBXL2, FNDC3B, GATAD2A, GIMAP5, GIMAP6, IL13, ISCU, KIAA1549, KLF8, MOB1B, NEFH, PLXDC2, RAB33A, RAI2, RALGPS1, RASSF3, SH3BP5, TWISTNB, UBC, ZBTB20</td><td align="center" rowspan="1" colspan="1">33</td><td align="center" rowspan="1" colspan="1">16</td><td align="left" rowspan="1" colspan="1">Cellular compromise, neurological disease, Organismal injury and abnormalities</td></tr><tr><td align="left" rowspan="1" colspan="1">4</td><td align="left" rowspan="1" colspan="1">ACACA, AFF3, AKR1B1, APOBR, BCL3, CASP2, CLCN4, CMC4, COA4, COPS7B, DDIT3, EPO, FAHD2A, FOXO1, GNS, Hdac, HNF4A, IRF2BP2, ITGA6, KAT2B, MON1B, NRIP1, NTN1, NUDCD3, OCLN, ONECUT1, PPP1R15A, RASSF1, STK4, TEX10, TNF, UNC5D, UXT, VASH1, ZBTB45</td><td align="center" rowspan="1" colspan="1">28</td><td align="center" rowspan="1" colspan="1">14</td><td align="left" rowspan="1" colspan="1">Cell death and survival, renal necrosis/cell death, Lymphoid tissue structure and development</td></tr><tr><td align="left" rowspan="1" colspan="1">5</td><td align="left" rowspan="1" colspan="1">AChR, Akap9, CDH1, CREB1, DRD2, ERBB2, ESR1, G6PC, GCG, GPATCH2, HDAC2, IFNG, ITPR1, KIAA2022, LDL, NFKB1, NPY, phosphatase, PI3K(complex), PIAS1, PIAS3, Pka catalytic subunit, PLAT, POU5F1, PPP1CA, PPP1R15A, PPP3CB, PRKACA, PTEN, PTPRN, SERPINI1, SHIP, sphingomyelinase, SYNJ2, VCPIP1</td><td align="center" rowspan="1" colspan="1">8</td><td align="center" rowspan="1" colspan="1">5</td><td align="left" rowspan="1" colspan="1">Cellular compromise, neurological disease, Organismal injury and abnormalities</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140326062519552"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec-a.r.c.d" class="sec"><h3 id="sec-a.r.c.dtitle">Genome-wide SNP association with HHT AUC</h3><p class="p p-first">We performed an analysis of the association of genome-wide SNPs with HHT AUC (Figure <a href="/pmc/articles/PMC4292778/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2B).</span></span><span>2B</span></a>). A total of 561,303 SNPs on the Illumina 550K SNP array and 487,044 SNPs on the Illumina 510S SNP array had been genotyped using DNA from each of these 278 cell lines. We also had access to publicly available Affymetrix 6.0 SNP array data. After quality control and genome wide imputation, a total of 6,750,581 SNPs were used in the association studies.</p><p>GWAS was performed between SNPs and HHT AUC values. Although none of the SNPs reached genome-wide significance (<em>P</em> &#x0003c; 10<sup>&#x02212;8</sup>), 18 SNPs had <em>P</em> &#x0003c; 10<sup>&#x02212;5</sup> and 281 SNPs had <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup> (Supplementary Table <a href="#SM2" rid="SM2" class=" supplementary-material">S2</a>). The <em>P</em> value for the most significant SNPs (rs1250991) was 3.1 &#x000d7; 10<sup>&#x02212;7</sup>.</p><p class="p p-last">For functional validation, we identified 15 regions/loci that contained at least two SNPs with <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup> within 50 kb in each of these regions (a total of 95 SNPs). We defined these regions as &#x0201c;SNP peak region.&#x0201d; From these regions, we picked up three genes which were also expressed in LCLs and leukemia cells for further analysis (Table <a href="/pmc/articles/PMC4292778/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1B</span></span><span>1B</span></a>).</p></div><div id="sec-a.r.c.e" class="sec"><h3 id="sec-a.r.c.etitle">Integrated analysis</h3><p class="p p-first">The effect of genetic variation on HHT-induced cytotoxicity might result in part from the regulation of gene expression. Therefore, we performed an integrated analysis that included data from SNPs, basal expression, and HHT AUC, as reported before (Li et al., <a href="#B11" rid="B11" class=" bibr popnode">2009</a>; Niu et al., <a href="#B14" rid="B14" class=" bibr popnode">2010</a>; Jiang et al., <a href="#B7" rid="B7" class=" bibr popnode">2013</a>).</p><p class="p p-last">Specifically, for the top HHT associated SNPs, (i.e., SNP with <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup>), we determined their association with gene expression using <em>P</em> &#x0003c; 10<sup>&#x02212;4</sup> as a cutoff. These SNP&#x02013;associated genes were then narrowed down to those whose mRNA gene expression probe sets that were also associated with HHT cytotoxicity (<em>P</em> &#x0003c; 10<sup>&#x02212;3</sup>). We used the less stringent criteria to capture more potential candidate genes for further functional validation. Through this integrated analysis, we identified 4 unique SNPs that were associated with expression and AUC (Table <a href="/pmc/articles/PMC4292778/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table1C).</span></span><span>1C</span></a>). None of the SNPs were in cis-regulatory region.</p></div><div id="sec-a.r.c.f" class="sec sec-last"><h3 id="sec-a.r.c.ftitle">Functional validation of candidate genes in human leukemia cells</h3><p class="p p-first">In summary, we selected 7 genes based on the strategy shown in Figure <a href="/pmc/articles/PMC4292778/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure3</span></span><span>3</span></a> to perform further functional validation in human leukemia cell lines: U937 and K562. Since HHT is commonly used in the patients of AML and CML, leukemic monocyte cell line U937 and human erythromyeloblastoid leukemia cell line K562 cells were used for further studies. These functional experiments involved siRNA knockdown, followed by MTS cytotoxicity. Knocking down <em>CCDC88A, CTBP2</em>, and <em>SOCS4</em> genes exhibited significant resistance to HHT sensitivity in U937 and K562 cells (Figure <a href="/pmc/articles/PMC4292778/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure5).</span></span><span>5</span></a>). The others did not show significantly impact on HHT sensitivity.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4292778/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140326072742912"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4292778_fgene-05-00465-g0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-05-00465-g0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4292778/bin/fgene-05-00465-g0003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140326072742912"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4292778/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p><strong>Schematic diagram of the strategy for selecting candidate genes for functional validation</strong>. A total of 13 candidate genes were selected based on genome-wide association of expression (Exp) vs. AUC, SNP vs. AUC and &#x0201c;Integrated&#x0201d; analysis, as described in the text. After removing those that were not expressed in our LCLs, 7 genes were further selected for functional validation.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC4292778/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140326071385072"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4292778_fgene-05-00465-g0005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-05-00465-g0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4292778/bin/fgene-05-00465-g0005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140326071385072"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC4292778/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p><strong>SiRNA screening of candidate genes by MTS assay in leukemia cell lines</strong>. Data are shown for 3 of the 7 candidate genes that were studied in U937, K562 leukemia cell lines by MTS assay after siRNA knockdown. Experiments were repeated in triplicate with at least two independent experiments. Error bars indicate standard error of the mean (SEM) values. Significance was defined by <em>P</em>-values. <strong>(A)</strong> knockdown efficiency was determined by qRT-PCR. The y-axis indicates relative gene expression after siRNA knockdown when compared with negative control siRNA. <strong>(B)</strong> MTS assays. The x-axis indicates the log transformed HHT dose, and the y-axis indicates the surviving fraction after exposure to HHT.</p></div></div></div><p class="p p-last">Taken together, through the initial screening studies in LCLs and further functional validation in leukemia cell lines, we found that the expression of three genes, <em>CCDC88A, CTBP2</em>, and <em>SOCS4</em>, were involved in HHT-induced response.</p></div></div><div id="s4" class="tsec sec"><h2 class="head no_bottom_margin" id="s4title">Discussion</h2><p class="p p-first">HHT is a commonly used drug for the treatment of acute and chronic myeloid leukemia in China. Recently it has also been approved by FDA for the treatment of CML resistant to TKIs. The addition of HHT improved both CR rate and Relapse Free Survival (RFS) rate. However, its major dose-limiting toxicity is bone marrow suppression. The response to HHT varied widely as some patients showed no response whereas the bone-marrow suppression was severe. Therefore, a better understanding of the biology underlying the variation to HHT could help us maximize HHT efficacy while minimizing drug related toxicity.</p><p>Recent studies have raised the possibility that genetic variation might contribute to individual variation in drug response. Many pharmacogenomic studies have successfully used LCL models to illustrate the contribution of germline genetic variations to variation in chemotherapeutic drug response. For example, using the LCL system, previous findings have identified potential biomarkers for response to a series of drugs including Ara-C, 5-FU, Asparaginase, Daunorubicin (Watters et al., <a href="#B26" rid="B26" class=" bibr popnode">2004</a>; Huang et al., <a href="#B6" rid="B6" class=" bibr popnode">2007</a>; Peters et al., <a href="#B17" rid="B17" class=" bibr popnode">2009</a>; Chen et al., <a href="#B4" rid="B4" class=" bibr popnode">2011</a>). In some cases, findings from the LCLs were further validated in clinical studies. <em>FKBP5</em>, which was found as a biomarker of Ara-C response using the LCL system, was further studied at mechanistic levels showing that FKBP5 promoted the dephosphorylation of AKT and downregulated the AKT activity, which in turn, increased cytotoxicity by triggering apoptosis in the presence of Ara-C (Pei et al., <a href="#B16" rid="B16" class=" bibr popnode">2009</a>). These findings were validated in a clinical study, in which SNPs within <em>FKBP5</em> were genotyped in a cohort of 187 pediatric acute myeloid leukemia patients treated with Ara-C. Two of the SNPs were associated with both event-free and overall survival (Mitra et al., <a href="#B12" rid="B12" class=" bibr popnode">2011</a>). Based on the potential impact of these findings in individualizing chemotherapy, we aimed to use this well-established LCLs model to find biomarkers responsible for HHT cytotoxicity.</p><p>In the present study, we performed a genome-wide association analysis using 278 LCLs for which we had 1.3 million SNPs, basal gene expression data, and HHT cytotoxicity AUC phenotype to identify genes which might be responsible for HHT sensitivity. Based on these data, our study identified top HHT cytotoxicity associated gene networks (Figure <a href="/pmc/articles/PMC4292778/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure4</span></span><span>4</span></a> and Table <a href="/pmc/articles/PMC4292778/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table2)</span></span><span>2</span></a>) that might help generate hypothesis as well as yield a total of 7 genes which were also expressed in LCLs and leukemia cell lines for further studies. The LCL system used in our screening studies has limitations. EBV transformation might cause chromosomal instability and the cellular changes in LCLs (Sie et al., <a href="#B21" rid="B21" class=" bibr popnode">2009</a>), and other factors such as cell growth rate and ATP level can also have effect on cytotoxicity (Choy et al., <a href="#B5" rid="B5" class=" bibr popnode">2008</a>). Since these LCL cell lines may not represent the response of leukemia cell lines to HHT, we identified 7 top candidate genes to perform functional validation studies using siRNA knockdown in two leukemia cell lines: U937 and K562 cells. U937 cells lines are derived from leukemic monocyte cells and K562 cells are also one of myelogenous leukemia cell lines. These two cell lines, tended to show similar results for the three genes (<em>CCDC88A, CTBP2, SOCS4</em>) tested. Our results suggested that knockdown of these genes made cells more resistant to HHT.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC4292778/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140326070336688"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4292778_fgene-05-00465-g0004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-05-00465-g0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4292778/bin/fgene-05-00465-g0004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140326070336688"><a target="object" rel="noopener" href="/pmc/articles/PMC4292778/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4292778/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p><strong>Gene network analysis of top HHT associated genes</strong>. Ingenuity pathway analysis was performed. Each node in the network represents a gene and edge represents a relationship, the pink colored nodes in the network are the significant genes that are correlated with HHT AUC phenotype.</p></div></div></div><p class="p p-last"><em>CCDC88A</em>, also named as Coiled-Coil Domain Containing 88A, encodes a member of the Girdin family of coiled-coil domain containing proteins. The encoded protein enhances Akt signaling by mediating phosphoinositide 3-kinase (PI3K)-dependent activation of Akt. AKT activation regulates DNA replication and cell proliferation by phosphorylating the downstream effectors GSK3 and FOXO1/FKHR. The PI3K/AKT and mTOR signaling pathways are activated in acute myeloid leukemia (Park et al., <a href="#B15" rid="B15" class=" bibr popnode">2010</a>). Although the mechanism responsible for CCDC88A involvement in HHT response is unclear, given the roles of AKT in the development of leukemia, it was not surprising that this gene might play a role in variable response to HHT. However, we recognized that the CCDC88A had a positive association with HHT cytotoxicity in LCLs, while our knockdown showed the opposite effect, which might be due to the difference between LCLs and Leukemia cell lines. Further mechanistic studies will be needed to confirm the results. The protein encoded by <em>SOCS4</em> belongs to the suppressor of cytokine signaling (SOCS), also known as STAT-induced STAT inhibitor (SSI), protein family. SOCS family members are known to be cytokine-inducible negative regulators of cytokine signaling, including JAK1/STAT3 pathway. SOCS4 has been also demonstrated to regulate EGFR signaling <em>in vitro</em> (Chan et al., <a href="#B3" rid="B3" class=" bibr popnode">2009</a>). Therefore, regulation of both pathways could contribute to HHT cytotoxicity. <em>CTBP2</em> gene produces alternative transcripts encoding two distinct proteins. One protein is a transcriptional repressor, while the other isoform is a major component of specialized synapses known as synaptic ribbons. Transcription corepressor CTBP2 has been reported to directly bind acinus, which is regulated by NGF (nerve growth factor), inhibiting its stimulatory effect on cyclin A1 expression in leukemia (Trengove and Ward, <a href="#B24" rid="B24" class=" bibr popnode">2013</a>). Although there is limited information with regard to the involvement of these genes to HHT response, our results suggest a possible role for their relationship with variation in HHT response. Further mechanistic studies of these genes as well as additional association studies in patients treated with HHT are needed to validate the genes as biomarkers of HHT response.</p></div><div id="s5" class="tsec sec"><h2 class="head no_bottom_margin" id="s5title">Conclusions</h2><p class="p p-first-last">In summary, we used a pharmacogenomics approach based on the use of genomic data rich LCLs system to identify genetic candidate that might contribute to HHT cytotoxicity. Functional validation of these candidate genes may further support the feasibility of utilizing these genes for clinical validation of HHT response. These results may enhance our ability to individualized treatment with HHT.</p></div><div id="sec-a.r.f" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.r.ftitle">Author contributions</h2><p class="p p-first-last">Yin Tong and Liewei Wang designed the study and wrote the manuscript. Yin Tong performed the experiments. Gregory Jenkins and Anthony Batzler performed the statistical analyses. All the authors read, revised the draft manuscript and approved the final version.</p></div><div id="sec-a.r.g" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.r.gtitle">Conflict of interest statement</h2><p class="p p-first-last">The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></div><div id="ack-a.s.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.s.atitle">Acknowledgments</h2><div class="sec"><p>This work was supported by National Natural Science foundation of China (81370646). Liewei Wang was supported by NIGMS U19 GM61388 (Pharmacogenomics Research Network).</p></div></div><div id="s6" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="s6title">Supplementary material</h2><!--/article/back/sec/--><p class="p p-first">The Supplementary Material for this article can be found online at: <a href="http://www.frontiersin.org/journal/10.3389/fgene.2014.00465/abstract" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.frontiersin.org/journal/10.3389/fgene.2014.00465/abstract</a></p><div class="sec suppmat" id="SM1"><div class="sup-box half_rhythm" id="media-a.s.b.c.a"><a href="/pmc/articles/PMC4292778/bin/Table1.PDF" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(28K, PDF)</sup></div></div><div class="sec suppmat" id="SM2"><div class="sup-box half_rhythm" id="media-a.s.b.d.a"><a href="/pmc/articles/PMC4292778/bin/Table2.PDF" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(76K, PDF)</sup></div></div></div><div id="ref-list-a.s.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.s.ctitle">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="B1"><span class="mixed-citation">Ballman K. V., Grill D. E., Oberg A. L., Therneau T. M. (2004). <span class="ref-title">Faster cyclic loess: normalizing RNA arrays via linear models</span>. <span class="ref-journal">Bioinformatics</span>
<span class="ref-vol">20</span>, 2778&#x02013;2786. 10.1093/bioinformatics/bth327 [<a href="https://pubmed.ncbi.nlm.nih.gov/15166021" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbth327" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Faster+cyclic+loess:+normalizing+RNA+arrays+via+linear+models&amp;author=K.+V.+Ballman&amp;author=D.+E.+Grill&amp;author=A.+L.+Oberg&amp;author=T.+M.+Therneau&amp;volume=20&amp;publication_year=2004&amp;pages=2778-2786&amp;pmid=15166021&amp;doi=10.1093/bioinformatics/bth327&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B2"><span class="mixed-citation">Browning S. R., Browning B. L. (2007). <span class="ref-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering</span>. <span class="ref-journal">Am. J. Hum. Genet</span>. <span class="ref-vol">81</span>, 1084&#x02013;1097. 10.1086/521987 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265661/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17924348" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1086%2F521987" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Rapid+and+accurate+haplotype+phasing+and+missing-data+inference+for+whole-genome+association+studies+by+use+of+localized+haplotype+clustering&amp;author=S.+R.+Browning&amp;author=B.+L.+Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;doi=10.1086/521987&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B3"><span class="mixed-citation">Chan C. B., Liu X., Jang S. W., Hsu S. I. H., Williams I., Kang S., et al.. (2009). <span class="ref-title">NGF inhibits human leukemia proliferation by downregulating cyclin A1 expression through promoting acinus/CtBP2 association</span>. <span class="ref-journal">Oncogene</span>
<span class="ref-vol">28</span>, 3825&#x02013;3836. 10.1038/onc.2009.236 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3481846/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19668232" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fonc.2009.236" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=NGF+inhibits+human+leukemia+proliferation+by+downregulating+cyclin+A1+expression+through+promoting+acinus/CtBP2+association&amp;author=C.+B.+Chan&amp;author=X.+Liu&amp;author=S.+W.+Jang&amp;author=S.+I.+H.+Hsu&amp;author=I.+Williams&amp;volume=28&amp;publication_year=2009&amp;pages=3825-3836&amp;pmid=19668232&amp;doi=10.1038/onc.2009.236&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B4"><span class="mixed-citation">Chen S. H., Yang W., Fan Y., Stocco G., Crews K. R., Yang J. J., et al.. (2011). <span class="ref-title">A genome-wide approach identifies that the aspartate metabolism pathway contributes to asparaginase sensitivity</span>. <span class="ref-journal">Leukemia</span>
<span class="ref-vol">25</span>, 66&#x02013;74. 10.1038/leu.2010.256 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3097057/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21072045" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fleu.2010.256" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=A+genome-wide+approach+identifies+that+the+aspartate+metabolism+pathway+contributes+to+asparaginase+sensitivity&amp;author=S.+H.+Chen&amp;author=W.+Yang&amp;author=Y.+Fan&amp;author=G.+Stocco&amp;author=K.+R.+Crews&amp;volume=25&amp;publication_year=2011&amp;pages=66-74&amp;pmid=21072045&amp;doi=10.1038/leu.2010.256&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B5"><span class="mixed-citation">Choy E., Yelensky R., Bonakdar S., Plenge R. M., Saxena R., De Jager P. L., et al.. (2008). <span class="ref-title">Genetic analysis of human traits <em>in vitro</em>: drug response and gene expression in lymphoblastoid cell lines</span>. <span class="ref-journal">PLoS Genet</span>
<span class="ref-vol">4</span>:e100028. 10.1371/journal.pgen.1000287 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2583954/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19043577" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pgen.1000287" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetic+analysis+of+human+traits+in+vitro:+drug+response+and+gene+expression+in+lymphoblastoid+cell+lines&amp;author=E.+Choy&amp;author=R.+Yelensky&amp;author=S.+Bonakdar&amp;author=R.+M.+Plenge&amp;author=R.+Saxena&amp;volume=4&amp;publication_year=2008&amp;pages=e100028&amp;pmid=19043577&amp;doi=10.1371/journal.pgen.1000287&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B6"><span class="mixed-citation">Huang R. S., Duan S., Bleibel W. K., Kistner E. O., Zhang W., Clark T. A., et al.. (2007). <span class="ref-title">A genome-wide approach to identify genetic variants that contribute to etoposide-induced cytotoxicity</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A</span>. <span class="ref-vol">104</span>, 9758&#x02013;9763. 10.1073/pnas.0703736104 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1887589/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17537913" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.0703736104" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A&amp;title=A+genome-wide+approach+to+identify+genetic+variants+that+contribute+to+etoposide-induced+cytotoxicity&amp;author=R.+S.+Huang&amp;author=S.+Duan&amp;author=W.+K.+Bleibel&amp;author=E.+O.+Kistner&amp;author=W.+Zhang&amp;volume=104&amp;publication_year=2007&amp;pages=9758-9763&amp;pmid=17537913&amp;doi=10.1073/pnas.0703736104&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B7"><span class="mixed-citation">Jiang J., Fridley B. L., Feng Q., Abo R., Brisbin A., Batzler A., et al.. (2013). <span class="ref-title">Genome-wide association study for biomarker identification of Rapamycin and Everolimus using a lymphoblastoid cell line system</span>. <span class="ref-journal">Front. Genet</span>. <span class="ref-vol">4</span>:<span class="ref-iss">166</span>. 10.3389/fgene.2013.00166 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3757297/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24009623" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffgene.2013.00166" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Genet&amp;title=Genome-wide+association+study+for+biomarker+identification+of+Rapamycin+and+Everolimus+using+a+lymphoblastoid+cell+line+system&amp;author=J.+Jiang&amp;author=B.+L.+Fridley&amp;author=Q.+Feng&amp;author=R.+Abo&amp;author=A.+Brisbin&amp;volume=4&amp;issue=166&amp;publication_year=2013&amp;pmid=24009623&amp;doi=10.3389/fgene.2013.00166&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B8"><span class="mixed-citation">Jin J., Wang J.-X., Chen F.-F., Wu D.-P., Hu J., Zhou J.-F., et al.. (2013). <span class="ref-title">Homoharringtonine-based induction regimens for patients with de-novo acute myeloid leukaemia: a multicentre, open-label, randomised, controlled phase 3 trial</span>. <span class="ref-journal">Lancet Oncol</span>. <span class="ref-vol">14</span>, 599&#x02013;608. 10.1016/S1470-2045(13)70152-9 [<a href="https://pubmed.ncbi.nlm.nih.gov/23664707" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS1470-2045(13)70152-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Oncol&amp;title=Homoharringtonine-based+induction+regimens+for+patients+with+de-novo+acute+myeloid+leukaemia:+a+multicentre,+open-label,+randomised,+controlled+phase+3+trial&amp;author=J.+Jin&amp;author=J.-X.+Wang&amp;author=F.-F.+Chen&amp;author=D.-P.+Wu&amp;author=J.+Hu&amp;volume=14&amp;publication_year=2013&amp;pages=599-608&amp;pmid=23664707&amp;doi=10.1016/S1470-2045(13)70152-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B9"><span class="mixed-citation">Kantarjian H. M., O'brien S., Cortes J. (2013). <span class="ref-title">Homoharringtonine/omacetaxine mepesuccinate: the long and winding road to food and drug administration approval</span>. <span class="ref-journal">Clin. Lymphoma Myeloma Leuk</span>. <span class="ref-vol">13</span>, 530&#x02013;533. 10.1016/j.clml.2013.03.017 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3775965/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23790799" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.clml.2013.03.017" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Lymphoma+Myeloma+Leuk&amp;title=Homoharringtonine/omacetaxine+mepesuccinate:+the+long+and+winding+road+to+food+and+drug+administration+approval&amp;author=H.+M.+Kantarjian&amp;author=S.+O'brien&amp;author=J.+Cortes&amp;volume=13&amp;publication_year=2013&amp;pages=530-533&amp;pmid=23790799&amp;doi=10.1016/j.clml.2013.03.017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B10"><span class="mixed-citation">Li L., Fridley B., Kalari K., Jenkins G., Batzler A., Safgren S., et al.. (2008). <span class="ref-title">Gemcitabine and cytosine arabinoside cytotoxicity: association with lymphoblastoid cell expression</span>. <span class="ref-journal">Cancer Res</span>. <span class="ref-vol">68</span>, 7050&#x02013;7058. 10.1158/0008-5472.CAN-08-0405 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2562356/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18757419" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F0008-5472.CAN-08-0405" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Gemcitabine+and+cytosine+arabinoside+cytotoxicity:+association+with+lymphoblastoid+cell+expression&amp;author=L.+Li&amp;author=B.+Fridley&amp;author=K.+Kalari&amp;author=G.+Jenkins&amp;author=A.+Batzler&amp;volume=68&amp;publication_year=2008&amp;pages=7050-7058&amp;pmid=18757419&amp;doi=10.1158/0008-5472.CAN-08-0405&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B11"><span class="mixed-citation">Li L., Fridley B. L., Kalari K., Jenkins G., Batzler A., Weinshilboum R. M., et al.. (2009). <span class="ref-title">Gemcitabine and arabinosylcytosin pharmacogenomics: genome-wide association and drug response biomarkers</span>. <span class="ref-journal">PLoS ONE</span>
<span class="ref-vol">4</span>:e7765. 10.1371/journal.pone.0007765 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2770319/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19898621" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0007765" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Gemcitabine+and+arabinosylcytosin+pharmacogenomics:+genome-wide+association+and+drug+response+biomarkers&amp;author=L.+Li&amp;author=B.+L.+Fridley&amp;author=K.+Kalari&amp;author=G.+Jenkins&amp;author=A.+Batzler&amp;volume=4&amp;publication_year=2009&amp;pages=e7765&amp;pmid=19898621&amp;doi=10.1371/journal.pone.0007765&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B12"><span class="mixed-citation">Mitra A. K., Crews K., Pounds S., Cao X., Downing J. R., Raimondi S., et al.. (2011). <span class="ref-title">Impact of genetic variation in FKBP5 on clinical response in pediatric acute myeloid leukemia patients: a pilot study</span>. <span class="ref-journal">Leukemia</span>
<span class="ref-vol">25</span>, 1354&#x02013;1356. 10.1038/leu.2011.74 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3238383/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21483441" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fleu.2011.74" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Impact+of+genetic+variation+in+FKBP5+on+clinical+response+in+pediatric+acute+myeloid+leukemia+patients:+a+pilot+study&amp;author=A.+K.+Mitra&amp;author=K.+Crews&amp;author=S.+Pounds&amp;author=X.+Cao&amp;author=J.+R.+Downing&amp;volume=25&amp;publication_year=2011&amp;pages=1354-1356&amp;pmid=21483441&amp;doi=10.1038/leu.2011.74&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B13"><span class="mixed-citation">Morley M., Molony C. M., Weber T. M., Devlin J. L., Ewens K. G., Spielman R. S., et al.. (2004). <span class="ref-title">Genetic analysis of genome-wide variation in human gene expression</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">430</span>, 743&#x02013;747. 10.1038/nature02797 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2966974/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15269782" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature02797" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+analysis+of+genome-wide+variation+in+human+gene+expression&amp;author=M.+Morley&amp;author=C.+M.+Molony&amp;author=T.+M.+Weber&amp;author=J.+L.+Devlin&amp;author=K.+G.+Ewens&amp;volume=430&amp;publication_year=2004&amp;pages=743-747&amp;pmid=15269782&amp;doi=10.1038/nature02797&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B14"><span class="mixed-citation">Niu N., Qin Y., Fridley B. L., Hou J., Kalari K. R., Zhu M., et al.. (2010). <span class="ref-title">Radiation pharmacogenomics: a genome-wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines</span>. <span class="ref-journal">Genome Res</span>. <span class="ref-vol">20</span>, 1482&#x02013;1492. 10.1101/gr.107672.110 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2963812/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20923822" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.107672.110" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Radiation+pharmacogenomics:+a+genome-wide+association+approach+to+identify+radiation+response+biomarkers+using+human+lymphoblastoid+cell+lines&amp;author=N.+Niu&amp;author=Y.+Qin&amp;author=B.+L.+Fridley&amp;author=J.+Hou&amp;author=K.+R.+Kalari&amp;volume=20&amp;publication_year=2010&amp;pages=1482-1492&amp;pmid=20923822&amp;doi=10.1101/gr.107672.110&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B15"><span class="mixed-citation">Park S., Chapuis N., Tamburini J., Bardet V., Cornillet-Lefebvre P., Willems L., et al.. (2010). <span class="ref-title">Role of the PI3K/AKT and mTOR signaling pathways in acute myeloid leukemia</span>. <span class="ref-journal">Haematologica</span>
<span class="ref-vol">95</span>, 819&#x02013;828. 10.3324/haematol.2009.013797 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2864389/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19951971" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3324%2Fhaematol.2009.013797" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Haematologica&amp;title=Role+of+the+PI3K/AKT+and+mTOR+signaling+pathways+in+acute+myeloid+leukemia&amp;author=S.+Park&amp;author=N.+Chapuis&amp;author=J.+Tamburini&amp;author=V.+Bardet&amp;author=P.+Cornillet-Lefebvre&amp;volume=95&amp;publication_year=2010&amp;pages=819-828&amp;pmid=19951971&amp;doi=10.3324/haematol.2009.013797&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B16"><span class="mixed-citation">Pei H., Li L., Fridley B. L., Jenkins G. D., Kalari K. R., Lingle W., et al.. (2009). <span class="ref-title">FKBP51 affects cancer cell response to chemotherapy by negatively regulating Akt</span>. <span class="ref-journal">Cancer Cell</span>
<span class="ref-vol">16</span>, 259&#x02013;266. 10.1016/j.ccr.2009.07.016 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2755578/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19732725" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ccr.2009.07.016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=FKBP51+affects+cancer+cell+response+to+chemotherapy+by+negatively+regulating+Akt&amp;author=H.+Pei&amp;author=L.+Li&amp;author=B.+L.+Fridley&amp;author=G.+D.+Jenkins&amp;author=K.+R.+Kalari&amp;volume=16&amp;publication_year=2009&amp;pages=259-266&amp;pmid=19732725&amp;doi=10.1016/j.ccr.2009.07.016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B17"><span class="mixed-citation">Peters E. J., Kraja A. T., Lin S. J., Yen-Revollo J. L., Marsh S., Province M. A., et al.. (2009). <span class="ref-title">Association of thymidylate synthase variants with 5-fluorouracil cytotoxicity</span>. <span class="ref-journal">Pharmacogenet. Genomics</span>
<span class="ref-vol">19</span>, 399&#x02013;401. 10.1097/FPC.0b013e328329fdec [<a href="https://pubmed.ncbi.nlm.nih.gov/19339911" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1097%2FFPC.0b013e328329fdec" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.+Genomics&amp;title=Association+of+thymidylate+synthase+variants+with+5-fluorouracil+cytotoxicity&amp;author=E.+J.+Peters&amp;author=A.+T.+Kraja&amp;author=S.+J.+Lin&amp;author=J.+L.+Yen-Revollo&amp;author=S.+Marsh&amp;volume=19&amp;publication_year=2009&amp;pages=399-401&amp;pmid=19339911&amp;doi=10.1097/FPC.0b013e328329fdec&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B18"><span class="mixed-citation">Price A. L., Patterson N. J., Plenge R. M., Weinblatt M. E., Shadick N. A., Reich D. (2006). <span class="ref-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <span class="ref-journal">Nat. Genet</span>. <span class="ref-vol">38</span>, 904&#x02013;909. 10.1038/ng1847 [<a href="https://pubmed.ncbi.nlm.nih.gov/16862161" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fng1847" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.+L.+Price&amp;author=N.+J.+Patterson&amp;author=R.+M.+Plenge&amp;author=M.+E.+Weinblatt&amp;author=N.+A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;doi=10.1038/ng1847&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B19"><span class="mixed-citation">Sabeti P. C., Varilly P., Fry B., Lohmueller J., Hostetter E., Cotsapas C., et al.. (2007). <span class="ref-title">Genome-wide detection and characterization of positive selection in human populations</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">449</span>, 913&#x02013;918. 10.1038/nature06250 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687721/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17943131" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature06250" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+detection+and+characterization+of+positive+selection+in+human+populations&amp;author=P.+C.+Sabeti&amp;author=P.+Varilly&amp;author=B.+Fry&amp;author=J.+Lohmueller&amp;author=E.+Hostetter&amp;volume=449&amp;publication_year=2007&amp;pages=913-918&amp;pmid=17943131&amp;doi=10.1038/nature06250&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B20"><span class="mixed-citation">Sachidanandam R., Weissman D., Schmidt S. C., Kakol J. M., Stein L. D., Marth G., et al.. (2001). <span class="ref-title">A map of human genome sequence variation containing 1.42 million single nucleotide polymorphisms</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">409</span>, 928&#x02013;933. 10.1038/35057149 [<a href="https://pubmed.ncbi.nlm.nih.gov/11237013" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F35057149" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+sequence+variation+containing+1.42+million+single+nucleotide+polymorphisms&amp;author=R.+Sachidanandam&amp;author=D.+Weissman&amp;author=S.+C.+Schmidt&amp;author=J.+M.+Kakol&amp;author=L.+D.+Stein&amp;volume=409&amp;publication_year=2001&amp;pages=928-933&amp;pmid=11237013&amp;doi=10.1038/35057149&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B21"><span class="mixed-citation">Sie L., Loong S., Tan E. K. (2009). <span class="ref-title">Utility of lymphoblastoid cell lines</span>. <span class="ref-journal">J. Neurosc. Res</span>. <span class="ref-vol">87</span>, 1953&#x02013;1959. 10.1002/jnr.22000 [<a href="https://pubmed.ncbi.nlm.nih.gov/19224581" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjnr.22000" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosc.+Res&amp;title=Utility+of+lymphoblastoid+cell+lines&amp;author=L.+Sie&amp;author=S.+Loong&amp;author=E.+K.+Tan&amp;volume=87&amp;publication_year=2009&amp;pages=1953-1959&amp;pmid=19224581&amp;doi=10.1002/jnr.22000&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B22"><span class="mixed-citation">Storey J. D. (2003). <span class="ref-title">The positive false discovery rate: a bayesian interpretation and the q-value</span>. <span class="ref-journal">Ann. Stat</span>. <span class="ref-vol">31</span>, 2013&#x02013;2015
 10.1214/aos/1074290335 [<a href="//doi.org/10.1214%2Faos%2F1074290335" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Stat&amp;title=The+positive+false+discovery+rate:+a+bayesian+interpretation+and+the+q-value&amp;author=J.+D.+Storey&amp;volume=31&amp;publication_year=2003&amp;pages=2013-2015&amp;doi=10.1214/aos/1074290335&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B23"><span class="mixed-citation">Tang R., Faussat A.-M., Majdak P., Marzac C., Dubrulle S., Marjanovic Z., et al.. (2006). <span class="ref-title">Semisynthetic homoharringtonine induces apoptosis via inhibition of protein synthesis and triggers rapid myeloid cell leukemia-1 down-regulation in myeloid leukemia cells</span>. <span class="ref-journal">Mol. Cancer Ther</span>. <span class="ref-vol">5</span>, 723&#x02013;731. 10.1158/1535-7163.MCT-05-0164 [<a href="https://pubmed.ncbi.nlm.nih.gov/16546987" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1158%2F1535-7163.MCT-05-0164" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cancer+Ther&amp;title=Semisynthetic+homoharringtonine+induces+apoptosis+via+inhibition+of+protein+synthesis+and+triggers+rapid+myeloid+cell+leukemia-1+down-regulation+in+myeloid+leukemia+cells&amp;author=R.+Tang&amp;author=A.-M.+Faussat&amp;author=P.+Majdak&amp;author=C.+Marzac&amp;author=S.+Dubrulle&amp;volume=5&amp;publication_year=2006&amp;pages=723-731&amp;pmid=16546987&amp;doi=10.1158/1535-7163.MCT-05-0164&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B24"><span class="mixed-citation">Trengove M. C., Ward A. C. (2013). <span class="ref-title">SOCS proteins in development and disease</span>. <span class="ref-journal">Am. J. Clin. Exp. Immunol</span>. <span class="ref-vol">2</span>, 1&#x02013;29.  <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3714205/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23885323" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Clin.+Exp.+Immunol&amp;title=SOCS+proteins+in+development+and+disease&amp;author=M.+C.+Trengove&amp;author=A.+C.+Ward&amp;volume=2&amp;publication_year=2013&amp;pages=1-29&amp;pmid=23885323&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B25"><span class="mixed-citation">Wang L., McLeod H. L., Weinshilboum R. M. (2011). <span class="ref-title">Genomics and drug response</span>. <span class="ref-journal">New. Engl. J. Med</span>. <span class="ref-vol">364</span>, 1144&#x02013;1153. 10.1056/NEJMra1010600 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3184612/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21428770" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1056%2FNEJMra1010600" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=New.+Engl.+J.+Med&amp;title=Genomics+and+drug+response&amp;author=L.+Wang&amp;author=H.+L.+McLeod&amp;author=R.+M.+Weinshilboum&amp;volume=364&amp;publication_year=2011&amp;pages=1144-1153&amp;pmid=21428770&amp;doi=10.1056/NEJMra1010600&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B26"><span class="mixed-citation">Watters J. W., Kraja A., Meucci M. A., Province M. A., McLeod H. L. (2004). <span class="ref-title">Genome-wide discovery of loci influencing chemotherapy cytotoxicity</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A</span>. <span class="ref-vol">101</span>, 11809&#x02013;11814. 10.1073/pnas.0404580101 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC511056/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15282376" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.0404580101" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A&amp;title=Genome-wide+discovery+of+loci+influencing+chemotherapy+cytotoxicity&amp;author=J.+W.+Watters&amp;author=A.+Kraja&amp;author=M.+A.+Meucci&amp;author=M.+A.+Province&amp;author=H.+L.+McLeod&amp;volume=101&amp;publication_year=2004&amp;pages=11809-11814&amp;pmid=15282376&amp;doi=10.1073/pnas.0404580101&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B27"><span class="mixed-citation">Wu Z., Irizarry R. A., Gentleman R., Nartinez-Murillo F., Spencer F. (2004). <span class="ref-title">A model-based background adjustment for oligonucleotide expression arrays</span>. <span class="ref-journal">J. Am. Stat. Assoc</span>. <span class="ref-vol">99</span>, 909&#x02013;917
 10.1198/016214504000000683 [<a href="//doi.org/10.1198%2F016214504000000683" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Stat.+Assoc&amp;title=A+model-based+background+adjustment+for+oligonucleotide+expression+arrays&amp;author=Z.+Wu&amp;author=R.+A.+Irizarry&amp;author=R.+Gentleman&amp;author=F.+Nartinez-Murillo&amp;author=F.+Spencer&amp;volume=99&amp;publication_year=2004&amp;pages=909-917&amp;doi=10.1198/016214504000000683&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B28"><span class="mixed-citation">Yinjun L., Jie J., Weilai X., Xiangming T. (2004). <span class="ref-title">Homoharringtonine mediates myeloid cell apoptosis via upregulation of pro-apoptotic bax and inducing caspase-3-mediated cleavage of poly(ADP-ribose) polymerase (PARP)</span>. <span class="ref-journal">Am. J. Hemat</span>. <span class="ref-vol">76</span>, 199&#x02013;204. 10.1002/ajh.20100 [<a href="https://pubmed.ncbi.nlm.nih.gov/15224352" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fajh.20100" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hemat&amp;title=Homoharringtonine+mediates+myeloid+cell+apoptosis+via+upregulation+of+pro-apoptotic+bax+and+inducing+caspase-3-mediated+cleavage+of+poly(ADP-ribose)+polymerase+(PARP)&amp;author=L.+Yinjun&amp;author=J.+Jie&amp;author=X.+Weilai&amp;author=T.+Xiangming&amp;volume=76&amp;publication_year=2004&amp;pages=199-204&amp;pmid=15224352&amp;doi=10.1002/ajh.20100&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B29"><span class="mixed-citation">Zhou D. C., Zittoun R., Marie J. P. (1995). <span class="ref-title">Homoharringtonine: an effective new natural product in cancer chemotherapy</span>. <span class="ref-journal">Bull. Cancer</span>
<span class="ref-vol">82</span>, 987&#x02013;995.  [<a href="https://pubmed.ncbi.nlm.nih.gov/8745664" ref="reftype=pubmed&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bull.+Cancer&amp;title=Homoharringtonine:+an+effective+new+natural+product+in+cancer+chemotherapy&amp;author=D.+C.+Zhou&amp;author=R.+Zittoun&amp;author=J.+P.+Marie&amp;volume=82&amp;publication_year=1995&amp;pages=987-995&amp;pmid=8745664&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=4292778&amp;issue-id=232658&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Frontiers in Genetics</span> are provided here courtesy of <strong>Frontiers Media SA</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC4292778/pdf/fgene-05-00465.pdf" class="int-view">PDF (4.2M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/4292778/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/4292778/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="4292778" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4292778%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="gatFRLvbEzFgB3Lt1Q1NEJIqEy9vMzy8GpJ8HTgaViD0gwdlUp7AUlyTtlGScut9">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4292778%2F&amp;text=Identification%20of%20genetic%20variants%20or%20genes%20that%20are%20associated%20with%20Homoharringtonine%20(HHT)%20response%20through%20a%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4292778%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4292778/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC4292778/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/25628645/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC4292778/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/25628645/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC4292778/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/4292778/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC4292778/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
