
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.e22e79d5a578.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.f012d399c229.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.4455260d4345.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Genome-Wide Gene-Environment Interaction Analysis Using Set-Based Association Tests - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939B9567B1550155000022759E6ADCF7.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.6.0.post2+24fead6" />

<meta name="ncbi_domain" content="frontgene" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC6339974/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6339974/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Frontiers in Genetics" /><meta name="citation_title" content="Genome-Wide Gene-Environment Interaction Analysis Using Set-Based Association Tests" /><meta name="citation_author" content="Wan-Yu Lin" /><meta name="citation_author_institution" content="1Institute of Epidemiology and Preventive Medicine, College of Public Health, National Taiwan University, Taipei, Taiwan" /><meta name="citation_author_institution" content="2Department of Public Health, College of Public Health, National Taiwan University, Taipei, Taiwan" /><meta name="citation_author" content="Ching-Chieh Huang" /><meta name="citation_author_institution" content="1Institute of Epidemiology and Preventive Medicine, College of Public Health, National Taiwan University, Taipei, Taiwan" /><meta name="citation_author" content="Yu-Li Liu" /><meta name="citation_author_institution" content="3Center for Neuropsychiatric Research, National Health Research Institutes, Zhunan, Taiwan" /><meta name="citation_author" content="Shih-Jen Tsai" /><meta name="citation_author_institution" content="4Department of Psychiatry, Taipei Veterans General Hospital, Taipei, Taiwan" /><meta name="citation_author_institution" content="5Division of Psychiatry, National Yang-Ming University, Taipei, Taiwan" /><meta name="citation_author" content="Po-Hsiu Kuo" /><meta name="citation_author_institution" content="1Institute of Epidemiology and Preventive Medicine, College of Public Health, National Taiwan University, Taipei, Taiwan" /><meta name="citation_author_institution" content="2Department of Public Health, College of Public Health, National Taiwan University, Taipei, Taiwan" /><meta name="citation_publication_date" content="2018" /><meta name="citation_volume" content="9" /><meta name="citation_doi" content="10.3389/fgene.2018.00715" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC6339974/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC6339974/" /><meta name="citation_pmid" content="30693016" /><meta name="DC.Title" content="Genome-Wide Gene-Environment Interaction Analysis Using Set-Based Association Tests" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Frontiers Media SA" /><meta name="DC.Contributor" content="Wan-Yu Lin" /><meta name="DC.Contributor" content="Ching-Chieh Huang" /><meta name="DC.Contributor" content="Yu-Li Liu" /><meta name="DC.Contributor" content="Shih-Jen Tsai" /><meta name="DC.Contributor" content="Po-Hsiu Kuo" /><meta name="DC.Date" content="2018" /><meta name="DC.Identifier" content="10.3389/fgene.2018.00715" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Genome-Wide Gene-Environment Interaction Analysis Using Set-Based Association Tests" /><meta property="og:type" content="article" /><meta property="og:description" content="The identification of gene-environment interactions (G &#x000d7; E) may eventually guide health-related choices and medical interventions for complex diseases. More powerful methods must be developed to identify G &#x000d7; E. The &#x0201c;adaptive combination ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6339974/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it�s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you�re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Genet%22[journal]" class="navlink">Front Genet</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC6339974
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC6339974/pdf/fgene-09-00715.pdf" class="int-view">PDF (3.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/6339974/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/6339974/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="6339974" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6339974%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="h6PLiFJlF9RMAOKwzjLQJ6JKSiaQ47MmOjwLw96VwVkI90VXbqLhCaLLKpxCbYWC">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6339974%2F&amp;text=Genome-Wide%20Gene-Environment%20Interaction%20Analysis%20Using%20Set-Based%20Association%20Tests" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6339974%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6339974/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC6339974/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/30693016/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC6339974/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/30693016/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC6339974/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/6339974/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Front%20Genet%22[journal]" class="navlink">Front Genet</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC6339974
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-frontgene.gif" alt="Logo of frontgene" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.frontiersin.org/genetics" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Front Genet.</span> 2018; 9: 715. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2019 Jan 14. </span>  <span class="doi"><span>doi:�</span><a href="//doi.org/10.3389%2Ffgene.2018.00715" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3389/fgene.2018.00715</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC6339974</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/30693016">30693016</a></div></div></div><h1 class="content-title">Genome-Wide Gene-Environment Interaction Analysis Using Set-Based Association Tests</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lin%20WY%5BAuthor%5D" class="affpopup" co-rid="_co_idm139691542036448" co-class="co-affbox">Wan-Yu Lin</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huang%20CC%5BAuthor%5D" class="affpopup" co-rid="_co_idm139691542033584" co-class="co-affbox">Ching-Chieh Huang</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20YL%5BAuthor%5D" class="affpopup" co-rid="_co_idm139691535680320" co-class="co-affbox">Yu-Li Liu</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tsai%20SJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm139691578980224" co-class="co-affbox">Shih-Jen Tsai</a>,<sup>4,</sup><sup>5</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kuo%20PH%5BAuthor%5D" class="affpopup" co-rid="_co_idm139691545719856" co-class="co-affbox">Po-Hsiu Kuo</a><sup>1,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139691542036448"><h3 class="no_margin">Wan-Yu Lin</h3><p><sup>1</sup>Institute of Epidemiology and Preventive Medicine, College of Public Health, National Taiwan University, Taipei, Taiwan</p><p><sup>2</sup>Department of Public Health, College of Public Health, National Taiwan University, Taipei, Taiwan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lin%20WY%5BAuthor%5D">Wan-Yu Lin</a></div></div><div id="_co_idm139691542033584"><h3 class="no_margin">Ching-Chieh Huang</h3><p><sup>1</sup>Institute of Epidemiology and Preventive Medicine, College of Public Health, National Taiwan University, Taipei, Taiwan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Huang%20CC%5BAuthor%5D">Ching-Chieh Huang</a></div></div><div id="_co_idm139691535680320"><h3 class="no_margin">Yu-Li Liu</h3><p><sup>3</sup>Center for Neuropsychiatric Research, National Health Research Institutes, Zhunan, Taiwan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20YL%5BAuthor%5D">Yu-Li Liu</a></div></div><div id="_co_idm139691578980224"><h3 class="no_margin">Shih-Jen Tsai</h3><p><sup>4</sup>Department of Psychiatry, Taipei Veterans General Hospital, Taipei, Taiwan</p><p><sup>5</sup>Division of Psychiatry, National Yang-Ming University, Taipei, Taiwan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tsai%20SJ%5BAuthor%5D">Shih-Jen Tsai</a></div></div><div id="_co_idm139691545719856"><h3 class="no_margin">Po-Hsiu Kuo</h3><p><sup>1</sup>Institute of Epidemiology and Preventive Medicine, College of Public Health, National Taiwan University, Taipei, Taiwan</p><p><sup>2</sup>Department of Public Health, College of Public Health, National Taiwan University, Taipei, Taiwan</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kuo%20PH%5BAuthor%5D">Po-Hsiu Kuo</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139691541415632_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139691541415632_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139691541415632_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139691541415632_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>1</sup>Institute of Epidemiology and Preventive Medicine, College of Public Health, National Taiwan University, Taipei, Taiwan</div><div class="fm-affl" id="aff2"><sup>2</sup>Department of Public Health, College of Public Health, National Taiwan University, Taipei, Taiwan</div><div class="fm-affl" id="aff3"><sup>3</sup>Center for Neuropsychiatric Research, National Health Research Institutes, Zhunan, Taiwan</div><div class="fm-affl" id="aff4"><sup>4</sup>Department of Psychiatry, Taipei Veterans General Hospital, Taipei, Taiwan</div><div class="fm-affl" id="aff5"><sup>5</sup>Division of Psychiatry, National Yang-Ming University, Taipei, Taiwan</div><div id="fn-a.v.b.j.a">Edited by: Mogens Fenger, Copenhagen University Hospital Hvidovre, Denmark</div><div id="fn-a.v.b.j.b">Reviewed by: Tao Wang, Medical College of Wisconsin, United States; Zhaoxia Yu, University of California, Irvine, United States</div><div id="c001">*Correspondence: Wan-Yu Lin <a href="mailto:dev@null" data-email="wt.ude.utn@ywnil" class="oemail">wt.ude.utn@ywnil</a></div><div id="fn001">This article was submitted to Statistical Genetics and Methodology, a section of the journal Frontiers in Genetics</div></div><div class="fm-article-notes hide half_rhythm" id="idm139691541415632_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2018 Sep 22; Accepted 2018 Dec 20.</div></div><div class="permissions half_rhythm hide" id="idm139691541415632_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2019 Lin, Huang, Liu, Tsai and Kuo.</div><div class="license half_rhythm">This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC6339974/bin/Data_Sheet_1.PDF" data-ga-action="click_feat_suppl">Data_Sheet_1.PDF</a><span style="color:gray"> (323K)</span></div><div class="small guid">GUID:�F53E1D27-558F-4A2D-9DA1-F8D8394CC7D7</div></div></dd></dl></div><div id="abstract-a.v.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.v.b.qtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">The identification of gene-environment interactions (G &#x000d7; E) may eventually guide health-related choices and medical interventions for complex diseases. More powerful methods must be developed to identify G &#x000d7; E. The &#x0201c;adaptive combination of Bayes factors method&#x0201d; (ADABF) has been proposed as a powerful genome-wide polygenic approach to detect G &#x000d7; E. In this work, we evaluate its performance when serving as a gene-based G &#x000d7; E test. We compare ADABF with six tests including the &#x0201c;Set-Based gene-EnviRonment InterAction test&#x0201d; (SBERIA), &#x0201c;gene-environment set association test&#x0201d; (GESAT), etc. With extensive simulations, SBERIA and ADABF are found to be more powerful than other G &#x000d7; E tests. However, SBERIA suffers from a power loss when 50% SNP main effects are in the same direction with the SNP &#x000d7; E interaction effects while 50% are in the opposite direction. We further applied these seven G &#x000d7; E methods to the Taiwan Biobank data to explore gene&#x000d7; alcohol interactions on blood pressure levels. The <em>ADAMTS7P1</em> gene at chromosome 15q25.2 was detected to interact with alcohol consumption on diastolic blood pressure (<em>p</em> = 9.5 &#x000d7; 10<sup>&#x02212;7</sup>, according to the GESAT test). At this gene, the <em>P</em>-values provided by other six tests all reached the suggestive significance level (<em>p</em> &#x0003c; 5 &#x000d7; 10<sup>&#x02212;5</sup>). Regarding the computation time required for a genome-wide G &#x000d7; E analysis, SBERIA is the fastest method, followed by ADABF. Considering the validity, power performance, robustness, and computation time, ADABF is recommended for genome-wide G &#x000d7; E analyses.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">diastolic blood pressure, systolic blood pressure, hypertension, gene-alcohol interaction, Taiwan Biobank, multiple testing correction</span></div></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title">Introduction</h2><p class="p p-first">&#x0201c;Gene-environment interaction&#x0201d; (G &#x000d7; E) is defined as &#x0201c;a different effect of an environmental exposure on disease risk in subjects with different genotypes&#x0201d; or &#x0201c;a different effect of a genotype on disease risk in subjects with different environmental exposures&#x0201d; (Ottman, <a href="#B30" rid="B30" class=" bibr popnode">1996</a>). &#x0201c;Gene-treatment interactions&#x0201d; are specific examples of G &#x000d7; E in pharmacogenomics. Searching for genes that may modify drug responses will significantly improve drug delivery by identifying subjects that can benefit from therapy and those at an increased risk of harm (He and Allen, <a href="#B9" rid="B9" class=" bibr popnode">2010</a>; Chen et al., <a href="#B4" rid="B4" class=" bibr popnode">2011</a>; Ko et al., <a href="#B16" rid="B16" class=" bibr popnode">2015</a>). The identification of G &#x000d7; E and gene-treatment interactions may eventually guide health-related choices and medical interventions for complex diseases (Franks and Pare, <a href="#B8" rid="B8" class=" bibr popnode">2016</a>). Clearly, more powerful methods must be developed to detect G &#x000d7; E (Hunter, <a href="#B10" rid="B10" class=" bibr popnode">2005</a>; Zhang and Biswas, <a href="#B44" rid="B44" class=" bibr popnode">2015</a>).</p><p>Exploring G &#x000d7; E is important for disease prevention. However, compared with the success achieved in identifying genetic main effects, very few G &#x000d7; E findings have been replicated partially due to the lack of power (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>). Single-nucleotide polymorphism (SNP) analysis (one SNP at a time) is a commonly used approach. Nonetheless, this approach suffers from a power loss due to a harsh penalty of multiple testing. Even true positives may not stand out under the stringent genome-wide significance level (Lin and Lee, <a href="#B20" rid="B20" class=" bibr popnode">2010</a>), i.e., 5 &#x000d7; 10<sup>&#x02212;8</sup>.</p><p>Several set-based (or gene-based) analysis methods have been developed to aggregate the G &#x000d7; E signals within a gene/region and alleviate the multiple-testing penalty (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>; Lin et al., <a href="#B21" rid="B21" class=" bibr popnode">2013</a>, <a href="#B22" rid="B22" class=" bibr popnode">2016</a>; Chen et al., <a href="#B3" rid="B3" class=" bibr popnode">2014</a>). Jiao et al. proposed a two-stage &#x0201c;Set-Based gene-EnviRonment InterAction test&#x0201d; for case-control studies, called &#x0201c;SBERIA&#x0201d; (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>). During the first stage, the SNPs are filtered according to their associations with E (this is Step 1 in Murcray et al., <a href="#B29" rid="B29" class=" bibr popnode">2009</a>). The sign and significance of the filtering statistics are then used to weight SNP &#x000d7; E in the second stage (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>).</p><p>Lin et al. proposed the &#x0201c;gene-environment set association test&#x0201d; (GESAT), which is a variance component (VC) test that estimates the SNP main effects using a ridge regression (Lin et al., <a href="#B21" rid="B21" class=" bibr popnode">2013</a>). These authors later developed an &#x0201c;interaction sequence kernel association test&#x0201d; (iSKAT) that is regarded as the optimal in the class of VC tests (Lin et al., <a href="#B22" rid="B22" class=" bibr popnode">2016</a>). While GESAT assumes no correlation among the SNP &#x000d7; E interaction effects, iSKAT searches for the optimal correlation coefficient among them. Therefore, GESAT is a specific case of iSKAT, and both approaches can be implemented using the &#x0201c;iSKAT&#x0201d; R package.</p><p>Chen et al. proposed a G &#x000d7; E test that treats the SNP main effects as fixed (designated &#x0201c;INT_FIX&#x0201d;) or random (designated &#x0201c;INT_RAN&#x0201d;). They also developed a joint test for detecting the genetic associations while allowing for G &#x000d7; E (designated &#x0201c;JOINT&#x0201d;) (Chen et al., <a href="#B3" rid="B3" class=" bibr popnode">2014</a>). These three methods belong to the class of VC tests and can be performed using the &#x0201c;rareGE&#x0201d; R package.</p><p class="p p-last">The abovementioned methods have been proposed with user-friendly analysis tools that are popular choices for G &#x000d7; E analyses. Recently, the &#x0201c;adaptive combination of Bayes factors method&#x0201d; (ADABF) has been proposed as a powerful polygenic approach to detect G &#x000d7; E (Lin et al., <a href="#B19" rid="B19" class=" bibr popnode">2018</a>). This method can also serve as a gene-based G &#x000d7; E test. In this study, we evaluate the performance of ADABF when detecting gene-based G &#x000d7; E signals. We compare ADABF with the abovementioned six tests. Using a sample of 16,555 subjects from the Taiwan Biobank (TWB) data, we perform a genome-wide gene-alcohol interaction analysis on diastolic blood pressure (DBP) and systolic blood pressure (SBP). The validity, power, robustness, and computation time of the seven G &#x000d7; E set-based tests are investigated through simulations or real data analyses.</p></div><div id="s2" class="tsec sec"><h2 class="head no_bottom_margin" id="s2title">Materials and Methods</h2><div id="sec-a.w.b.b" class="sec sec-first"><h3 id="sec-a.w.b.btitle">Adaptive Combination of Bayes Factors Method</h3><p class="p p-first">Suppose a gene or an analysis region contains <em>L</em> SNPs. Let <em>Y</em> be the phenotype, <em>g</em>[&#x000b7;] be the link function, <em>G</em><sub><em>l</em></sub> be the number of minor allele (0, 1, or 2) at the <em>l</em><sup>th</sup> SNP (<em>l</em> = 1, &#x02026;, <em>L</em>), <em>E</em> be the environmental factor, and <em><strong>X</strong></em> be the vector of potential confounder covariates. First, we assess each SNP &#x000d7; E interaction by considering the following generalized linear model (GLM):</p><div class="disp-formula" id="E1"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mi>g</mi><mrow><mo>[</mo><mrow><mi>E</mi><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>]</mo></mrow><mo>=</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>E</mi></mrow></msub><mi>E</mi><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mi>E</mi><mo>+</mo><msub><mrow><msup><mrow><mstyle mathvariant="bold-italic"><mi>&#x003b2;</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mi>X</mi></mrow></msub><mstyle mathvariant="bold"><mi>X</mi></mstyle><mo>,</mo><mtext>&#x000a0;&#x000a0;&#x000a0;&#x000a0;&#x000a0;</mtext><mi>l</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>&#x022ef;</mo><mspace width="0.3em" class="thinspace"></mspace><mo>,</mo><mi>L</mi><mo>.</mo></mtd></mtr></mtable></math></div><div class="l">(1)</div></div><p>For simplicity, we omit the subscript &#x0201c;<em>i</em>&#x0201d; that represents the data of the <em>i</em><sup>th</sup> subject. The SNP &#x000d7; E interaction is of interest, and therefore <em>H</em><sub>0</sub>:&#x003b2;<sub><em>GE</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b2;<sub><em>GE</em></sub> &#x02260; 0. Let <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b2;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub></math> be the maximum likelihood estimate (MLE) of &#x003b2;<sub><em>GE</em></sub>. According to the asymptotic normality of MLE, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b2;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub></math> follows a normal distribution with a mean of &#x003b2;<sub><em>GE</em></sub> and a variance of <em>V</em>, i.e., <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b2;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><mo>~</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><mo>,</mo><mi>V</mi></mrow><mo stretchy="false">)</mo></mrow></math>.</p><p>We assume that the true interaction effects follow a normal distribution with a mean of 0 and a variance of <em>W</em>, i.e., &#x003b2;<sub><em>GE</em></sub> ~ <em>N</em>(0, <em>W</em>). The Bayes factor (BF) (Wakefield, <a href="#B38" rid="B38" class=" bibr popnode">2007</a>, <a href="#B39" rid="B39" class=" bibr popnode">2009</a>) of the SNP &#x000d7; E interaction is</p><div class="disp-formula" id="E2"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M5" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mi>B</mi><mi>F</mi><mo>=</mo><mfrac><mrow><mo class="qopname">Pr</mo><mrow><mo stretchy="false">(</mo><mrow><mi>D</mi><mi>a</mi><mi>t</mi><mi>a</mi><mo>|</mo><msub><mrow><mi>H</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow><mrow><mo class="qopname">Pr</mo><mrow><mo stretchy="false">(</mo><mrow><mi>D</mi><mi>a</mi><mi>t</mi><mi>a</mi><mo>|</mo><msub><mrow><mi>H</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mfrac><mo>=</mo><msqrt><mrow><mfrac><mrow><mover accent="true"><mrow><mi>V</mi></mrow><mo>^</mo></mover></mrow><mrow><mover accent="true"><mrow><mi>V</mi></mrow><mo>^</mo></mover><mo>+</mo><mi>W</mi></mrow></mfrac></mrow></msqrt><mo class="qopname">exp</mo><mrow><mo stretchy="true">(</mo><mrow><mfrac><mrow><msubsup><mrow><mover accent="true"><mrow><mi>&#x003b2;</mi></mrow><mo class="qopname">^</mo></mover></mrow><mrow><mi>G</mi><mi>E</mi></mrow><mrow><mn>2</mn></mrow></msubsup><mi>W</mi></mrow><mrow><mn>2</mn><mover accent="true"><mrow><mi>V</mi></mrow><mo class="qopname">^</mo></mover><mrow><mo stretchy="false">(</mo><mrow><mover accent="true"><mrow><mi>V</mi></mrow><mo class="qopname">^</mo></mover><mo>+</mo><mi>W</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mfrac></mrow><mo stretchy="true">)</mo></mrow><mo>,</mo></mtd></mtr></mtable></math></div><div class="l">(2)</div></div><p>where <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" overflow="scroll"><msubsup><mrow><mover accent="true"><mrow><mi>&#x003b2;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi><mi>E</mi></mrow><mrow><mo>&#x0200a;</mo></mrow></msubsup></math> is the MLE of &#x003b2;<sub><em>GE</em></sub>, and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M7" overflow="scroll"><mover accent="true"><mrow><mi>V</mi></mrow><mo>^</mo></mover></math> is the estimated variance of <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M8" overflow="scroll"><msubsup><mrow><mover accent="true"><mrow><mi>&#x003b2;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi><mi>E</mi></mrow><mrow><mo>&#x0200a;</mo></mrow></msubsup></math>. To propose a prior that can be applicable to most situations, we first scale the environmental factor E to range from 0 to 1. A dichotomous E will be coded as 0 or 1 whereas a continuous E will be first scaled to be <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M9" overflow="scroll"><msup><mrow><msub><mrow><mi>E</mi></mrow><mrow><mo>&#x0200a;</mo></mrow></msub></mrow><mrow><mi>&#x02032;</mi></mrow></msup><mo>=</mo><mrow><mo stretchy="false">(</mo><mrow><mi>E</mi><mo>-</mo><msub><mrow><mi>E</mi></mrow><mrow><mo class="qopname">min</mo></mrow></msub></mrow><mo stretchy="false">)</mo></mrow><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><msub><mrow><mi>E</mi></mrow><mrow><mo class="qopname">max</mo></mrow></msub><mo>-</mo><msub><mrow><mi>E</mi></mrow><mrow><mo class="qopname">min</mo></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></math>, in which <em>E</em><sub>min</sub> and <em>E</em><sub>max</sub> are the minimum and maximum of E, respectively. In this way, <em>G</em><sub><em>l</em></sub><em>E</em> in Equation (1) will be between 0 and 2, in the same range as <em>G</em><sub><em>l</em></sub>.</p><p>The Wellcome Trust Case Control Consortium GWAS (WTCCC, <a href="#B41" rid="B41" class=" bibr popnode">2007</a>) specified the prior for SNP main effects as &#x003b2;<sub><em>G</em></sub> ~ <em>N</em>(0, <em>W</em>), where <em>W</em> = 0.2<sup>2</sup> = 0.04. This prior implies that we believe 95% of odds ratios (ORs) range from exp(&#x02212;2 &#x000d7; 0.2) = 0.67 to exp(2 &#x000d7; 0.2) = 1.49. Now that <em>G</em><sub><em>l</em></sub><em>E</em> is in the same range as <em>G</em><sub><em>l</em></sub>, we consider using the same prior for SNP &#x000d7; E interaction, i.e., &#x003b2;<sub><em>GE</em></sub> ~ <em>N</em>(0, <em>W</em>) where <em>W</em> = 0.2<sup>2</sup> = 0.04. Reported SNP &#x000d7; E interactions have been of modest effect sizes that can be positive or negative (Simino et al., <a href="#B35" rid="B35" class=" bibr popnode">2013</a>; Rudolph et al., <a href="#B34" rid="B34" class=" bibr popnode">2016</a>; Sung et al., <a href="#B36" rid="B36" class=" bibr popnode">2018</a>), and therefore <em>N</em>(0, <em>W</em> = 0.04) may be a reasonable prior for &#x003b2;<sub><em>GE</em></sub> (Lin et al., <a href="#B19" rid="B19" class=" bibr popnode">2018</a>).</p><p>To apply ADABF to continuous traits, we should first standardize the traits to have a mean of 0 and a standard deviation of 1, as implemented in our ADABF R code that can be downloaded from <a href="http://homepage.ntu.edu.tw/~linwy/ADABFGE.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://homepage.ntu.edu.tw/~linwy/ADABFGE.html</a>. The prior of <em>N</em>(0, <em>W</em> = 0.04) implies that 95% of &#x003b2;<sub><em>G</em><sub><em>E</em></sub><sub><em>l</em></sub></sub>s range from (&#x02212;2 &#x000d7; 0.2) = &#x02212;0.4 to (2 &#x000d7; 0.2) = 0.4. This may also be a reasonable prior for &#x003b2;<sub><em>GE</em></sub> when traits are continuous.</p><p>Because SNP &#x000d7; E interaction effects reported by empirical studies have been modest (Simino et al., <a href="#B35" rid="B35" class=" bibr popnode">2013</a>; Rudolph et al., <a href="#B34" rid="B34" class=" bibr popnode">2016</a>; Sung et al., <a href="#B36" rid="B36" class=" bibr popnode">2018</a>), this prior variance (<em>W</em> = 0.2<sup>2</sup> = 0.04) may be slightly large for &#x003b2;<sub><em>G</em><sub><em>E</em></sub><sub><em>l</em></sub></sub>s. However, a larger prior variance can just reflect our uncertainty of the prior information (Wang et al., <a href="#B40" rid="B40" class=" bibr popnode">2009</a>).</p><p>After calculating the BFs of all the <em>L</em> SNP &#x000d7; E, we sort these <em>L</em> BFs from the largest to the smallest, and denote them as <em>BF</em><sub>(1)</sub> &#x02265; <em>BF</em><sub>(2)</sub> &#x02265; &#x022ef; &#x02265; <em>BF</em><sub>(<em>L</em>)</sub>. The leading <em>k</em> BFs are summarized by <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M10" overflow="scroll"><msub><mrow><mi>S</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>=</mo><msubsup><mrow><mo>&#x02211;</mo></mrow><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>k</mi></mrow></msubsup><mo class="qopname">log</mo><mrow><mo stretchy="false">(</mo><mrow><mi>B</mi><msub><mrow><mi>F</mi></mrow><mrow><mrow><mo stretchy="false">(</mo><mrow><mi>l</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></math>, where <em>k</em> = 1, &#x000b7;&#x000b7;&#x000b7;, <em>L</em>. Let <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M11" overflow="scroll"><mrow><msub><mstyle mathvariant="bold" mathsize="normal"><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></mstyle><mrow><mi>G</mi><mi>E</mi><mo>,</mo><msub><mi>H</mi><mn>0</mn></msub></mrow></msub></mrow></math> be the vector containing <em>L</em>
<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M12" overflow="scroll"><msubsup><mrow><mover accent="true"><mrow><mi>&#x003b2;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi><mi>E</mi></mrow><mrow><mo>&#x0200a;</mo></mrow></msubsup></math>s under the null hypothesis <em>H</em><sub>0</sub> (none of the <em>L</em> SNPs interact with E). We draw <em>B</em> sets of <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M13" overflow="scroll"><mrow><msub><mstyle mathvariant="bold" mathsize="normal"><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></mstyle><mrow><mi>G</mi><mi>E</mi><mo>,</mo><msub><mi>H</mi><mn>0</mn></msub></mrow></msub></mrow></math> from the multivariate normal distribution with a mean vector of <strong>0</strong> and a variance-covariance matrix incorporating the pairwise linkage disequilibrium (LD) among the <em>L</em> SNPs, and then calculate <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M14" overflow="scroll"><msubsup><mrow><mi>S</mi></mrow><mrow><mi>k</mi></mrow><mrow><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn></mrow><mo stretchy="false">)</mo></mrow></mrow></msubsup></math>, &#x02026;, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M15" overflow="scroll"><msubsup><mrow><mi>S</mi></mrow><mrow><mi>k</mi></mrow><mrow><mrow><mo stretchy="false">(</mo><mrow><mi>B</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></msubsup></math> accordingly. The details can be found from Lin et al. (<a href="#B19" rid="B19" class=" bibr popnode">2018</a>).</p><p>By comparing <em>S</em><sub><em>k</em></sub> with its counterparts from <em>H</em><sub>0</sub> (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M16" overflow="scroll"><msubsup><mrow><mi>S</mi></mrow><mrow><mi>k</mi></mrow><mrow><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn></mrow><mo stretchy="false">)</mo></mrow></mrow></msubsup></math>, &#x02026;, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M17" overflow="scroll"><msubsup><mrow><mi>S</mi></mrow><mrow><mi>k</mi></mrow><mrow><mrow><mo stretchy="false">(</mo><mrow><mi>B</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></msubsup></math>), we obtain the <em>P</em>-value regarding <em>S</em><sub><em>k</em></sub>, where <em>k</em> = 1, &#x000b7;&#x000b7;&#x000b7;, <em>L</em>. We then find the minimum <em>P</em>-values (across <em>k</em> = 1, &#x000b7;&#x000b7;&#x000b7;, <em>L</em>) for the observed sample and for each of the resampling replicates. By comparing these minimum <em>P</em>-values, we obtain the significance of G &#x000d7; E for the observed sample. The efficient sequential resampling procedure (Liu et al., <a href="#B24" rid="B24" class=" bibr popnode">2016</a>) is used to speed up ADABF, in which the minimum and maximum numbers of resampling were set at 10<sup>3</sup> and 10<sup>7</sup>, respectively. The resampling procedure is repeated until the <em>P</em> &#x0003e; 100/<em>B</em>, where <em>B</em> is the number of resampling.</p><p class="p p-last">Because the same prior variance <em>W</em> is used for the observed sample and for each of the resampling replicates, the performance of ADABF is robust to the selection of <em>W</em> (Lin et al., <a href="#B19" rid="B19" class=" bibr popnode">2018</a>). The R code of the ADABF method can be downloaded from <a href="http://homepage.ntu.edu.tw/~linwy/ADABFGE.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://homepage.ntu.edu.tw/~linwy/ADABFGE.html</a>. A Perl script is also provided to facilitate genome-wide analyses, <a href="http://homepage.ntu.edu.tw/~linwy/ADABFGEfromPLINK.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://homepage.ntu.edu.tw/~linwy/ADABFGEfromPLINK.html</a>.</p></div><div id="sec-a.w.b.c" class="sec"><h3 id="sec-a.w.b.ctitle">Set-Based Gene-EnviRonment InterAction Test (SBERIA)</h3><p class="p p-first">There are two steps in SBERIA (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>): the filtering stage and the G &#x000d7; E stage. For case-control studies, a commonly-used filtering stage is to regress E on each SNP and assess the association of each SNP with E, by fitting a logistic regression for binary E or a linear regression for continuous E (Murcray et al., <a href="#B29" rid="B29" class=" bibr popnode">2009</a>; Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>). This strategy is referred to as the &#x0201c;SNP-E association filtering.&#x0201d;</p><p>Suppose a positive interaction between SNP (coded as 0, 1, or 2) and E (coded as 0 or 1) is responsible for the susceptibility of a rare disease. Subjects with E = 1 and SNP = 2 will have an increased disease risk. If cases are ascertained, more E = 1 and SNP = 2 combinations will be observed in cases, representing that SNP and E will be positively associated in cases. Assuming SNP and E are approximately independent in controls, they will be also positively associated in the combined case-control data (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>). Similarly, if there is a negative interaction between SNP and E, they will be negatively associated in the combined case-control data. Therefore, for rare-disease studies with ascertained cases, the association between SNP and E in combined case-control samples can be an efficient filtering statistic for detecting SNP &#x000d7; E interaction (Murcray et al., <a href="#B28" rid="B28" class=" bibr popnode">2011</a>). Dai et al. (Proposition 3) has justified the validity of using this filtering stage in G &#x000d7; E studies (Dai et al., <a href="#B5" rid="B5" class=" bibr popnode">2012</a>).</p><p>In the subsequent G &#x000d7; E stage, the hypothesis of interest is <em>H</em><sub>0</sub>:&#x003b1;<sub><em>GE</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b1;<sub><em>GE</em></sub> &#x02260; 0 in the following GLM,</p><div class="disp-formula" id="E3"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M18" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mi>g</mi><mrow><mo>[</mo><mrow><mi>E</mi><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>]</mo></mrow><mo>=</mo><msub><mrow><mi>&#x003b1;</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><msup><mrow><mstyle mathvariant="bold"><mi>&#x003b1;</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mstyle mathvariant="bold"><mi>G</mi></mstyle></mrow></msub><mi>G</mi><mo>+</mo><msub><mrow><mi>&#x003b1;</mi></mrow><mrow><mi>E</mi></mrow></msub><mi>E</mi><mo>+</mo><msub><mrow><mi>&#x003b1;</mi></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><mi>E</mi><msup><mrow><mstyle mathvariant="bold"><mi>G</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup><mstyle mathvariant="bold"><mi>&#x00175;</mi></mstyle><mo>+</mo><msub><mrow><msup><mrow><mi>&#x003b1;</mi></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mi>X</mi></mrow></msub><mstyle mathvariant="bold"><mi>X</mi></mstyle><mo>,</mo></mtd></mtr></mtable></math></div><div class="l">(3)</div></div><p>where <em><strong>G</strong></em> is the vector of the numbers of minor allele (0, 1, or 2) at the <em>L</em> SNPs, and <strong>&#x00175;</strong> is the vector of weights given to the <em>L</em> SNPs. The weight is determined by the sign and significance of the filtering statistic. The weight given to a SNP is 1 if it is positively associated with E, &#x02212;1 if it is negatively associated with E, and is a very small value (e.g., 0.0001) if the SNP is not statistically associated with E (i.e., filtering test <em>P</em>-value &#x0003e; a pre-specified significance level, say, 0.10 in Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>). The SBERIA approach uses this weighting scheme because the SNP-E association test has been shown to be asymptotically independent of the SNP &#x000d7; E interaction test (Murcray et al., <a href="#B29" rid="B29" class=" bibr popnode">2009</a>; Dai et al., <a href="#B5" rid="B5" class=" bibr popnode">2012</a>) and is powerful for filtering (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>).</p><p>Another commonly-used screening strategy is the &#x0201c;main-effect filtering.&#x0201d; Each SNP is first screened by testing <em>H</em><sub>0</sub>:&#x003b3;<sub><em>G</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b3;<sub><em>G</em></sub> &#x02260; 0 in the following GLM:</p><div class="disp-formula" id="E4"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M19" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mi>g</mi><mrow><mo>[</mo><mrow><mi>E</mi><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>]</mo></mrow><mo>=</mo><msub><mrow><mi>&#x003b3;</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>&#x003b3;</mi></mrow><mrow><mi>G</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mo>+</mo><msub><mrow><msup><mrow><mstyle mathvariant="bold"><mi>&#x003b3;</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mi>X</mi></mrow></msub><mstyle mathvariant="bold-italic"><mi>X</mi></mstyle><mo>.</mo></mtd></mtr></mtable></math></div><div class="l">(4)</div></div><p class="p p-last">To preserve the type I error rates, the filtering statistic (stage 1) and the following interaction test statistic (stage 2) must be asymptotically independent under the null hypothesis. Dai et al. have proven the validity of using the &#x0201c;main-effect filtering&#x0201d; as the screening strategy (Dai et al., <a href="#B5" rid="B5" class=" bibr popnode">2012</a>). Each element in <strong>&#x00175;</strong> represents the weight given to a SNP, which is 1 if the SNP is positively associated with <em>Y</em>, &#x02212;1 if it is negatively associated with <em>Y</em>, and is a very small value (0.0001) if the SNP is not statistically associated with <em>Y</em> (i.e., <em>P</em>-value &#x0003e; a pre-specified significance level, say, 0.10 in Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>).</p></div><div id="sec-a.w.b.d" class="sec sec-last"><h3 id="sec-a.w.b.dtitle">Variance Component (VC) Test</h3><p class="p p-first">The class of VC tests include iSKAT (Lin et al., <a href="#B22" rid="B22" class=" bibr popnode">2016</a>), GESAT (Lin et al., <a href="#B21" rid="B21" class=" bibr popnode">2013</a>), INT_FIX, INT_RAN, and JOINT (Chen et al., <a href="#B3" rid="B3" class=" bibr popnode">2014</a>). VC tests are based on the following GLM:</p><div class="disp-formula" id="E5"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M20" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mi>g</mi><mrow><mo>[</mo><mrow><mi>E</mi><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>]</mo></mrow><mo>=</mo><msub><mrow><mi>&#x003b4;</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><msup><mrow><mstyle mathvariant="bold"><mi>&#x003b4;</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mi>G</mi></mrow></msub><mstyle mathvariant="bold-italic"><mi>G</mi></mstyle><mo>+</mo><msub><mrow><mi>&#x003b4;</mi></mrow><mrow><mi>E</mi></mrow></msub><mi>E</mi><mo>+</mo><msub><mrow><msup><mrow><mstyle mathvariant="bold"><mi>&#x003b4;</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><mstyle mathvariant="bold-italic"><mi>S</mi></mstyle><mo>+</mo><msub><mrow><msup><mrow><mstyle mathvariant="bold"><mi>&#x003b4;</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mi>X</mi></mrow></msub><mstyle mathvariant="bold-italic"><mi>X</mi></mstyle><mo>,</mo></mtd></mtr></mtable></math></div><div class="l">(5)</div></div><p>where <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M21" overflow="scroll"><mstyle mathvariant="bold-italic"><mi>S</mi></mstyle><mo>=</mo><msup><mrow><mrow><mo>[</mo><mrow><mtable style="text-align:axis;" equalrows="false" columnlines="none none none none none none none none none" equalcolumns="false" class="array"><mtr><mtd><mi>E</mi><msub><mrow><mi>G</mi></mrow><mrow><mn>1</mn></mrow></msub></mtd><mtd><mi>E</mi><msub><mrow><mi>G</mi></mrow><mrow><mn>2</mn></mrow></msub></mtd><mtd><mo>&#x022ef;</mo></mtd><mtd><mi>E</mi><msub><mrow><mi>G</mi></mrow><mrow><mi>L</mi></mrow></msub></mtd></mtr></mtable></mrow><mo>]</mo></mrow></mrow><mrow><mi>&#x02032;</mi></mrow></msup></math>. The vector <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M22" overflow="scroll"><msub><mrow><mstyle mathvariant="bold"><mi>&#x003b4;</mi></mstyle></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><mo>=</mo><msup><mrow><mrow><mo>[</mo><mrow><mtable style="text-align:axis;" equalrows="false" columnlines="none none none none none none none none none" equalcolumns="false" class="array"><mtr><mtd><msub><mrow><mi>&#x003b4;</mi></mrow><mrow><mi>G</mi><msub><mrow><mi>E</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></msub></mtd><mtd><msub><mrow><mi>&#x003b4;</mi></mrow><mrow><mi>G</mi><msub><mrow><mi>E</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></msub></mtd><mtd><mo>&#x022ef;</mo></mtd><mtd><msub><mrow><mi>&#x003b4;</mi></mrow><mrow><mi>G</mi><msub><mrow><mi>E</mi></mrow><mrow><mi>L</mi></mrow></msub></mrow></msub></mtd></mtr></mtable></mrow><mo>]</mo></mrow></mrow><mrow><mi>&#x02032;</mi></mrow></msup></math> contains the <em>L</em> SNP &#x000d7; E interaction effects. Assuming &#x003b4;<sub><em>G</em><sub><em>E</em></sub><sub><em>l</em></sub></sub>s (<em>l</em> = 1, &#x02026;, <em>L</em>) follow a distribution with a mean of 0 and a variance of &#x003c4;<sub>2</sub>. The null hypothesis <em>H</em><sub>0</sub>:<strong>&#x003b4;</strong><sub><em>GE</em></sub> = 0 is then reduced to <em>H</em><sub>0</sub>:&#x003c4;<sub>2</sub> = 0. The score statistic to test <em>H</em><sub>0</sub>:&#x003c4;<sub>2</sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003c4;<sub>2</sub> &#x0003e; 0 can be referred to Equation (6) in Lin et al. (<a href="#B22" rid="B22" class=" bibr popnode">2016</a>).</p><p>Similar among the five VC tests, &#x003b4;<sub><em>G</em><sub><em>E</em></sub><sub><em>l</em></sub></sub>s (<em>l</em> = 1, &#x02026;, <em>L</em>) are assumed to be random effects that follow a distribution. Therefore, testing <em>H</em><sub>0</sub>:<strong>&#x003b4;</strong><sub><em>GE</em></sub> = 0 can be reduced to testing <em>H</em><sub>0</sub>:&#x003c4;<sub>2</sub> = 0. However, these five VC tests are dissimilar in two aspects.</p><p>First, they take different approaches to estimate the SNP main effects, &#x003b4;<sub><em>G</em><sub><em>l</em></sub></sub>s (<em>l</em> = 1, &#x02026;, <em>L</em>). INT_FIX treats &#x003b4;<sub><em>G</em><sub><em>l</em></sub></sub>s as fixed effects, whereas INT_RAN assumes &#x003b4;<sub><em>G</em><sub><em>l</em></sub></sub>s follow a distribution with a mean of 0 and a variance of &#x003c4;<sub>1</sub>. GESAT and iSKAT use ridge regression to estimate &#x003b4;<sub><em>G</em><sub><em>l</em></sub></sub>s under the null hypothesis of <em>H</em><sub>0</sub>:<strong>&#x003b4;</strong><sub><em>GE</em></sub> = 0. JOINT simultaneously tests whether SNP main effects or G &#x000d7; E interaction effects exist, i.e., <em>H</em><sub>0</sub>:&#x003c4;<sub>1</sub> = &#x003c4;<sub>2</sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003c4;<sub>1</sub> &#x0003e; 0 or &#x003c4;<sub>2</sub> &#x0003e; 0. Therefore, it is not a pure test for detecting G &#x000d7; E.</p><p class="p p-last">Second, iSKAT allows an exchangeable correlation &#x003c1; among &#x003b4;<sub><em>G</em><sub><em>E</em></sub><sub><em>l</em></sub></sub>s (<em>l</em> = 1, &#x02026;, <em>L</em>) and searches for the optimal &#x003c1;. The other four VC tests all assume that &#x003b4;<sub><em>G</em><sub><em>E</em></sub><sub><em>l</em></sub></sub>s are independent to each other (i.e., &#x003c1; = 0).</p></div></div><div id="s3" class="tsec sec"><h2 class="head no_bottom_margin" id="s3title">Results</h2><p class="p p-first">To reflect the real LD structures of the human genome, we used GWAS data from the TWB as our simulation material. The TWB aims to build a research database that integrates the genomic profiles, lifestyle patterns, dietary habits, and environmental exposures of residents aged 30&#x02013;70 years in Taiwan (Chen et al., <a href="#B2" rid="B2" class=" bibr popnode">2016</a>). Community-based volunteers donated blood, took a physical examination, and completed a questionnaire with a face-to-face interview.</p><p>Most of these community-based volunteers were unrelated subjects. To exclude subjects with cryptic relatedness, we first estimated the genome-wide identity by descent (IBD) sharing coefficients among seemingly unrelated individuals from the whole-genome data. Using PLINK-1.9 (Purcell et al., <a href="#B33" rid="B33" class=" bibr popnode">2007</a>), we obtained the IBD scores for all pairs of subjects, i.e., PI-HAT = Pr(IBD = 2) + 0.5&#x000d7; Pr(IBD = 1). &#x0201c;PI-HAT&#x0201d; is a parameter used in PLINK to quantify pairwise IBD scores. Some GWAS excluded relatives within third-degree consanguinity, and therefore removed one person from a pair with PI-HAT &#x02265; 0.125 (Lowe et al., <a href="#B25" rid="B25" class=" bibr popnode">2009</a>; Mok et al., <a href="#B27" rid="B27" class=" bibr popnode">2014</a>). We here use a slightly more stringent threshold, 0.1. After removing subjects with cryptic relatedness (PI-HAT &#x0003e; 0.1), our analysis data included 16,555 unrelated subjects (8,213 males and 8,342 females).</p><p>The whole-genome genotyping of the TWB data revealed 631,941 autosomal SNPs. We excluded 22,212 SNPs with genotyping rates &#x0003c;95% and 5,988 SNPs with Hardy-Weinberg test <em>P</em> &#x0003c; 5.7 &#x000d7; 10<sup>&#x02212;7</sup> (WTCCC, <a href="#B41" rid="B41" class=" bibr popnode">2007</a>). The remaining 603,741 SNPs were used for the simulations and the following real data analysis. Because the SNP positions in the TWB data were based on the human genome GRCh37/hg19 assembly, we mapped the variants into genes according to the same assembly in the UCSC Genome Bioinformatics database (<a href="http://www.genome.ucsc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.ucsc.edu</a>). In total, 24,769 autosomal genes were identified. Furthermore, following the conventional gene-based tests (Liu et al., <a href="#B23" rid="B23" class=" bibr popnode">2010</a>), we incorporated 50 kb in the 3&#x02032; and 5&#x02032; regions that might regulate a gene.</p><p class="p">We assessed the type I error rates and power of the seven tests using simulations. Our ADABF was compared with rareGE (Chen et al., <a href="#B3" rid="B3" class=" bibr popnode">2014</a>), SBERIA (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>), GESAT (Lin et al., <a href="#B21" rid="B21" class=" bibr popnode">2013</a>), and iSKAT (Lin et al., <a href="#B22" rid="B22" class=" bibr popnode">2016</a>). These competitor methods have been developed with user-friendly analysis tools that are popular choices for G &#x000d7; E studies. The &#x0201c;rareGE&#x0201d; function in the &#x0201c;rareGE&#x0201d; R package (version 0.1) provides <em>P</em>-values for the following three tests: (1) INT_FIX: a G &#x000d7; E test that treats the SNP main effects as fixed effects; (2) INT_RAN: a G &#x000d7; E test that treats the SNP main effects as random effects; and (3) JOINT: a joint test of the genetic main effects and G &#x000d7; E interactions. Both GESAT (Lin et al., <a href="#B21" rid="B21" class=" bibr popnode">2013</a>) and iSKAT (Lin et al., <a href="#B22" rid="B22" class=" bibr popnode">2016</a>) were implemented using the &#x0201c;iSKAT&#x0201d; R package (version 1.2).</p><div id="sec-a.w.c.f" class="sec"><h3 id="sec-a.w.c.ftitle">Type I Error Rates</h3><p class="p p-first">Given the genotypes of each subject from the TWB, his/her continuous trait was simulated according to</p><div class="disp-formula" id="E6"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M23" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mi>Y</mi><mo>=</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mi>E</mi><mo>+</mo><mi>e</mi><mo>,</mo></mtd></mtr></mtable></math></div><div class="l">(6)</div></div><p>where <em>G</em><sub><em>l</em></sub> is the minor allele count (0, 1, or 2) at the <em>l</em><sup>th</sup> SNP, <em>E</em> is the environmental factor, and <em>e</em> is the random error term following the standard normal distribution. Moreover, we simulated binary traits according to</p><div class="disp-formula" id="E7"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M24" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mo class="qopname">log</mo><mfrac><mrow><mo class="qopname">Pr</mo><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi><mo>=</mo><mn>1</mn></mrow><mo stretchy="false">)</mo></mrow></mrow><mrow><mn>1</mn><mo>-</mo><mo class="qopname">Pr</mo><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi><mo>=</mo><mn>1</mn></mrow><mo stretchy="false">)</mo></mrow></mrow></mfrac><mo>=</mo><mo>-</mo><mn>0</mn><mo>.</mo><mn>4</mn><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mi>E</mi><mo>,</mo></mtd></mtr></mtable></math></div><div class="l">(7)</div></div><p>where the intercept was <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M25" overflow="scroll"><mrow><mi>log</mi><mtext>&#x000a0;</mtext><mo stretchy="false">(</mo><mfrac bevelled="true"><mrow><mn>0.4</mn></mrow><mrow><mn>0.6</mn></mrow></mfrac><mo stretchy="false">)</mo><mo>=</mo><mo>&#x02212;</mo><mn>0.4</mn></mrow></math>, corresponding to a disease prevalence of 0.4, which was the worldwide prevalence of hypertension among adults aged &#x02265; 25 years (Abebe et al., <a href="#B1" rid="B1" class=" bibr popnode">2015</a>).</p><p>In Equations (6) and (7), E was a binary environmental factor taking a value of 0 or 1 each with a probability of 0.5. Because E was randomly sampled, the &#x0201c;SNP-E association filtering&#x0201d; (Murcray et al., <a href="#B29" rid="B29" class=" bibr popnode">2009</a>; Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>) that computes the association between E and each SNP was inefficient. Therefore, in our simulations, we used the above-mentioned &#x0201c;main-effect filtering&#x0201d; (Dai et al., <a href="#B5" rid="B5" class=" bibr popnode">2012</a>) in SBERIA.</p><p>To evaluate the validity of these G &#x000d7; E tests, we let &#x003b2;<sub><em>G</em></sub> = &#x003b2;<sub><em>GE</em></sub> = 0 and generated the phenotypes of 16,555 subjects according to Equations (6) or (7). We repeated 41 rounds of genome-wide G &#x000d7; E analysis for the 24,769 autosomal genes so that each G &#x000d7; E test was evaluated at least one million times (24, 769 &#x000d7; 41 = 1, 015, 529). Following the conventional gene-based tests (Liu et al., <a href="#B23" rid="B23" class=" bibr popnode">2010</a>), we incorporated 50 kb in the 3&#x02032; and 5&#x02032; regions that might regulate a gene. The number of SNPs involved in a gene depends on the length of the gene. Table <a href="/pmc/articles/PMC6339974/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1</span></span><span>1</span></a> presents the empirical type I error rates under various nominal significance levels based on 1,015,529 replications of the continuous traits and binary traits separately. All the tests preserved the type I error rates. Tables <a href="#SM1" rid="SM1" class=" supplementary-material">S1</a>&#x02013;<a href="#SM1" rid="SM1" class=" supplementary-material">S3</a> in our Supplementary Materials further present the type I error rates stratified by the number of SNPs involved in a gene. The results are similar to Table <a href="/pmc/articles/PMC6339974/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1,</span></span><span>1</span></a>, indicating that the type I error rates do not much depend on the number of SNPs in a gene.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Empirical type I error rates in the simulation study.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th valign="top" align="left" rowspan="1" colspan="1"><strong>Traits</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Nominal significance levels</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>ADABF</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>INT_FIX</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>INT_RAN</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>JOINT</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>SBERIA</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>iSKAT</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>GESAT</strong></th></tr></thead><tbody><tr><td valign="top" align="left" rowspan="1" colspan="1">Continuous, &#x003b2;<sub><em>G</em></sub> = &#x003b2;<sub><em>GE</em></sub> = 0 assigned to Equation (6)</td><td valign="top" align="center" rowspan="1" colspan="1">0.05</td><td valign="top" align="center" rowspan="1" colspan="1">0.049807</td><td valign="top" align="center" rowspan="1" colspan="1">0.050379</td><td valign="top" align="center" rowspan="1" colspan="1">0.050163</td><td valign="top" align="center" rowspan="1" colspan="1">0.050147</td><td valign="top" align="center" rowspan="1" colspan="1">0.050389</td><td valign="top" align="center" rowspan="1" colspan="1">0.052703</td><td valign="top" align="center" rowspan="1" colspan="1">0.052583</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">0.01</td><td valign="top" align="center" rowspan="1" colspan="1">0.009549</td><td valign="top" align="center" rowspan="1" colspan="1">0.009944</td><td valign="top" align="center" rowspan="1" colspan="1">0.009882</td><td valign="top" align="center" rowspan="1" colspan="1">0.010036</td><td valign="top" align="center" rowspan="1" colspan="1">0.010081</td><td valign="top" align="center" rowspan="1" colspan="1">0.011072</td><td valign="top" align="center" rowspan="1" colspan="1">0.011570</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">0.001</td><td valign="top" align="center" rowspan="1" colspan="1">0.000927</td><td valign="top" align="center" rowspan="1" colspan="1">0.000962</td><td valign="top" align="center" rowspan="1" colspan="1">0.000948</td><td valign="top" align="center" rowspan="1" colspan="1">0.001168</td><td valign="top" align="center" rowspan="1" colspan="1">0.001008</td><td valign="top" align="center" rowspan="1" colspan="1">0.001166</td><td valign="top" align="center" rowspan="1" colspan="1">0.001300</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">0.0001</td><td valign="top" align="center" rowspan="1" colspan="1">0.000080</td><td valign="top" align="center" rowspan="1" colspan="1">0.000088</td><td valign="top" align="center" rowspan="1" colspan="1">0.000091</td><td valign="top" align="center" rowspan="1" colspan="1">0.000156</td><td valign="top" align="center" rowspan="1" colspan="1">0.000115</td><td valign="top" align="center" rowspan="1" colspan="1">0.000125</td><td valign="top" align="center" rowspan="1" colspan="1">0.000139</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">5 &#x000d7; 10<sup>&#x02212;5</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.000040</td><td valign="top" align="center" rowspan="1" colspan="1">0.000039</td><td valign="top" align="center" rowspan="1" colspan="1">0.000042</td><td valign="top" align="center" rowspan="1" colspan="1">0.000068</td><td valign="top" align="center" rowspan="1" colspan="1">0.000057</td><td valign="top" align="center" rowspan="1" colspan="1">0.000061</td><td valign="top" align="center" rowspan="1" colspan="1">0.000066</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">2.5 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.000000</td><td valign="top" align="center" rowspan="1" colspan="1">0.000001</td><td valign="top" align="center" rowspan="1" colspan="1">0.000001</td><td valign="top" align="center" rowspan="1" colspan="1">0.000002</td><td valign="top" align="center" rowspan="1" colspan="1">0.000001</td><td valign="top" align="center" rowspan="1" colspan="1">0.000002</td><td valign="top" align="center" rowspan="1" colspan="1">0.000003</td></tr><tr style="border-top: thin solid #000000;"><td valign="top" align="left" rowspan="1" colspan="1">Binary, &#x003b2;<sub><em>G</em></sub> = &#x003b2;<sub><em>GE</em></sub> = 0 assigned to Equation (7)</td><td valign="top" align="center" rowspan="1" colspan="1">0.05</td><td valign="top" align="center" rowspan="1" colspan="1">0.050339</td><td valign="top" align="center" rowspan="1" colspan="1">0.050739</td><td valign="top" align="center" rowspan="1" colspan="1">0.050540</td><td valign="top" align="center" rowspan="1" colspan="1">0.050385</td><td valign="top" align="center" rowspan="1" colspan="1">0.049992</td><td valign="top" align="center" rowspan="1" colspan="1">0.052701</td><td valign="top" align="center" rowspan="1" colspan="1">0.053239</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">0.01</td><td valign="top" align="center" rowspan="1" colspan="1">0.008855</td><td valign="top" align="center" rowspan="1" colspan="1">0.009989</td><td valign="top" align="center" rowspan="1" colspan="1">0.009913</td><td valign="top" align="center" rowspan="1" colspan="1">0.010193</td><td valign="top" align="center" rowspan="1" colspan="1">0.010111</td><td valign="top" align="center" rowspan="1" colspan="1">0.011149</td><td valign="top" align="center" rowspan="1" colspan="1">0.011655</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">0.001</td><td valign="top" align="center" rowspan="1" colspan="1">0.000865</td><td valign="top" align="center" rowspan="1" colspan="1">0.000972</td><td valign="top" align="center" rowspan="1" colspan="1">0.000965</td><td valign="top" align="center" rowspan="1" colspan="1">0.001200</td><td valign="top" align="center" rowspan="1" colspan="1">0.001011</td><td valign="top" align="center" rowspan="1" colspan="1">0.001191</td><td valign="top" align="center" rowspan="1" colspan="1">0.001333</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">0.0001</td><td valign="top" align="center" rowspan="1" colspan="1">0.000082</td><td valign="top" align="center" rowspan="1" colspan="1">0.000100</td><td valign="top" align="center" rowspan="1" colspan="1">0.000105</td><td valign="top" align="center" rowspan="1" colspan="1">0.000138</td><td valign="top" align="center" rowspan="1" colspan="1">0.000103</td><td valign="top" align="center" rowspan="1" colspan="1">0.000121</td><td valign="top" align="center" rowspan="1" colspan="1">0.000146</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">5 &#x000d7; 10<sup>&#x02212;5</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.000054</td><td valign="top" align="center" rowspan="1" colspan="1">0.000053</td><td valign="top" align="center" rowspan="1" colspan="1">0.000058</td><td valign="top" align="center" rowspan="1" colspan="1">0.000071</td><td valign="top" align="center" rowspan="1" colspan="1">0.000054</td><td valign="top" align="center" rowspan="1" colspan="1">0.000062</td><td valign="top" align="center" rowspan="1" colspan="1">0.000076</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">2.5 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.000003</td><td valign="top" align="center" rowspan="1" colspan="1">0.000003</td><td valign="top" align="center" rowspan="1" colspan="1">0.000003</td><td valign="top" align="center" rowspan="1" colspan="1">0.000003</td><td valign="top" align="center" rowspan="1" colspan="1">0.000003</td><td valign="top" align="center" rowspan="1" colspan="1">0.000005</td><td valign="top" align="center" rowspan="1" colspan="1">0.000004</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139691533616336"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p><em>Each entry represents the proportion of P-values smaller than the corresponding nominal significance level based on 1,015,529 simulation replicates</em>.</p></div></div><p class="p p-last">We also evaluated the validity of these G &#x000d7; E tests in the presence of genetic main effects. If &#x003b2;<sub><em>GE</em></sub> = 0 but &#x003b2;<sub><em>G</em></sub> &#x02260; 0, all tests, except for JOINT, were valid (results not shown). Thus, if we obtain a significant test result using the JOINT method, we cannot know whether this significance is contributed by G &#x000d7; E or not. Therefore, the JOINT test should not be used if G &#x000d7; E is of the main interest. It is suitable for detecting genetic main effects while allowing for G &#x000d7; E.</p></div><div id="sec-a.w.c.g" class="sec"><h3 id="sec-a.w.c.gtitle">Power</h3><p class="p p-first">The true number of SNPs interacting with E may not be large in the genome (McCarthy et al., <a href="#B26" rid="B26" class=" bibr popnode">2008</a>; Liu et al., <a href="#B24" rid="B24" class=" bibr popnode">2016</a>). Therefore, we simulated one or four non-null &#x003b2;<sub><em>GE</em></sub>s in a gene. To investigate the impact of the gene length on power, we randomly drew three genes (i.e., <em>CHD5, TNNT3</em>, and <em>RFX3</em>), respectively, incorporating 20, 50, and 100 SNPs, for simulations. Assuming <em>d</em> SNPs interact with E (<em>d</em> = 1 or 4), the continuous traits of the 16,555 subjects were generated according to</p><div class="disp-formula" id="E8"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M26" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mi>Y</mi><mo>=</mo><mstyle displaystyle="true"><munderover accentunder="false" accent="false"><mrow><mo>&#x02211;</mo></mrow><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>d</mi></mrow></munderover></mstyle><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mo>+</mo><mstyle displaystyle="true"><munderover accentunder="false" accent="false"><mrow><mo>&#x02211;</mo></mrow><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>d</mi></mrow></munderover></mstyle><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi><msub><mrow><mi>E</mi></mrow><mrow><mi>l</mi></mrow></msub></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mi>E</mi><mo>+</mo><mi>e</mi><mo>,</mo></mtd></mtr></mtable></math></div><div class="l">(8)</div></div><p>where &#x003b2;<sub><em>G</em><sub><em>l</em></sub></sub> is the SNP main effect, &#x003b2;<sub><em>G</em><sub><em>E</em></sub><sub><em>l</em></sub></sub> is the effect size of SNP &#x000d7; E, <em>G</em><sub><em>l</em></sub> is the minor allele count (0, 1, or 2) at the <em>l</em><sup>th</sup> SNP that interacts with E (<em>l</em> = 1, &#x02026;, <em>d</em>), and <em>e</em> is the random error term following the standard normal distribution. Moreover, the binary traits were simulated according to</p><div class="disp-formula" id="E9"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M27" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mo class="qopname">log</mo><mfrac><mrow><mo class="qopname">Pr</mo><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi><mo>=</mo><mn>1</mn></mrow><mo stretchy="false">)</mo></mrow></mrow><mrow><mn>1</mn><mo>-</mo><mo class="qopname">Pr</mo><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi><mo>=</mo><mn>1</mn></mrow><mo stretchy="false">)</mo></mrow></mrow></mfrac><mo>=</mo><mo>-</mo><mn>0</mn><mo>.</mo><mn>4</mn><mo>+</mo><mstyle displaystyle="true"><munderover accentunder="false" accent="false"><mrow><mo>&#x02211;</mo></mrow><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>d</mi></mrow></munderover></mstyle><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mo>+</mo><mstyle displaystyle="true"><munderover accentunder="false" accent="false"><mrow><mo>&#x02211;</mo></mrow><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>d</mi></mrow></munderover></mstyle><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi><msub><mrow><mi>E</mi></mrow><mrow><mi>l</mi></mrow></msub></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mi>E</mi><mo>.</mo></mtd></mtr></mtable></math></div><div class="l">(9)</div></div><p>The magnitudes of SNP main effects (|&#x003b2;<sub><em>G</em><sub><em>l</em></sub></sub>|) and SNP &#x000d7; E interaction effects (|&#x003b2;<sub><em>G</em><sub><em>E</em></sub><sub><em>l</em></sub></sub>|) were evaluated at three levels: small, medium, and large. For continuous traits, the effect sizes were uniformly drawn from [0.08, 0.12] (small), [0.13, 0.17] (medium), and [0.18, 0.22] (large), respectively. For binary traits, the effect sizes were uniformly drawn from [log(1.05), log(1.15)] (small), [log(1.25), log(1.35)] (medium), and [log(1.45), log(1.55)] (large), respectively.</p><p>Table <a href="/pmc/articles/PMC6339974/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table2</span></span><span>2</span></a> lists the 11 simulation scenarios for power comparison, including 3 for <em>d</em> = 1 and 8 for <em>d</em> = 4. Scenarios (1-1), (4-1), and (4-5) are pure interaction models without SNP main effect. Scenarios (1-2), (4-2), and (4-6) include SNP &#x000d7; E interaction effects with SNP main effects in the same direction. Scenarios (1-3), (4-3), and (4-7) include SNP &#x000d7; E interaction effects with SNP main effects in the opposite direction. Scenarios (4-4) and (4-8) include SNP &#x000d7; E interaction effects with 50% SNP main effects in the same direction and 50% in the opposite direction.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>The 11 simulation scenarios for power comparison.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th valign="top" align="center" colspan="4" style="border-bottom: thin solid #000000;" rowspan="1"><strong>SNP main effects</strong></th><th valign="top" align="center" colspan="4" style="border-bottom: thin solid #000000;" rowspan="1"><strong>SNP x E interaction effects</strong></th></tr><tr><th valign="top" align="left" rowspan="1" colspan="1"><strong>Scenario</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>&#x003b2;<sub><em>G</em></sub><sub>1</sub></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>&#x003b2;<sub><em>G</em></sub><sub>2</sub></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>&#x003b2;<sub><em>G</em></sub><sub>3</sub></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>&#x003b2;<sub><em>G</em></sub><sub>4</sub></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>&#x003b2;<sub><em>G</em></sub><sub><em>E</em></sub><sub>1</sub></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>&#x003b2;<sub><em>G</em></sub><sub><em>E</em></sub><sub>2</sub></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>&#x003b2;<sub><em>G</em></sub><sub><em>E</em></sub><sub>3</sub></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>&#x003b2;<sub><em>G</em></sub><sub><em>E</em></sub><sub>4</sub></strong></th></tr></thead><tbody><tr><td valign="top" align="left" rowspan="1" colspan="1">(1-1)</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">One positive SNP x E interaction effect without SNP main effect.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(1-2)</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">One positive SNP x E interaction effect, with SNP main effect in the same direction.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(1-3)</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">One positive SNP x E interaction effect, with SNP main effect in the opposite direction.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(4-1)</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">Four positive SNP x E interaction effects without SNP main effect.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(4-2)</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">Four positive SNP x E interaction effects, all with SNP main effects in the same direction.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(4-3)</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">Four positive SNP x E interaction effects, all with SNP main effects in the opposite direction.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(4-4)</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">Four positive SNP x E interaction effects, two with SNP main effects in the same direction and the other two in the opposite direction.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(4-5)</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">0</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">Two positive and two negative SNP x E interaction effects, without SNP main effect.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(4-6)</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">Two positive and two negative SNP x E interaction effects, all with SNP main effects in the same direction.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(4-7)</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">Two positive and two negative SNP x E interaction effects, all with SNP main effects in the opposite direction.</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">(4-8)</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">+</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" colspan="8" rowspan="1">Two positive and two negative SNP x E interaction effects, two with SNP main effects in the same direction and the other two in the opposite direction.</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139691533004480"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/table/T2/?report=objectonly">Open in a separate window</a></div></div><p>Based on 1,000 replications for each scenario, Figures <a href="/pmc/articles/PMC6339974/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures1,</span></span><span>1</span></a>, <a href="/pmc/articles/PMC6339974/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,2</span></span><span>2</span></a> present the results of 1 SNP &#x000d7; E (i.e., <em>d</em> = 1) for continuous and binary traits, respectively. The results of 4 SNP &#x000d7; E (i.e., <em>d</em> = 4) are shown in Figures <a href="/pmc/articles/PMC6339974/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures3,</span></span><span>3</span></a>, <a href="/pmc/articles/PMC6339974/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,4</span></span><span>4</span></a> (for continuous traits) and Figures <a href="/pmc/articles/PMC6339974/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures5,</span></span><span>5</span></a>, <a href="/pmc/articles/PMC6339974/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,6</span></span><span>6</span></a> (for binary traits). Under the same scenario and the same level of effect sizes, the power of all tests decreased as the number of SNPs increased. This was because the proportion of non-null &#x003b2;<sub><em>GE</em></sub>s was decreasing as the number of SNPs increased. For example, when <em>d</em> = 4, the proportions of non-null &#x003b2;<sub><em>GE</em></sub>s were 4/20, 4/50, and 4/100, respectively.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC6339974/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139691539697552"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6339974_fgene-09-00715-g0001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-09-00715-g0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6339974/bin/fgene-09-00715-g0001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139691539697552"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC6339974/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p>Power of the seven tests for continuous traits (1 SNP &#x000d7; E). The x-axis represents the number of SNPs in the gene, whereas the y-axis depicts the power at the nominal significance level &#x003b1; = 2.5 &#x000d7; 10<sup>&#x02212;6</sup>. From the top row to the bottom row, the magnitudes of SNP main effects and SNP &#x000d7; E interaction effects were evaluated at three levels: small, medium, and large, respectively.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC6339974/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139691539694384"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6339974_fgene-09-00715-g0002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-09-00715-g0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6339974/bin/fgene-09-00715-g0002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139691539694384"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC6339974/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p>Power of the seven tests for binary traits (1 SNP &#x000d7; E). The x-axis represents the number of SNPs in the gene, whereas the y-axis depicts the power at the nominal significance level &#x003b1; = 2.5 &#x000d7; 10<sup>&#x02212;6</sup>. From the top row to the bottom row, the magnitudes of SNP main effects and SNP &#x000d7; E interaction effects were evaluated at three levels: small, medium, and large, respectively.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC6339974/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139691541299808"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6339974_fgene-09-00715-g0003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-09-00715-g0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6339974/bin/fgene-09-00715-g0003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139691541299808"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC6339974/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p>Power of the seven tests for continuous traits (4 SNP &#x000d7; E, from Scenario 4-1 to 4-4). The x-axis represents the number of SNPs in the gene, whereas the y-axis depicts the power at the nominal significance level &#x003b1; = 2.5 &#x000d7; 10<sup>&#x02212;6</sup>. From the top row to the bottom row, the magnitudes of SNP main effects and SNP &#x000d7; E interaction effects were evaluated at three levels: small, medium, and large, respectively.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC6339974/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139691539966928"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6339974_fgene-09-00715-g0004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-09-00715-g0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6339974/bin/fgene-09-00715-g0004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139691539966928"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC6339974/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p>Power of the seven tests for continuous traits (4 SNP &#x000d7; E, from Scenario 4-5 to 4-8). The x-axis represents the number of SNPs in the gene, whereas the y-axis depicts the power at the nominal significance level &#x003b1; = 2.5 &#x000d7; 10<sup>&#x02212;6</sup>. From the top row to the bottom row, the magnitudes of SNP main effects and SNP &#x000d7; E interaction effects were evaluated at three levels: small, medium, and large, respectively.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC6339974/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139691539963744"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6339974_fgene-09-00715-g0005.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-09-00715-g0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6339974/bin/fgene-09-00715-g0005.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139691539963744"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC6339974/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p>Power of the seven tests for binary traits (4 SNP &#x000d7; E, from Scenario 4-1 to 4-4). The x-axis represents the number of SNPs in the gene, whereas the y-axis depicts the power at the nominal significance level &#x003b1; = 2.5 &#x000d7; 10<sup>&#x02212;6</sup>. From the top row to the bottom row, the magnitudes of SNP main effects and SNP &#x000d7; E interaction effects were evaluated at three levels: small, medium, and large, respectively.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC6339974/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139691541423648"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6339974_fgene-09-00715-g0006.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-09-00715-g0006.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6339974/bin/fgene-09-00715-g0006.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139691541423648"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC6339974/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6</a></div><!--caption a7--><div class="caption"><p>Power of the seven tests for binary traits (4 SNP &#x000d7; E, from Scenario 4-5 to 4-8). The x-axis represents the number of SNPs in the gene, whereas the y-axis depicts the power at the nominal significance level &#x003b1; = 2.5 &#x000d7; 10<sup>&#x02212;6</sup>. From the top row to the bottom row, the magnitudes of SNP main effects and SNP &#x000d7; E interaction effects were evaluated at three levels: small, medium, and large, respectively.</p></div></div></div><p>The JOINT test was generally the most powerful test. However, as mentioned above, it is not a pure G &#x000d7; E test. Among the 6 pure G &#x000d7; E tests, ADABF was more powerful under 1 SNP &#x000d7; E (i.e., <em>d</em> = 1, Figures <a href="/pmc/articles/PMC6339974/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures1,</span></span><span>1</span></a>, <a href="/pmc/articles/PMC6339974/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,2).</span></span><span>2</span></a>). Let <em>m</em> be the number of SNPs in a gene, where <em>m</em> = 20, 50, or 100 in our power comparison. When <em>d</em> = 1, <em>m</em> &#x02212; 1 SNPs exhibit no interactions with E. ADABF outperformed the other tests because it excluded SNP &#x000d7; E with smaller BF; thus, ADABF was more robust to the inclusion of many (<em>m</em> &#x02212; 1) null &#x003b2;<sub><em>GE</em></sub>s.</p><p class="p p-last">Among the 6 pure G &#x000d7; E tests, SBERIA can be more powerful than ADABF under 4 SNP &#x000d7; E (i.e., <em>d</em> = 4, Figures <a href="/pmc/articles/PMC6339974/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures3</span></span><span>3</span></a>&#x02013;<a href="/pmc/articles/PMC6339974/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>6</span></a>). However, SBERIA suffered from a power loss in Scenarios (4-4) and (4-8), where 50% SNP main effects were in the same direction with the SNP &#x000d7; E interaction effects while 50% were in the opposite direction. This is because SBERIA builds a G &#x000d7; E term by incorporating the SNPs that pass the filtering stage (i.e., <em>E</em>G&#x02032;<strong>&#x00175;</strong> in Equation 3). The weight (elements in <strong>&#x00175;</strong>) given to a SNP is 1 if it is positively associated with <em>Y</em>, &#x02212;1 if it is negatively associated with <em>Y</em>, and is a very small value (e.g., 0.0001) if the SNP is not statistically associated with <em>Y</em>. When 50% SNP main effects were in the same direction with the SNP &#x000d7; E interaction effects while 50% were in the opposite direction, the positive and negative SNP &#x000d7; E interactions in <em>E</em>G&#x02032;<strong>&#x00175;</strong> were canceled out. Therefore, SBERIA suffered from a power loss in Scenarios (4-4) and (4-8).</p></div><div id="sec-a.w.c.h" class="sec sec-last"><h3 id="sec-a.w.c.htitle">Application to the Taiwan Biobank Data</h3><p class="p p-first">Subsequently, we applied these G &#x000d7; E methods to the TWB data. Among the TWB subjects, ~79.9% were of the southern Han Chinese ancestry, ~5% were of the northern Han Chinese ancestry, and ~14.5% belonged to a third group (Chen et al., <a href="#B2" rid="B2" class=" bibr popnode">2016</a>). To adjust for the population substructure, the 603,741 SNPs that passed the quality-control stage were used to construct the principal components (PCs). We aim to explore the interaction effects between genes and alcohol consumption on blood pressure levels. Our study was approved by the Research Ethics Committee of National Taiwan University Hospital (NTUH-REC no. 201612188RINA).</p><p>In the TWB data, &#x0201c;alcohol drinking&#x0201d; is defined as a weekly alcohol intake &#x0003e;150 c.c. for at least 6 months. Among the 16,555 subjects, 14,779 subjects answered &#x0201c;no&#x0201d; to alcohol drinking, whereas 1,764 subjects answered &#x0201c;yes.&#x0201d; Totally 12 subjects did not respond to this question. Therefore, the environmental factor (&#x0201c;alcohol drinking&#x0201d;) was binary here. Both DBP and SBP were measured twice in a sitting position, with a 5-min interval between the two measurements. As suggested by Jamieson et al. (<a href="#B12" rid="B12" class=" bibr popnode">1990</a>) and others (Husemoen et al., <a href="#B11" rid="B11" class=" bibr popnode">2008</a>), two measurements of blood pressure should routinely be taken, and the average recorded. Therefore, in the following analysis, we used the average of the two measurements of DBP (or SBP) as the phenotype.</p><p class="p">Prior to the G &#x000d7; E analysis, we first regressed DBP (the average of two measured DBPs) and SBP (the average of two measured SBPs) on gender, age, alcohol drinking, body mass index (BMI), and the first seven PCs. In Table <a href="/pmc/articles/PMC6339974/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table3,</span></span><span>3</span></a>, we list the regression coefficients regarding gender, age, alcohol drinking, and BMI. Males, elder subjects, subjects consuming alcohol, and subjects with larger BMI exhibit a significantly higher mean blood pressure than females, younger subjects, subjects without alcohol consumption, and subjects with smaller BMI. On average, alcohol drinking results in an increase of ~1.51 mmHg in DBP and ~2.10 mmHg in SBP. This finding that an increased alcohol intake elevates blood pressures is consistent with the conclusions of numerous studies (Xin et al., <a href="#B43" rid="B43" class=" bibr popnode">2001</a>; Puddey and Beilin, <a href="#B32" rid="B32" class=" bibr popnode">2006</a>; Tomson and Lip, <a href="#B37" rid="B37" class=" bibr popnode">2006</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>The regression models for the DBP and SBP analyses (prior to the G &#x000d7; E analysis).</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th valign="top" align="center" colspan="4" style="border-bottom: thin solid #000000;" rowspan="1"><strong>DBP</strong><sup><strong><sup>a</sup></strong></sup></th><th valign="top" align="center" colspan="4" style="border-bottom: thin solid #000000;" rowspan="1"><strong>SBP</strong><sup><strong><sup>a</sup></strong></sup></th></tr><tr><th valign="top" align="left" rowspan="1" colspan="1"><strong>Important explanatory variables in the regression model</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Regression coefficient</strong> (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M42" overflow="scroll"><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></math>)</th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Standard error of</strong>
<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M40" overflow="scroll"><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></math></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Wald statistic</strong>, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M41" overflow="scroll"><mrow><msup><mrow><mo>&#x0200a;</mo></mrow><mstyle mathvariant="bold" mathsize="normal"><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></mstyle></msup><mtext>&#x0200b;&#x0200b;</mtext><mo>&#x02571;</mo><mtext>&#x0200b;</mtext><msub><mtext>&#x0200b;</mtext><mrow><mi>s</mi><mo>.</mo><mi>e</mi><mo>.</mo><mrow><mo>(</mo><mstyle mathvariant="bold" mathsize="normal"><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></mstyle><mo>)</mo></mrow></mrow></msub><mtext>&#x02009;</mtext></mrow></math></th><th valign="top" align="center" rowspan="1" colspan="1"><em><strong>P</strong></em><strong>-value</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Regression coefficient</strong> (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M52" overflow="scroll"><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></math>)</th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Standard error of</strong>
<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M43" overflow="scroll"><mstyle mathvariant="bold" mathsize="normal"><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></mstyle></math></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Wald statistic</strong>, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M44" overflow="scroll"><mrow><msup><mrow><mo>&#x0200a;</mo></mrow><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover></msup><mtext>&#x0200b;&#x0200b;</mtext><mo>&#x02571;</mo><mtext>&#x0200b;</mtext><msub><mtext>&#x0200b;</mtext><mrow><mi>s</mi><mo>.</mo><mi>e</mi><mo>.</mo><mrow><mo>(</mo><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover><mo>)</mo></mrow></mrow></msub><mtext>&#x02009;</mtext></mrow></math></th><th valign="top" align="center" rowspan="1" colspan="1"><em><strong>P</strong></em><strong>-value</strong></th></tr></thead><tbody><tr><td valign="top" align="left" rowspan="1" colspan="1">Gender<sup>b</sup> (1: female; 0: male)</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;5.8385</td><td valign="top" align="center" rowspan="1" colspan="1">0.1613</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;36.188</td><td valign="top" align="center" rowspan="1" colspan="1">&#x0003c; 2 &#x000d7; 10<sup>&#x02212;16</sup></td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;5.7753</td><td valign="top" align="center" rowspan="1" colspan="1">0.2440</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;23.669</td><td valign="top" align="center" rowspan="1" colspan="1">&#x0003c; 2 &#x000d7; 10<sup>&#x02212;16</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Age<sup>c</sup>(in year, continuous variable)</td><td valign="top" align="center" rowspan="1" colspan="1">0.1380</td><td valign="top" align="center" rowspan="1" colspan="1">0.0069</td><td valign="top" align="center" rowspan="1" colspan="1">19.961</td><td valign="top" align="center" rowspan="1" colspan="1">&#x0003c; 2 &#x000d7; 10<sup>&#x02212;16</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.6018</td><td valign="top" align="center" rowspan="1" colspan="1">0.0105</td><td valign="top" align="center" rowspan="1" colspan="1">57.579</td><td valign="top" align="center" rowspan="1" colspan="1">&#x0003c; 2 &#x000d7; 10<sup>&#x02212;16</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Alcohol drinking<sup>d</sup> (1: yes; 0: no)</td><td valign="top" align="center" rowspan="1" colspan="1">1.5107</td><td valign="top" align="center" rowspan="1" colspan="1">0.2552</td><td valign="top" align="center" rowspan="1" colspan="1">5.920</td><td valign="top" align="center" rowspan="1" colspan="1">3.29 &#x000d7; 10<sup>&#x02212;9</sup></td><td valign="top" align="center" rowspan="1" colspan="1">2.0961</td><td valign="top" align="center" rowspan="1" colspan="1">0.3860</td><td valign="top" align="center" rowspan="1" colspan="1">5.431</td><td valign="top" align="center" rowspan="1" colspan="1">5.69 &#x000d7; 10<sup>&#x02212;8</sup></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">Body mass index (BMI)<sup>e</sup>(in kg/m<sup>2</sup>, continuous variable)</td><td valign="top" align="center" rowspan="1" colspan="1">0.8884</td><td valign="top" align="center" rowspan="1" colspan="1">0.0215</td><td valign="top" align="center" rowspan="1" colspan="1">41.380</td><td valign="top" align="center" rowspan="1" colspan="1">&#x0003c; 2 &#x000d7; 10<sup>&#x02212;16</sup></td><td valign="top" align="center" rowspan="1" colspan="1">1.2633</td><td valign="top" align="center" rowspan="1" colspan="1">0.0325</td><td valign="top" align="center" rowspan="1" colspan="1">38.907</td><td valign="top" align="center" rowspan="1" colspan="1">&#x0003c; 2 &#x000d7; 10<sup>&#x02212;16</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139691545504048"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN1"><sup>a</sup><em>The first seven PCs were also adjusted in the model</em>.</div><div id="TN2"><sup>b</sup><em>Interpretation of gender, Males have significantly higher mean blood pressure than females</em>.</div><div id="TN3"><sup>c</sup><em>Interpretation of age, Elder subjects have significantly higher mean blood pressure than younger subjects</em>.</div><div id="TN4"><sup>d</sup><em>Interpretation of alcohol drinking, Subjects consuming alcohol have significantly higher mean blood pressure than subjects without alcohol consumption</em>.</div><div id="TN5"><sup>e</sup><em>Interpretation of BMI, Subjects with larger BMI have significantly higher mean blood pressure than subjects with smaller BMI</em>.</div></div></div><div id="sec-a.w.c.h.f" class="sec"><p></p><h4 id="sec-a.w.c.h.ftitle" class="inline">Single Marker Analysis </h4><p class="p p-first">The first strategy to detect G &#x000d7; E is single SNP analysis. Let <em>Y</em> be DBP or SBP, <em>G</em><sub><em>l</em></sub> be the number of minor allele (0, 1, or 2) at the <em>l</em><sup>th</sup> SNP, <em>E</em> be the environmental factor (&#x0201c;alcohol drinking&#x0201d;), and <em><strong>X</strong></em> be the vector of covariates, including age, gender, BMI, and the first seven PCs. We fitted a linear regression for each of the 603,741 SNPs,</p><div class="disp-formula" id="E10"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M28" overflow="scroll"><mtable class="eqnarray" columnalign="left"><mtr><mtd><mi>E</mi><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>E</mi></mrow></msub><mi>E</mi><mo>+</mo><msub><mrow><mi>&#x003b2;</mi></mrow><mrow><mi>G</mi><mi>E</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mi>E</mi><mo>+</mo><msub><mrow><msup><mrow><mstyle mathvariant="bold-italic"><mi>&#x003b2;</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mi>X</mi></mrow></msub><mstyle mathvariant="bold-italic"><mi>X</mi></mstyle><mo>,</mo><mi>l</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>&#x022ef;</mo><mspace width="0.3em" class="thinspace"></mspace><mo>,</mo><mn>603741</mn><mo>.</mo></mtd></mtr></mtable></math></div><div class="l">(10)</div></div><p class="p p-last">This single marker analysis was performed using PLINK (version 1.9) (Purcell et al., <a href="#B33" rid="B33" class=" bibr popnode">2007</a>). PLINK reported low genomic inflation factors after adjusting the first seven PCs, i.e., &#x003bb;<sub><em>GC</em></sub> = 1.081 for DBP and 1.058 for SBP. As suggested by GWAS such as Li et al. (<a href="#B18" rid="B18" class=" bibr popnode">2015</a>), these &#x003bb;<sub><em>GC</em></sub>s represented minimal effects of population stratification. We then tested <em>H</em><sub>0</sub>:&#x003b2;<sub><em>GE</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b2;<sub><em>GE</em></sub> &#x02260; 0, and compared the <em>P</em>-value with the commonly-used genome-wide significance level, 5 &#x000d7; 10<sup>&#x02212;8</sup>. No significant SNP &#x000d7; E were identified for DBP or SBP.</p></div><div id="sec-a.w.c.h.g" class="sec"><p></p><h4 id="sec-a.w.c.h.gtitle" class="inline">Gene-Based Analysis </h4><p class="p p-first">Then, we performed the seven gene-based tests. According to the human genome GRCh37/hg19 assembly, there are 24,769 autosomal genes. We followed the conventional gene-based tests (Liu et al., <a href="#B23" rid="B23" class=" bibr popnode">2010</a>), and therefore incorporated 50 kb in the 3&#x02032; and 5&#x02032; regions that might regulate genes. The &#x0201c;main-effect filtering&#x0201d; and &#x0201c;SNP-E association filtering&#x0201d; were both used in the SBERIA approach, and they were referred to as &#x0201c;SBERIA1&#x0201d; and &#x0201c;SBERIA2,&#x0201d; respectively.</p><ol class="enumerated" style="list-style-type:decimal"><li><div>SBERIA1 (main-effect filtering): In the filtering stage, a linear regression was fitted for each SNP, i.e., <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M29" overflow="scroll"><mi>E</mi><mrow><mo stretchy="false">(</mo><mrow><mi>Y</mi></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><msub><mrow><mi>&#x003b3;</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>&#x003b3;</mi></mrow><mrow><mi>G</mi></mrow></msub><msub><mrow><mi>G</mi></mrow><mrow><mi>l</mi></mrow></msub><mo>+</mo><msub><mrow><msup><mrow><mstyle mathvariant="bold"><mi>&#x003b3;</mi></mstyle></mrow><mrow><mi>&#x02032;</mi></mrow></msup></mrow><mrow><mi>X</mi></mrow></msub><mstyle mathvariant="bold-italic"><mi>X</mi></mstyle></math>. The validity of using this filtering stage was justified by Corollary 1 proposed by Dai et al. (<a href="#B5" rid="B5" class=" bibr popnode">2012</a>). Using this filtering strategy into Jiao et al.'s SBERIA approach, when the <em>P</em>-value of testing <em>H</em><sub>0</sub>:&#x003b3;<sub><em>G</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b3;<sub><em>G</em></sub> &#x02260; 0 was smaller than 0.1, the weight given to the <em>l</em><sup>th</sup> SNP was 1 if <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M30" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b3;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi></mrow></msub><mo>&#x0003e;</mo><mn>0</mn></math> and was &#x02212;1 if <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M31" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b3;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi></mrow></msub><mo>&#x0003c;</mo><mn>0</mn></math> (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M32" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b3;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi></mrow></msub></math> was the MLE of &#x003b3;<sub><em>G</em></sub>). When the <em>P</em>-value of testing <em>H</em><sub>0</sub>:&#x003b3;<sub><em>G</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b3;<sub><em>G</em></sub> &#x02260; 0 was larger than 0.1, the weight given to the <em>l</em><sup>th</sup> SNP was 0.0001 (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>).</div></li><li><div>SBERIA2 (SNP-E association filtering): In the filtering stage, a logistic regression was fitted for each SNP, i.e., <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M33" overflow="scroll"><mrow><mi>log</mi><mi>i</mi><mi>t</mi><mrow><mo>(</mo><mi>E</mi><mo>)</mo></mrow><mo>=</mo><msub><mi>&#x003b4;</mi><mn>0</mn></msub><mo>+</mo><msub><mi>&#x003b4;</mi><mi>G</mi></msub><msub><mi>G</mi><mi>l</mi></msub><mo>+</mo><msub><mstyle mathvariant="bold" mathsize="normal"><msup><mi>&#x003b4;</mi><mo>&#x02032;</mo></msup></mstyle><mi>X</mi></msub><mi>X</mi></mrow></math>, where <em>E</em> = &#x0201c;alcohol drinking&#x0201d; was binary. The validity of using this filtering stage was justified by Proposition 3 of Dai et al. (<a href="#B5" rid="B5" class=" bibr popnode">2012</a>). According to Jiao et al.'s SBERIA approach, when the <em>P</em>-value of testing <em>H</em><sub>0</sub>:&#x003b4;<sub><em>G</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b4;<sub><em>G</em></sub> &#x02260; 0 was smaller than 0.1, the weight given to the <em>l</em><sup>th</sup> SNP was 1 if <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M34" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b4;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi></mrow></msub><mo>&#x0003e;</mo><mn>0</mn></math> and was &#x02212;1 if <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M35" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b4;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi></mrow></msub><mo>&#x0003c;</mo><mn>0</mn></math> (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M36" overflow="scroll"><msub><mrow><mover accent="true"><mrow><mi>&#x003b4;</mi></mrow><mo>^</mo></mover></mrow><mrow><mi>G</mi></mrow></msub></math> was the MLE of &#x003b4;<sub><em>G</em></sub>). When the <em>P</em>-value of testing <em>H</em><sub>0</sub>:&#x003b4;<sub><em>G</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b4;<sub><em>G</em></sub> &#x02260; 0 was larger than 0.1, the weight given to the <em>l</em><sup>th</sup> SNP was 0.0001 (Jiao et al., <a href="#B14" rid="B14" class=" bibr popnode">2013</a>).</div></li></ol><p>Table <a href="/pmc/articles/PMC6339974/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table4</span></span><span>4</span></a> lists the genes that are significant according to at least one of the analysis methods, where the statistical significance is claimed if a <em>P</em> &#x0003c; 2.5 &#x000d7; 10<sup>&#x02212;6</sup>, where <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M37" overflow="scroll"><mn>2</mn><mo>.</mo><mn>5</mn><mo>&#x000d7;</mo><msup><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>6</mn></mrow></msup><msup><mrow><mo>=</mo></mrow><mrow><mn>0</mn><mo>.</mo><mn>05</mn></mrow></msup><msub><mrow><mo class="MathClass-ord">/</mo></mrow><mrow><mn>20000</mn></mrow></msub></math> is the commonly-used genome-wide significance level in gene-based analyses (Epstein et al., <a href="#B7" rid="B7" class=" bibr popnode">2015</a>). Regarding DBP, the <em>ADAMTS7P1</em> gene was identified by the GESAT test (<em>P</em> = 9.5 &#x000d7; 10<sup>&#x02212;7</sup>). At this gene, the <em>P</em>-values provided by other 6 tests all reached the suggestive significance level (<em>P</em>&#x0003c;<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M38" overflow="scroll"><mn>5</mn><mo>&#x000d7;</mo><msup><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>5</mn></mrow></msup><msup><mrow><mo>=</mo></mrow><mrow><mn>1</mn></mrow></msup><msub><mrow><mo class="MathClass-ord">/</mo></mrow><mrow><mn>20000</mn></mrow></msub></math>). The other genes listed in Table <a href="/pmc/articles/PMC6339974/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table4</span></span><span>4</span></a> were presumably to have genetic main effects rather than G &#x000d7; E interactions, because they were only identified by the JOINT test.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p>Significant genes (<em>P</em>-value &#x0003c; 2.5 &#x000d7; 10<sup>&#x02212;6</sup>) identified by at least one of the G &#x000d7; E tests.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th valign="top" align="left" rowspan="1" colspan="1"><strong>Phenotype</strong></th><th valign="top" align="left" rowspan="1" colspan="1"><strong>Gene</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Chr</strong>.</th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Analysis region</strong><sup><strong><em>a</em></strong></sup></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>#(SNPs)</strong></th><th valign="top" align="center" colspan="8" rowspan="1"><em><strong>P</strong></em><strong>-values (highlighted if smaller than the genome-wide significance level</strong>
<strong>=</strong>
<strong>2.5 &#x000d7; 10</strong><sup><strong>&#x02212;6</strong></sup><strong>)</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong><em>GxE</em></strong>
<sup><strong><em>d</em></strong></sup></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>ADABF</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>INT_FIX</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>INT_RAN</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>JOINT</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>SBERIA1</strong><sup><strong><em>b</em></strong></sup></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>SBERIA2</strong><sup><strong><em>c</em></strong></sup></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>iSKAT</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>GESAT</strong></th><th rowspan="1" colspan="1"></th></tr></thead><tbody><tr><td valign="top" align="left" rowspan="1" colspan="1">DBP</td><td valign="top" align="left" rowspan="1" colspan="1"><em>CCDC66</em></td><td valign="top" align="center" rowspan="1" colspan="1">3</td><td valign="top" align="center" rowspan="1" colspan="1">56541184&#x02013;56705864</td><td valign="top" align="center" rowspan="1" colspan="1">14</td><td valign="top" align="center" rowspan="1" colspan="1">0.94</td><td valign="top" align="center" rowspan="1" colspan="1">0.71</td><td valign="top" align="center" rowspan="1" colspan="1">0.72</td><td valign="top" align="center" rowspan="1" colspan="1">3.9 &#x000d7; 10<sup>&#x02212;7</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.42</td><td valign="top" align="center" rowspan="1" colspan="1">0.34</td><td valign="top" align="center" rowspan="1" colspan="1">0.55</td><td valign="top" align="center" rowspan="1" colspan="1">0.74</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1"><em>PRDM8</em></td><td valign="top" align="center" rowspan="1" colspan="1">4</td><td valign="top" align="center" rowspan="1" colspan="1">81055033&#x02013;81175483</td><td valign="top" align="center" rowspan="1" colspan="1">49</td><td valign="top" align="center" rowspan="1" colspan="1">5.7 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">7.6 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">6.1 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">1.4 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">1.4 &#x000d7; 10<sup>&#x02212;2</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.42</td><td valign="top" align="center" rowspan="1" colspan="1">1.1 &#x000d7; 10<sup>&#x02212;3</sup></td><td valign="top" align="center" rowspan="1" colspan="1">5.2 &#x000d7; 10<sup>&#x02212;4</sup></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1"><em>FGF5</em></td><td valign="top" align="center" rowspan="1" colspan="1">4</td><td valign="top" align="center" rowspan="1" colspan="1">81137742&#x02013;81262171</td><td valign="top" align="center" rowspan="1" colspan="1">41</td><td valign="top" align="center" rowspan="1" colspan="1">0.02</td><td valign="top" align="center" rowspan="1" colspan="1">0.12</td><td valign="top" align="center" rowspan="1" colspan="1">0.11</td><td valign="top" align="center" rowspan="1" colspan="1">7.1 &#x000d7; 10<sup>&#x02212;10</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.71</td><td valign="top" align="center" rowspan="1" colspan="1">1.00</td><td valign="top" align="center" rowspan="1" colspan="1">0.20</td><td valign="top" align="center" rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1"><em>ADAMTS7P1</em></td><td valign="top" align="center" rowspan="1" colspan="1">15</td><td valign="top" align="center" rowspan="1" colspan="1">82535621&#x02013;82676915</td><td valign="top" align="center" rowspan="1" colspan="1">4</td><td valign="top" align="center" rowspan="1" colspan="1">6.5 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">6.0 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">5.7 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">6.5 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">2.4 &#x000d7; 10<sup>&#x02212;5</sup></td><td valign="top" align="center" rowspan="1" colspan="1">1.9 &#x000d7; 10<sup>&#x02212;5</sup></td><td valign="top" align="center" rowspan="1" colspan="1">3.7 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">9.5 &#x000d7; 10<sup>&#x02212;7</sup></td><td valign="top" align="center" rowspan="1" colspan="1">Yes</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1"><em>BICDL2</em></td><td valign="top" align="center" rowspan="1" colspan="1">16</td><td valign="top" align="center" rowspan="1" colspan="1">3027683&#x02013;3136950</td><td valign="top" align="center" rowspan="1" colspan="1">8</td><td valign="top" align="center" rowspan="1" colspan="1">4.4 &#x000d7; 10<sup>&#x02212;3</sup></td><td valign="top" align="center" rowspan="1" colspan="1">6.0 &#x000d7; 10<sup>&#x02212;3</sup></td><td valign="top" align="center" rowspan="1" colspan="1">6.0 &#x000d7; 10<sup>&#x02212;3</sup></td><td valign="top" align="center" rowspan="1" colspan="1">1.5 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.02</td><td valign="top" align="center" rowspan="1" colspan="1">0.03</td><td valign="top" align="center" rowspan="1" colspan="1">5.6 &#x000d7; 10<sup>&#x02212;3</sup></td><td valign="top" align="center" rowspan="1" colspan="1">2.4 &#x000d7; 10<sup>&#x02212;3</sup></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1">Time spent (in hours) <sup>e</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">82</td><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">470</td><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">8</td><td valign="top" align="center" rowspan="1" colspan="1">10</td><td valign="top" align="center" rowspan="1" colspan="1">330</td><td valign="top" align="center" rowspan="1" colspan="1">312</td><td rowspan="1" colspan="1"></td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">SBP</td><td valign="top" align="left" rowspan="1" colspan="1"><em>FGF5</em></td><td valign="top" align="center" rowspan="1" colspan="1">4</td><td valign="top" align="center" rowspan="1" colspan="1">81137742&#x02013;81262171</td><td valign="top" align="center" rowspan="1" colspan="1">41</td><td valign="top" align="center" rowspan="1" colspan="1">0.12</td><td valign="top" align="center" rowspan="1" colspan="1">0.17</td><td valign="top" align="center" rowspan="1" colspan="1">0.17</td><td valign="top" align="center" rowspan="1" colspan="1">3.2 &#x000d7; 10<sup>&#x02212;11</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.73</td><td valign="top" align="center" rowspan="1" colspan="1">0.69</td><td valign="top" align="center" rowspan="1" colspan="1">0.29</td><td valign="top" align="center" rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1"><em>ATP2B1</em></td><td valign="top" align="center" rowspan="1" colspan="1">12</td><td valign="top" align="center" rowspan="1" colspan="1">89931826&#x02013;90153130</td><td valign="top" align="center" rowspan="1" colspan="1">27</td><td valign="top" align="center" rowspan="1" colspan="1">0.59</td><td valign="top" align="center" rowspan="1" colspan="1">0.58</td><td valign="top" align="center" rowspan="1" colspan="1">0.56</td><td valign="top" align="center" rowspan="1" colspan="1">2.1 &#x000d7; 10<sup>&#x02212;7</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.78</td><td valign="top" align="center" rowspan="1" colspan="1">0.38</td><td valign="top" align="center" rowspan="1" colspan="1">0.49</td><td valign="top" align="center" rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1"><em>ATP2B1-AS1</em></td><td valign="top" align="center" rowspan="1" colspan="1">12</td><td valign="top" align="center" rowspan="1" colspan="1">90052732&#x02013;90155729</td><td valign="top" align="center" rowspan="1" colspan="1">12</td><td valign="top" align="center" rowspan="1" colspan="1">0.68</td><td valign="top" align="center" rowspan="1" colspan="1">0.59</td><td valign="top" align="center" rowspan="1" colspan="1">0.57</td><td valign="top" align="center" rowspan="1" colspan="1">8.8 &#x000d7; 10<sup>&#x02212;8</sup></td><td valign="top" align="center" rowspan="1" colspan="1">0.96</td><td valign="top" align="center" rowspan="1" colspan="1">0.16</td><td valign="top" align="center" rowspan="1" colspan="1">0.22</td><td valign="top" align="center" rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1">Time spent (in hours) <sup>e</sup></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">78</td><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">463</td><td rowspan="1" colspan="1"></td><td valign="top" align="center" rowspan="1" colspan="1">8</td><td valign="top" align="center" rowspan="1" colspan="1">10</td><td valign="top" align="center" rowspan="1" colspan="1">327</td><td valign="top" align="center" rowspan="1" colspan="1">310</td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139691537950576"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN4a"><sup>a</sup><em>The analysis regions were based on the human GRCh37/hg19 assembly. Following the conventional gene-based tests, we also incorporated 50 kb in the 3&#x02032; and 5&#x02032; regions that might regulate the gene</em>.</div><div id="TN5a"><sup>b</sup>SBERIA1, SBERIA coupled with &#x0201c;main-effect filtering.&#x0201d;</div><div id="TN6"><sup>c</sup>SBERIA2, SBERIA coupled with &#x0201c;SNP-E association filtering.&#x0201d;</div><div id="TN7"><sup>d</sup><em>Only ADAMTS7P1 was supported to have interaction effects with alcohol consumption, on DBP. The other genes listed in this table were suggested to have genetic main effects because they were only identified by the JOINT test</em>.</div><div id="TN8"><sup>e</sup><em>The total time for analyzing all the 24,769 autosomal genes</em>.</div></div></div><p class="p">Table <a href="/pmc/articles/PMC6339974/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table5</span></span><span>5</span></a> presents the information regarding the four SNPs in the analysis region of the <em>ADAMTS7P1</em> gene. For DBP analysis, two BFs of SNP&#x000d7; alcohol interactions were &#x0003e;100 (representing decisive evidence against the null hypothesis Jeffreys, <a href="#B13" rid="B13" class=" bibr popnode">1961</a>; Kass and Raftery, <a href="#B15" rid="B15" class=" bibr popnode">1995</a>), including rs16973457 and rs4238534. Plots of the SNP &#x000d7; alcohol interaction effects on DBP and SBP are presented in Figure <a href="/pmc/articles/PMC6339974/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">Figure7.</span></span><span>7</span></a>. Here, non-drinkers (black curves) exhibit similar blood pressure values across different genotypes. However, drinkers (red dashed curves) exhibit elevated blood pressure if they possess certain genotypes. Interestingly, if we ignore <em>G</em><sub><em>l</em></sub><em>E</em> from Equation (10), the main effects of these SNPs are not significant (shown in the final column of Table <a href="/pmc/articles/PMC6339974/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table5).</span></span><span>5</span></a>). This finding highlights the importance of considering the SNP &#x000d7; alcohol interaction effect on blood pressure.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p>The four SNPs in the analysis region of the <em>ADAMTS7P1</em> gene.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th valign="top" align="left" rowspan="1" colspan="1"><strong>Phenotype</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>SNP</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Position (base pair)</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Minor allele</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Major allele</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>MAF</strong></th><th valign="top" align="center" colspan="5" style="border-bottom: thin solid #000000;" rowspan="1"><strong>SNP</strong>
<strong>&#x000d7;</strong>
<strong>alcohol interaction test</strong><sup><strong>a</strong></sup></th><th valign="top" align="center" style="border-bottom: thin solid #000000;" rowspan="1" colspan="1"><strong>If SNP &#x000d7; alcohol interaction was not incorporated in the model</strong><sup><strong>b</strong></sup></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th valign="top" align="left" rowspan="1" colspan="1"><strong><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M48" overflow="scroll"><mrow><msub><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover><mrow><mi>G</mi><mi>E</mi></mrow></msub></mrow></math></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M49" overflow="scroll"><mrow><mstyle mathvariant="bold" mathsize="normal"><mi>s</mi></mstyle><mstyle mathvariant="bold" mathsize="normal"><mo>.</mo><mi>e</mi></mstyle><mstyle mathvariant="bold" mathsize="normal"><mo>.</mo></mstyle><mrow><mo>(</mo><mrow><msub><mover accent="true"><mi>&#x003b2;</mi><mo>^</mo></mover><mrow><mstyle mathvariant="bold" mathsize="normal"><mi>G</mi><mi>E</mi></mstyle></mrow></msub></mrow><mo>)</mo></mrow></mrow></math></strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Wald statistic</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong><em>P</em>-value (<em>H</em><sub>0</sub>:&#x003b2;<sub><em>GE</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b2;<sub><em>GE</em></sub> &#x02260; 0</strong>)</th><th valign="top" align="center" rowspan="1" colspan="1"><strong>Bayes factor</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong><em>P</em>-value (<em>H</em><sub>0</sub>:&#x003b2;<sub><em>G</em></sub> = 0 vs. <em>H</em><sub>1</sub>:&#x003b2;<sub><em>G</em></sub> &#x02260; 0</strong>)</th></tr></thead><tbody><tr><td valign="top" align="left" rowspan="1" colspan="1">DBP</td><td valign="top" align="center" rowspan="1" colspan="1">rs74249839</td><td valign="top" align="center" rowspan="1" colspan="1">82537997</td><td valign="top" align="center" rowspan="1" colspan="1">G</td><td valign="top" align="center" rowspan="1" colspan="1">T</td><td valign="top" align="center" rowspan="1" colspan="1">0.0544</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;1.5477</td><td valign="top" align="center" rowspan="1" colspan="1">0.7757</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;1.995</td><td valign="top" align="center" rowspan="1" colspan="1">0.0460</td><td valign="top" align="center" rowspan="1" colspan="1">1.95</td><td valign="top" align="center" rowspan="1" colspan="1">0.58</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1">rs7183805</td><td valign="top" align="center" rowspan="1" colspan="1">82539431</td><td valign="top" align="center" rowspan="1" colspan="1">A</td><td valign="top" align="center" rowspan="1" colspan="1">G</td><td valign="top" align="center" rowspan="1" colspan="1">0.1761</td><td valign="top" align="center" rowspan="1" colspan="1">0.8042</td><td valign="top" align="center" rowspan="1" colspan="1">0.4595</td><td valign="top" align="center" rowspan="1" colspan="1">1.750</td><td valign="top" align="center" rowspan="1" colspan="1">0.0801</td><td valign="top" align="center" rowspan="1" colspan="1">0.88</td><td valign="top" align="center" rowspan="1" colspan="1">0.82</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1">rs16973457</td><td valign="top" align="center" rowspan="1" colspan="1">82563991</td><td valign="top" align="center" rowspan="1" colspan="1">T</td><td valign="top" align="center" rowspan="1" colspan="1">C</td><td valign="top" align="center" rowspan="1" colspan="1">0.1924</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;1.8607</td><td valign="top" align="center" rowspan="1" colspan="1">0.4309</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;4.318</td><td valign="top" align="center" rowspan="1" colspan="1">1.59 &#x000d7; 10<sup>&#x02212;5</sup></td><td valign="top" align="center" rowspan="1" colspan="1">1519.09</td><td valign="top" align="center" rowspan="1" colspan="1">0.65</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1">rs4238534</td><td valign="top" align="center" rowspan="1" colspan="1">82564555</td><td valign="top" align="center" rowspan="1" colspan="1">T</td><td valign="top" align="center" rowspan="1" colspan="1">C</td><td valign="top" align="center" rowspan="1" colspan="1">0.1627</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;2.1048</td><td valign="top" align="center" rowspan="1" colspan="1">0.4642</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;4.535</td><td valign="top" align="center" rowspan="1" colspan="1">5.81 &#x000d7; 10<sup>&#x02212;6</sup></td><td valign="top" align="center" rowspan="1" colspan="1">3883.43</td><td valign="top" align="center" rowspan="1" colspan="1">0.76</td></tr><tr><td valign="top" align="left" rowspan="1" colspan="1">SBP</td><td valign="top" align="center" rowspan="1" colspan="1">rs74249839</td><td valign="top" align="center" rowspan="1" colspan="1">82537997</td><td valign="top" align="center" rowspan="1" colspan="1">G</td><td valign="top" align="center" rowspan="1" colspan="1">T</td><td valign="top" align="center" rowspan="1" colspan="1">0.0544</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;1.6558</td><td valign="top" align="center" rowspan="1" colspan="1">1.1731</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;1.411</td><td valign="top" align="center" rowspan="1" colspan="1">0.1581</td><td valign="top" align="center" rowspan="1" colspan="1">0.78</td><td valign="top" align="center" rowspan="1" colspan="1">0.99</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1">rs7183805</td><td valign="top" align="center" rowspan="1" colspan="1">82539431</td><td valign="top" align="center" rowspan="1" colspan="1">A</td><td valign="top" align="center" rowspan="1" colspan="1">G</td><td valign="top" align="center" rowspan="1" colspan="1">0.1761</td><td valign="top" align="center" rowspan="1" colspan="1">0.9474</td><td valign="top" align="center" rowspan="1" colspan="1">0.6949</td><td valign="top" align="center" rowspan="1" colspan="1">1.363</td><td valign="top" align="center" rowspan="1" colspan="1">0.1728</td><td valign="top" align="center" rowspan="1" colspan="1">0.48</td><td valign="top" align="center" rowspan="1" colspan="1">0.76</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1">rs16973457</td><td valign="top" align="center" rowspan="1" colspan="1">82563991</td><td valign="top" align="center" rowspan="1" colspan="1">T</td><td valign="top" align="center" rowspan="1" colspan="1">C</td><td valign="top" align="center" rowspan="1" colspan="1">0.1924</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;2.4675</td><td valign="top" align="center" rowspan="1" colspan="1">0.6518</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;3.786</td><td valign="top" align="center" rowspan="1" colspan="1">0.000154</td><td valign="top" align="center" rowspan="1" colspan="1">186.85</td><td valign="top" align="center" rowspan="1" colspan="1">0.19</td></tr><tr><td rowspan="1" colspan="1"></td><td valign="top" align="left" rowspan="1" colspan="1">rs4238534</td><td valign="top" align="center" rowspan="1" colspan="1">82564555</td><td valign="top" align="center" rowspan="1" colspan="1">T</td><td valign="top" align="center" rowspan="1" colspan="1">C</td><td valign="top" align="center" rowspan="1" colspan="1">0.1627</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;2.4668</td><td valign="top" align="center" rowspan="1" colspan="1">0.7024</td><td valign="top" align="center" rowspan="1" colspan="1">&#x02212;3.512</td><td valign="top" align="center" rowspan="1" colspan="1">0.000446</td><td valign="top" align="center" rowspan="1" colspan="1">74.07</td><td valign="top" align="center" rowspan="1" colspan="1">0.56</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139691535845952"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN9"><sup>a</sup><em>The DBP (or SBP) was regressed by Equation (10) and &#x003b2;<sub>GE</sub> was of the main interest</em>.</div><div id="TN10"><sup>b</sup><em>If we ignored G<sub>l</sub>E from Equation (10), the main effects of these four SNPs were not significant</em>.</div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC6339974/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139691537879792"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6339974_fgene-09-00715-g0007.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is fgene-09-00715-g0007.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC6339974/bin/fgene-09-00715-g0007.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139691537879792"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/figure/F7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC6339974/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Figure 7</a></div><!--caption a7--><div class="caption"><p>Plots of SNP&#x000d7;alcohol interaction effects on DBP and SBP. These are the interaction plots of the four SNPs in Table <a href="/pmc/articles/PMC6339974/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table5.</span></span><span>5</span></a>. As shown in these plots, the SNP&#x000d7;alcohol interaction patterns in DBP are similar to those in SBP. The black curves depict the mean of DBP or SBP among the non-drinkers, whereas the red dashed curves depict that among the drinkers. The number shown on each point represents the sample size of that category.</p></div></div></div></div><div id="sec-a.w.c.h.h" class="sec sec-last"><p></p><h4 id="sec-a.w.c.h.htitle" class="inline">Computation Time </h4><p class="p p-first-last">As shown in Table <a href="/pmc/articles/PMC6339974/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table4,</span></span><span>4</span></a>, we also provide the time spent for analyzing the 24,769 autosomal genes, using a Linux platform with a Dell Intel Xeon E5-2690 2.9 GHz processor and 8 GB memory. SBERIA (8~10 h for a phenotype) is the fastest method, followed by ADABF (~80 h). iSKAT and GESAT both required more than 300 h. INT_FIX, INT_RAN, and JOINT were conducted using a function in the &#x0201c;rareGE&#x0201d; package, and these three tests totally required more than 400 h.</p></div></div></div><div id="s4" class="tsec sec"><h2 class="head no_bottom_margin" id="s4title">Discussion</h2><p class="p p-first">Environmental factors, such as diet, exercise, alcohol intake and tobacco use, can modify the associations of genetic variants with disease (Lee et al., <a href="#B17" rid="B17" class=" bibr popnode">2011</a>). G &#x000d7; E can shed light on biological processes leading to disease, identify high-risk subjects, and improve disease prediction (Hunter, <a href="#B10" rid="B10" class=" bibr popnode">2005</a>; Dudbridge and Fletcher, <a href="#B6" rid="B6" class=" bibr popnode">2014</a>).</p><p>Our ADABF method has been proposed as a powerful polygenic approach to detect G &#x000d7; E (Lin et al., <a href="#B19" rid="B19" class=" bibr popnode">2018</a>). This method can also serve as a gene-based G &#x000d7; E test. In this study, we compare our ADABF method with six existing gene-based tests, through extensive simulations and real data analyses. Our ADABF method is among the most powerful tests. Although the JOINT test is typically the most powerful method, it is not appropriate for assessing G &#x000d7; E in the presence of genetic main effects. As presented by Table <a href="/pmc/articles/PMC6339974/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table4,</span></span><span>4</span></a>, although seven genes were identified as significant by JOINT, none of them were replicated by any of the six pure G &#x000d7; E tests at the genome-wide significance level (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M45" overflow="scroll"><mn>2</mn><mo>.</mo><mn>5</mn><mo>&#x000d7;</mo><msup><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>6</mn></mrow></msup><msup><mrow><mo>=</mo></mrow><mrow><mn>0</mn><mo>.</mo><mn>05</mn></mrow></msup><msub><mrow><mo class="MathClass-ord">/</mo></mrow><mrow><mn>20000</mn></mrow></msub></math>) or at the suggestive significance level (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M46" overflow="scroll"><mn>5</mn><mo>&#x000d7;</mo><msup><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>5</mn></mrow></msup><msup><mrow><mo>=</mo></mrow><mrow><mn>1</mn></mrow></msup><msub><mrow><mo class="MathClass-ord">/</mo></mrow><mrow><mn>20000</mn></mrow></msub></math>). The JOINT test should not be used if G &#x000d7; E is of the main interest, but it is useful in detecting genetic main effects while allowing for G &#x000d7; E.</p><p>Notably, all gene-based tests can be performed using a pre-specified weighting scheme for SNPs. For example, if rare variants are believed to have stronger interactions with E, the beta distribution density function with parameters 1 and 25 evaluated at the sample MAF, i.e., <em>Beta</em>(<em>MAF</em>; 1, 25), is commonly used to weight the SNPs (Wu et al., <a href="#B42" rid="B42" class=" bibr popnode">2011</a>; Lin et al., <a href="#B22" rid="B22" class=" bibr popnode">2016</a>). However, to present a fair comparison, we do not impose any additional weighting on these seven tests.</p><p>The <em>ADAMTS7P1</em> gene at 15q25.2 has not been reported to be associated with blood pressures or hypertension. Indeed, after removing the SNP&#x000d7;alcohol interaction (<em>G</em><sub><em>l</em></sub><em>E</em>) from Equation (10), the main effects of all the four SNPs within <em>ADAMTS7P1</em> were not significant. Although the <em>ADAMTS7P1</em>-alcohol interaction effect on SBP did not achieve the suggestive significance level (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M47" overflow="scroll"><mn>5</mn><mo>&#x000d7;</mo><msup><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>5</mn></mrow></msup><msup><mrow><mo>=</mo></mrow><mrow><mn>1</mn></mrow></msup><msub><mrow><mo class="MathClass-ord">/</mo></mrow><mrow><mn>20000</mn></mrow></msub></math>), the <em>P</em>-values of the 6 pure G &#x000d7; E tests were all &#x0003c;10<sup>&#x02212;3</sup> (Table <a href="/pmc/articles/PMC6339974/table/T6/" target="table" class="fig-table-link figpopup" rid-figpopup="T6" rid-ob="ob-T6" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -3em;">(Table6).</span></span><span>6</span></a>). Moreover, as presented in the bottom part of Table <a href="/pmc/articles/PMC6339974/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -2.5em;">Table5,</span></span><span>5</span></a>, the BF of rs16973457&#x000d7; alcohol interaction on SBP was &#x0003e;100 (representing decisive evidence against the null hypothesis Jeffreys, <a href="#B13" rid="B13" class=" bibr popnode">1961</a>; Kass and Raftery, <a href="#B15" rid="B15" class=" bibr popnode">1995</a>). Further gene&#x000d7;alcohol studies investigating this chromosome region will be warranted.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T6"><h3>Table 6</h3><!--caption a7--><div class="caption"><p>Analysis of the <em>ADAMTS7P1</em>-alcohol interaction effect on SBP.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th valign="top" align="left" rowspan="1" colspan="1"><strong>Gene</strong></th><th valign="top" align="left" rowspan="1" colspan="1"><strong>Chr</strong>.</th><th valign="top" align="left" rowspan="1" colspan="1"><strong>Analysis region</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>#(SNPs)</strong></th><th valign="top" align="center" colspan="8" style="border-bottom: thin solid #000000;" rowspan="1"><em><strong>P</strong></em><strong>-values</strong></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>ADABF</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>INT_FIX</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>INT_RAN</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>JOINT</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>SBERIA1</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>SBERIA2</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>iSKAT</strong></th><th valign="top" align="center" rowspan="1" colspan="1"><strong>GESAT</strong></th></tr></thead><tbody><tr><td valign="top" align="left" rowspan="1" colspan="1"><em>ADAMTS7P1</em></td><td valign="top" align="left" rowspan="1" colspan="1">15</td><td valign="top" align="left" rowspan="1" colspan="1">82535621&#x02013;82676915</td><td valign="top" align="center" rowspan="1" colspan="1">4</td><td valign="top" align="center" rowspan="1" colspan="1">3.5 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">2.2 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">2.0 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">1.3 &#x000d7; 10<sup>&#x02212;3</sup></td><td valign="top" align="center" rowspan="1" colspan="1">5.0 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">1.7 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">2.0 &#x000d7; 10<sup>&#x02212;4</sup></td><td valign="top" align="center" rowspan="1" colspan="1">8.2 &#x000d7; 10<sup>&#x02212;5</sup></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139691535656512"><a target="object" rel="noopener" href="/pmc/articles/PMC6339974/table/T6/?report=objectonly">Open in a separate window</a></div></div><p>In this study, we extend our ADABF to G &#x000d7; E detection and compare it with six existing tests. Their validity, power, robustness, and computation time are investigated. SBERIA builds a G &#x000d7; E term by incorporating the SNPs that pass the filtering stage (i.e., <em>E<strong>G</strong></em><strong>&#x02032;&#x00175;</strong> in Equation 3); ADABF removes the SNP &#x000d7; E with smaller BFs. Both approaches take the advantage of screening out noises, and therefore they are usually more powerful than other pure G &#x000d7; E tests (Figures <a href="/pmc/articles/PMC6339974/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -4em;">(Figures1</span></span><span>1</span></a>&#x02013;<a href="/pmc/articles/PMC6339974/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>6</span></a>). However, it is worth noting that SBERIA suffers from a power loss when 50% SNP main effects are in the same direction with the SNP &#x000d7; E interaction effects while 50% are in the opposite direction. Considering the validity, power performance, robustness, and computation time, ADABF is recommended for genome-wide G &#x000d7; E analyses.</p><p class="p p-last">To detect G &#x000d7; E on a genome-wide scale, ADABF polygenic test (Lin et al., <a href="#B19" rid="B19" class=" bibr popnode">2018</a>) and ADABF gene-based test are two strategies with different aims. The ADABF polygenic test combines all SNPs that pass the pruning and filtering stages into a test, and therefore it does not suffer from a power loss due to the multiple-testing correction. A <em>P</em> &#x0003c; 0.05 or 0.01 is sufficient to reject <em>H</em><sub>0</sub> of no polygenic G &#x000d7; E interactions (Pan et al., <a href="#B31" rid="B31" class=" bibr popnode">2015</a>). By contrast, the power of ADABF gene-based test is compromised by the penalty of multiple testing. A <em>P</em>&#x0003c;<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M50" overflow="scroll"><mn>2</mn><mo>.</mo><mn>5</mn><mo>&#x000d7;</mo><msup><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>6</mn></mrow></msup><msup><mrow><mo>=</mo></mrow><mrow><mn>0</mn><mo>.</mo><mn>05</mn></mrow></msup><msub><mrow><mo class="MathClass-ord">/</mo></mrow><mrow><mn>20000</mn></mrow></msub></math> is required to claim a significant gene-based test (Epstein et al., <a href="#B7" rid="B7" class=" bibr popnode">2015</a>). Despite a much more stringent significance threshold, the ADABF gene-based test can make statistical inference for specific chromosomal regions, whereas the ADABF polygenic test (Lin et al., <a href="#B19" rid="B19" class=" bibr popnode">2018</a>) make an inference for SNPs (passing the pruning and filtering stages) spread out the whole genome.</p></div><div id="s5" class="tsec sec"><h2 class="head no_bottom_margin" id="s5title">Author Contributions</h2><p class="p p-first">W-YL developed the ADABF method and the analysis tool, designed and performed the simulation study, analyzed the TWB data, and wrote the manuscript. C-CH contributed to the review and coding for the competitor methods. Y-LL, S-JT, and P-HK contributed to the writing of the manuscript, and provided the TWB data. All authors reviewed the manuscript.</p><div id="sec-a.w.e.c" class="sec sec-last"><h3 id="sec-a.w.e.ctitle">Conflict of Interest Statement</h3><p class="p p-first-last">The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></div></div><div id="ack-a.x.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.x.atitle">Acknowledgments</h2><div class="sec"><p>We would like to thank the reviewers for their insightful and constructive comments, and Mr. Ya-Chin Lee for assisting with the acquisition of TWB data. This work was supported by the Ministry of Science and Technology of Taiwan (grant number MOST 107-2314-B-002-195-MY3 to W-YL). The acquisition of TWB data was supported by a MOST grant (grant number MOST 102-2314-B-002-117-MY3 to P-HK) and a collaboration grant (National Taiwan University Hospital: grant number UN106-050 to Shyr-Chyr Chen and P-HK).</p></div></div><div id="s6" class="tsec bk-sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="s6title">Supplementary Material</h2><!--/article/back/sec/--><p class="p p-first">The Supplementary Material for this article can be found online at: <a href="https://www.frontiersin.org/articles/10.3389/fgene.2018.00715/full#supplementary-material" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.frontiersin.org/articles/10.3389/fgene.2018.00715/full#supplementary-material</a></p><div class="sec suppmat" id="SM1"><div class="sup-box half_rhythm" id="media-a.x.b.c.a"><a href="/pmc/articles/PMC6339974/bin/Data_Sheet_1.PDF" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(323K, PDF)</sup></div></div></div><div id="ref-list-a.x.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.x.ctitle">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="B1"><span class="mixed-citation">Abebe S. M., Berhane Y., Worku A., Getachew A. (2015). <span class="ref-title">Prevalence and associated factors of hypertension: a crossectional community based study in Northwest Ethiopia</span>. <span class="ref-journal">PLos ONE</span>
<span class="ref-vol">10</span>:e0125210. 10.1371/journal.pone.0125210 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4409323/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25909382" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0125210" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLos+ONE&amp;title=Prevalence+and+associated+factors+of+hypertension:+a+crossectional+community+based+study+in+Northwest+Ethiopia&amp;author=S.+M.+Abebe&amp;author=Y.+Berhane&amp;author=A.+Worku&amp;author=A.+Getachew&amp;volume=10&amp;publication_year=2015&amp;pages=e0125210&amp;pmid=25909382&amp;doi=10.1371/journal.pone.0125210&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B2"><span class="mixed-citation">Chen C. H., Yang J. H., Chiang C. W. K., Hsiung C. N., Wu P. E., Chang L. C., et al.. (2016). <span class="ref-title">Population structure of Han Chinese in the modern Taiwanese population based on 10,000 participants in the Taiwan Biobank project</span>. <span class="ref-journal">Hum. Mol. Genet</span>. <span class="ref-vol">25</span>, 5321&#x02013;5331. 10.1093/hmg/ddw346 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6078601/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27798100" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fhmg%2Fddw346" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Population+structure+of+Han+Chinese+in+the+modern+Taiwanese+population+based+on+10,000+participants+in+the+Taiwan+Biobank+project&amp;author=C.+H.+Chen&amp;author=J.+H.+Yang&amp;author=C.+W.+K.+Chiang&amp;author=C.+N.+Hsiung&amp;author=P.+E.+Wu&amp;volume=25&amp;publication_year=2016&amp;pages=5321-5331&amp;pmid=27798100&amp;doi=10.1093/hmg/ddw346&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B3"><span class="mixed-citation">Chen H., Meigs J. B., Dupuis J. (2014). <span class="ref-title">Incorporating gene-environment interaction in testing for association with rare genetic variants</span>. <span class="ref-journal">Hum. Hered</span>.<span class="ref-vol">78</span>, 81&#x02013;90. 10.1159/000363347 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4169076/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25060534" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1159%2F000363347" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered&amp;title=Incorporating+gene-environment+interaction+in+testing+for+association+with+rare+genetic+variants&amp;author=H.+Chen&amp;author=J.+B.+Meigs&amp;author=J.+Dupuis&amp;volume=78&amp;publication_year=2014&amp;pages=81-90&amp;pmid=25060534&amp;doi=10.1159/000363347&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B4"><span class="mixed-citation">Chen P., Lin J. J., Lu C. S., Ong C. T., Hsieh P. F., Yang C. C., et al.. (2011). <span class="ref-title">Carbamazepine-induced toxic effects and HLA-B<sup>*</sup>1502 screening in Taiwan</span>. <span class="ref-journal">N. Engl. J. Med</span>. <span class="ref-vol">364</span>, 1126&#x02013;1133. 10.1056/NEJMoa1009717 [<a href="https://pubmed.ncbi.nlm.nih.gov/21428768" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1056%2FNEJMoa1009717" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Carbamazepine-induced+toxic+effects+and+HLA-B*1502+screening+in+Taiwan&amp;author=P.+Chen&amp;author=J.+J.+Lin&amp;author=C.+S.+Lu&amp;author=C.+T.+Ong&amp;author=P.+F.+Hsieh&amp;volume=364&amp;publication_year=2011&amp;pages=1126-1133&amp;pmid=21428768&amp;doi=10.1056/NEJMoa1009717&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B5"><span class="mixed-citation">Dai J. Y., Kooperberg C., Leblanc M., Prentice R. L. (2012). <span class="ref-title">Two-stage testing procedures with independent filtering for genome-wide gene-environment interaction</span>. <span class="ref-journal">Biometrika</span>
<span class="ref-vol">99</span>, 929&#x02013;944. 10.1093/biomet/ass044 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3629859/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23843674" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbiomet%2Fass044" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=Two-stage+testing+procedures+with+independent+filtering+for+genome-wide+gene-environment+interaction&amp;author=J.+Y.+Dai&amp;author=C.+Kooperberg&amp;author=M.+Leblanc&amp;author=R.+L.+Prentice&amp;volume=99&amp;publication_year=2012&amp;pages=929-944&amp;pmid=23843674&amp;doi=10.1093/biomet/ass044&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B6"><span class="mixed-citation">Dudbridge F., Fletcher O. (2014). <span class="ref-title">Gene-environment dependence creates spurious gene-environment interaction</span>. <span class="ref-journal">Am. J. Hum. Genet</span>. <span class="ref-vol">95</span>, 301&#x02013;307. 10.1016/j.ajhg.2014.07.014 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4157149/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25152454" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ajhg.2014.07.014" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Gene-environment+dependence+creates+spurious+gene-environment+interaction&amp;author=F.+Dudbridge&amp;author=O.+Fletcher&amp;volume=95&amp;publication_year=2014&amp;pages=301-307&amp;pmid=25152454&amp;doi=10.1016/j.ajhg.2014.07.014&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B7"><span class="mixed-citation">Epstein M. P., Duncan R., Ware E. B., Jhun M. A., Bielak L. F., Zhao W., et al.. (2015). <span class="ref-title">A statistical approach for rare-variant association testing in affected sibships</span>. <span class="ref-journal">Am. J. Hum. Genet</span>. <span class="ref-vol">96</span>, 543&#x02013;554. 10.1016/j.ajhg.2015.01.020 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4385187/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25799106" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ajhg.2015.01.020" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+statistical+approach+for+rare-variant+association+testing+in+affected+sibships&amp;author=M.+P.+Epstein&amp;author=R.+Duncan&amp;author=E.+B.+Ware&amp;author=M.+A.+Jhun&amp;author=L.+F.+Bielak&amp;volume=96&amp;publication_year=2015&amp;pages=543-554&amp;pmid=25799106&amp;doi=10.1016/j.ajhg.2015.01.020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B8"><span class="mixed-citation">Franks P. W., Pare G. (2016). <span class="ref-title">Putting the genome in context: gene-environment interactions in type 2 diabetes</span>. <span class="ref-journal">Curr. Diab. Rep</span>. <span class="ref-vol">16</span>:57. 10.1007/s11892-016-0758-y [<a href="https://pubmed.ncbi.nlm.nih.gov/27155607" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs11892-016-0758-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Diab.+Rep&amp;title=Putting+the+genome+in+context:+gene-environment+interactions+in+type+2+diabetes&amp;author=P.+W.+Franks&amp;author=G.+Pare&amp;volume=16&amp;publication_year=2016&amp;pages=57&amp;pmid=27155607&amp;doi=10.1007/s11892-016-0758-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B9"><span class="mixed-citation">He M., Allen A. (2010). <span class="ref-title">Testing gene-treatment interactions in pharmacogenetic studies</span>. <span class="ref-journal">J. Biopharm. Stat</span>. <span class="ref-vol">20</span>, 301&#x02013;314. 10.1080/10543400903572761 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3706096/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20309760" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1080%2F10543400903572761" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biopharm.+Stat&amp;title=Testing+gene-treatment+interactions+in+pharmacogenetic+studies&amp;author=M.+He&amp;author=A.+Allen&amp;volume=20&amp;publication_year=2010&amp;pages=301-314&amp;pmid=20309760&amp;doi=10.1080/10543400903572761&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B10"><span class="mixed-citation">Hunter D. J. (2005). <span class="ref-title">Gene-environment interactions in human diseases</span>. <span class="ref-journal">Nat. Rev. Genet</span>. <span class="ref-vol">6</span>, 287&#x02013;298. 10.1038/nrg1578 [<a href="https://pubmed.ncbi.nlm.nih.gov/15803198" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrg1578" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Gene-environment+interactions+in+human+diseases&amp;author=D.+J.+Hunter&amp;volume=6&amp;publication_year=2005&amp;pages=287-298&amp;pmid=15803198&amp;doi=10.1038/nrg1578&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B11"><span class="mixed-citation">Husemoen L. L. N., Fenger M., Friedrich N., Tolstrup J. S., Fredriksen S. B., Linneberg A. (2008). <span class="ref-title">The association of ADH and ALDH gene variants with alcohol drinking habits and cardiovascular disease risk factors</span>. <span class="ref-journal">Alcohol Clin. Exp. Res</span>. <span class="ref-vol">32</span>, 1984&#x02013;1991. 10.1111/j.1530-0277.2008.00780.x [<a href="https://pubmed.ncbi.nlm.nih.gov/18782342" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1530-0277.2008.00780.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alcohol+Clin.+Exp.+Res&amp;title=The+association+of+ADH+and+ALDH+gene+variants+with+alcohol+drinking+habits+and+cardiovascular+disease+risk+factors&amp;author=L.+L.+N.+Husemoen&amp;author=M.+Fenger&amp;author=N.+Friedrich&amp;author=J.+S.+Tolstrup&amp;author=S.+B.+Fredriksen&amp;volume=32&amp;publication_year=2008&amp;pages=1984-1991&amp;pmid=18782342&amp;doi=10.1111/j.1530-0277.2008.00780.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B12"><span class="mixed-citation">Jamieson M. J., Webster J., Philips S., Jeffers T. A., Scott A. K., Robb O. J., et al. (1990). <span class="ref-title">The measurement of blood pressure: sitting or supine, once or twice?</span>
<span class="ref-journal">J. Hypertens</span>. <span class="ref-vol">8</span>, 635&#x02013;640. 10.1097/00004872-199007000-00006 [<a href="https://pubmed.ncbi.nlm.nih.gov/2168453" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1097%2F00004872-199007000-00006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hypertens&amp;title=The+measurement+of+blood+pressure:+sitting+or+supine,+once+or+twice?&amp;author=M.+J.+Jamieson&amp;author=J.+Webster&amp;author=S.+Philips&amp;author=T.+A.+Jeffers&amp;author=A.+K.+Scott&amp;volume=8&amp;publication_year=1990&amp;pages=635-640&amp;pmid=2168453&amp;doi=10.1097/00004872-199007000-00006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B13"><span class="mixed-citation">Jeffreys H. (1961). <span class="ref-journal">Theory of Probability, 3rd Edn</span>. Oxford: Oxford University Press. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Theory+of+Probability,+3rd+Edn&amp;author=H.+Jeffreys&amp;publication_year=1961&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B14"><span class="mixed-citation">Jiao S., Hsu L., Bezieau S., Brenner H., Chan A. T., Chang-Claude J., et al.. (2013). <span class="ref-title">SBERIA: set-based gene-environment interaction test for rare and common variants in complex diseases</span>. <span class="ref-journal">Genet. Epidemiol</span>. <span class="ref-vol">37</span>, 452&#x02013;464. 10.1002/gepi.21735 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3713231/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23720162" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fgepi.21735" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=SBERIA:+set-based+gene-environment+interaction+test+for+rare+and+common+variants+in+complex+diseases&amp;author=S.+Jiao&amp;author=L.+Hsu&amp;author=S.+Bezieau&amp;author=H.+Brenner&amp;author=A.+T.+Chan&amp;volume=37&amp;publication_year=2013&amp;pages=452-464&amp;pmid=23720162&amp;doi=10.1002/gepi.21735&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B15"><span class="mixed-citation">Kass R. E., Raftery A. E. (1995). <span class="ref-title">Bayes factors</span>. <span class="ref-journal">J. Am. Stat. Assoc</span>.<span class="ref-vol">90</span>, 773&#x02013;795. 10.1080/01621459.1995.10476572 [<a href="//doi.org/10.1080%2F01621459.1995.10476572" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Stat.+Assoc&amp;title=Bayes+factors&amp;author=R.+E.+Kass&amp;author=A.+E.+Raftery&amp;volume=90&amp;publication_year=1995&amp;pages=773-795&amp;doi=10.1080/01621459.1995.10476572&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B16"><span class="mixed-citation">Ko T. M., Tsai C. Y., Chen S. Y., Chen K. S., Yu K. H., Chu C. S., et al.. (2015). <span class="ref-title">Use of HLA-B<sup>*</sup>58:01 genotyping to prevent allopurinol induced severe cutaneous adverse reactions in Taiwan: national prospective cohort study</span>. <span class="ref-journal">BMJ</span>
<span class="ref-vol">351</span>:h4848. 10.1136/bmj.h4848 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4579807/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26399967" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1136%2Fbmj.h4848" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Use+of+HLA-B*58:01+genotyping+to+prevent+allopurinol+induced+severe+cutaneous+adverse+reactions+in+Taiwan:+national+prospective+cohort+study&amp;author=T.+M.+Ko&amp;author=C.+Y.+Tsai&amp;author=S.+Y.+Chen&amp;author=K.+S.+Chen&amp;author=K.+H.+Yu&amp;volume=351&amp;publication_year=2015&amp;pages=h4848&amp;pmid=26399967&amp;doi=10.1136/bmj.h4848&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B17"><span class="mixed-citation">Lee Y. C., Lai C. Q., Ordovas J. M., Parnell L. D. (2011). <span class="ref-title">A database of gene-environment interactions pertaining to blood lipid traits, cardiovascular disease and type 2 diabetes</span>. <span class="ref-journal">J. Data Mining Genomics Proteomics</span>
<span class="ref-vol">2</span>:106. 10.4172/2153-0602.1000106 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3275815/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22328972" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.4172%2F2153-0602.1000106" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Data+Mining+Genomics+Proteomics&amp;title=A+database+of+gene-environment+interactions+pertaining+to+blood+lipid+traits,+cardiovascular+disease+and+type+2+diabetes&amp;author=Y.+C.+Lee&amp;author=C.+Q.+Lai&amp;author=J.+M.+Ordovas&amp;author=L.+D.+Parnell&amp;volume=2&amp;publication_year=2011&amp;pages=106&amp;pmid=22328972&amp;doi=10.4172/2153-0602.1000106&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B18"><span class="mixed-citation">Li M., Foo J. N., Wang J. Q., Low H. Q., Tang X. Q., Toh K. Y., et al.. (2015). <span class="ref-title">Identification of new susceptibility loci for IgA nephropathy in Han Chinese</span>. <span class="ref-journal">Nat. Commun</span>. <span class="ref-vol">6</span>:7270. 10.1038/ncomms8270 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4458882/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26028593" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fncomms8270" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Identification+of+new+susceptibility+loci+for+IgA+nephropathy+in+Han+Chinese&amp;author=M.+Li&amp;author=J.+N.+Foo&amp;author=J.+Q.+Wang&amp;author=H.+Q.+Low&amp;author=X.+Q.+Tang&amp;volume=6&amp;publication_year=2015&amp;pages=7270&amp;pmid=26028593&amp;doi=10.1038/ncomms8270&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B19"><span class="mixed-citation">Lin W. Y., Huang C. C., Liu Y. L., Tsai S. J., Kuo P. H. (2018). <span class="ref-title">Polygenic approaches to detect gene-environment interactions when external information is unavailable</span>. <span class="ref-journal">Brief Bioinform.</span> [Epub ahead of print]. 10.1093/bib/bby086. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6954453/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30219835" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbib%2Fbby086." target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brief+Bioinform.&amp;title=Polygenic+approaches+to+detect+gene-environment+interactions+when+external+information+is+unavailable&amp;author=W.+Y.+Lin&amp;author=C.+C.+Huang&amp;author=Y.+L.+Liu&amp;author=S.+J.+Tsai&amp;author=P.+H.+Kuo&amp;publication_year=2018&amp;pmid=30219835&amp;doi=10.1093/bib/bby086.&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B20"><span class="mixed-citation">Lin W. Y., Lee W. C. (2010). <span class="ref-title">Incorporating prior knowledge to facilitate discoveries in a genome-wide association study on age-related macular degeneration</span>. <span class="ref-journal">BMC Res Notes</span>
<span class="ref-vol">3</span>:26. 10.1186/1756-0500-3-26 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2843735/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20181037" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1756-0500-3-26" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Res+Notes&amp;title=Incorporating+prior+knowledge+to+facilitate+discoveries+in+a+genome-wide+association+study+on+age-related+macular+degeneration&amp;author=W.+Y.+Lin&amp;author=W.+C.+Lee&amp;volume=3&amp;publication_year=2010&amp;pages=26&amp;pmid=20181037&amp;doi=10.1186/1756-0500-3-26&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B21"><span class="mixed-citation">Lin X. Y., Lee S., Christiani D. C., Lin X. H. (2013). <span class="ref-title">Test for interactions between a genetic marker set and environment in generalized linear models</span>. <span class="ref-journal">Biostatistics</span>
<span class="ref-vol">14</span>, 667&#x02013;681. 10.1093/biostatistics/kxt006 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3769996/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23462021" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbiostatistics%2Fkxt006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biostatistics&amp;title=Test+for+interactions+between+a+genetic+marker+set+and+environment+in+generalized+linear+models&amp;author=X.+Y.+Lin&amp;author=S.+Lee&amp;author=D.+C.+Christiani&amp;author=X.+H.+Lin&amp;volume=14&amp;publication_year=2013&amp;pages=667-681&amp;pmid=23462021&amp;doi=10.1093/biostatistics/kxt006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B22"><span class="mixed-citation">Lin X. Y., Lee S., Wu M. C., Wang C. L., Chen H., Li Z. L., et al.. (2016). <span class="ref-title">Test for rare variants by environment interactions in sequencing association studies</span>. <span class="ref-journal">Biometrics</span>
<span class="ref-vol">72</span>, 156&#x02013;164. 10.1111/biom.12368 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4733434/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26229047" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fbiom.12368" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Test+for+rare+variants+by+environment+interactions+in+sequencing+association+studies&amp;author=X.+Y.+Lin&amp;author=S.+Lee&amp;author=M.+C.+Wu&amp;author=C.+L.+Wang&amp;author=H.+Chen&amp;volume=72&amp;publication_year=2016&amp;pages=156-164&amp;pmid=26229047&amp;doi=10.1111/biom.12368&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B23"><span class="mixed-citation">Liu J. Z., Mcrae A. F., Nyholt D. R., Medland S. E., Wray N. R., Brown K. M., et al.. (2010). <span class="ref-title">A versatile gene-based test for genome-wide association studies</span>. <span class="ref-journal">Am. J. Hum. Genet</span>. <span class="ref-vol">87</span>, 139&#x02013;145. 10.1016/j.ajhg.2010.06.009 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896770/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20598278" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ajhg.2010.06.009" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+versatile+gene-based+test+for+genome-wide+association+studies&amp;author=J.+Z.+Liu&amp;author=A.+F.+Mcrae&amp;author=D.+R.+Nyholt&amp;author=S.+E.+Medland&amp;author=N.+R.+Wray&amp;volume=87&amp;publication_year=2010&amp;pages=139-145&amp;pmid=20598278&amp;doi=10.1016/j.ajhg.2010.06.009&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B24"><span class="mixed-citation">Liu Q., Chen L. S., Nicolae D. L., Pierce B. L. (2016). <span class="ref-title">A unified set-based test with adaptive filtering for gene-environment interaction analyses</span>. <span class="ref-journal">Biometrics</span>
<span class="ref-vol">72</span>, 629&#x02013;638. 10.1111/biom.12428 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4842175/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26496228" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fbiom.12428" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=A+unified+set-based+test+with+adaptive+filtering+for+gene-environment+interaction+analyses&amp;author=Q.+Liu&amp;author=L.+S.+Chen&amp;author=D.+L.+Nicolae&amp;author=B.+L.+Pierce&amp;volume=72&amp;publication_year=2016&amp;pages=629-638&amp;pmid=26496228&amp;doi=10.1111/biom.12428&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B25"><span class="mixed-citation">Lowe J. K., Maller J. B., Pe'er I., Neale B. M., Salit J., Kenny E. E., et al.. (2009). <span class="ref-title">Genome-wide association studies in an isolated founder population from the Pacific Island of Kosrae</span>. <span class="ref-journal">PLoS Genet</span>. <span class="ref-vol">5</span>:e1000365. 10.1371/journal.pgen.1000365 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628735/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19197348" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pgen.1000365" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+studies+in+an+isolated+founder+population+from+the+Pacific+Island+of+Kosrae&amp;author=J.+K.+Lowe&amp;author=J.+B.+Maller&amp;author=I.+Pe'er&amp;author=B.+M.+Neale&amp;author=J.+Salit&amp;volume=5&amp;publication_year=2009&amp;pages=e1000365&amp;pmid=19197348&amp;doi=10.1371/journal.pgen.1000365&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B26"><span class="mixed-citation">McCarthy M. I., Abecasis G. R., Cardon L. R., Goldstein D. B., Little J., Ioannidis J. P., et al.. (2008). <span class="ref-title">Genome-wide association studies for complex traits: consensus, uncertainty and challenges</span>. <span class="ref-journal">Nat. Rev. Genet</span>. <span class="ref-vol">9</span>, 356&#x02013;369. 10.1038/nrg2344 [<a href="https://pubmed.ncbi.nlm.nih.gov/18398418" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrg2344" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Genome-wide+association+studies+for+complex+traits:+consensus,+uncertainty+and+challenges&amp;author=M.+I.+McCarthy&amp;author=G.+R.+Abecasis&amp;author=L.+R.+Cardon&amp;author=D.+B.+Goldstein&amp;author=J.+Little&amp;volume=9&amp;publication_year=2008&amp;pages=356-369&amp;pmid=18398418&amp;doi=10.1038/nrg2344&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B27"><span class="mixed-citation">Mok K. Y., Schneider S. A., Trabzuni D., Stamelou M., Edwards M., Kasperaviciute D., et al.. (2014). <span class="ref-title">Genomewide association study in cervical dystonia demonstrates possible association with sodium leak channel</span>. <span class="ref-journal">Mov. Disord</span>. <span class="ref-vol">29</span>, 245&#x02013;251. 10.1002/mds.25732 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4208301/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24227479" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fmds.25732" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mov.+Disord&amp;title=Genomewide+association+study+in+cervical+dystonia+demonstrates+possible+association+with+sodium+leak+channel&amp;author=K.+Y.+Mok&amp;author=S.+A.+Schneider&amp;author=D.+Trabzuni&amp;author=M.+Stamelou&amp;author=M.+Edwards&amp;volume=29&amp;publication_year=2014&amp;pages=245-251&amp;pmid=24227479&amp;doi=10.1002/mds.25732&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B28"><span class="mixed-citation">Murcray C. E., Lewinger J. P., Conti D. V., Thomas D. C., Gauderman W. J. (2011). <span class="ref-title">Sample size requirements to detect gene-environment interactions in genome-wide association studies</span>. <span class="ref-journal">Genet. Epidemiol</span>. <span class="ref-vol">35</span>, 201&#x02013;210. 10.1002/gepi.20569 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3076801/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21308767" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fgepi.20569" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Sample+size+requirements+to+detect+gene-environment+interactions+in+genome-wide+association+studies&amp;author=C.+E.+Murcray&amp;author=J.+P.+Lewinger&amp;author=D.+V.+Conti&amp;author=D.+C.+Thomas&amp;author=W.+J.+Gauderman&amp;volume=35&amp;publication_year=2011&amp;pages=201-210&amp;pmid=21308767&amp;doi=10.1002/gepi.20569&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B29"><span class="mixed-citation">Murcray C. E., Lewinger J. P., Gauderman W. J. (2009). <span class="ref-title">Gene-environment interaction in genome-wide association studies</span>. <span class="ref-journal">Am. J. Epidemiol</span>. <span class="ref-vol">169</span>, 219&#x02013;226. 10.1093/aje/kwn353 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2732981/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19022827" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Faje%2Fkwn353" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol&amp;title=Gene-environment+interaction+in+genome-wide+association+studies&amp;author=C.+E.+Murcray&amp;author=J.+P.+Lewinger&amp;author=W.+J.+Gauderman&amp;volume=169&amp;publication_year=2009&amp;pages=219-226&amp;pmid=19022827&amp;doi=10.1093/aje/kwn353&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B30"><span class="mixed-citation">Ottman R. (1996). <span class="ref-title">Gene-environment interaction: definitions and study designs</span>. <span class="ref-journal">Prevent. Med</span>. <span class="ref-vol">25</span>, 764&#x02013;770. 10.1006/pmed.1996.0117 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2823480/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8936580" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1006%2Fpmed.1996.0117" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prevent.+Med&amp;title=Gene-environment+interaction:+definitions+and+study+designs&amp;author=R.+Ottman&amp;volume=25&amp;publication_year=1996&amp;pages=764-770&amp;pmid=8936580&amp;doi=10.1006/pmed.1996.0117&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B31"><span class="mixed-citation">Pan W., Chen Y. M., Wei P. (2015). <span class="ref-title">Testing for polygenic effects in genome-wide association studies</span>. <span class="ref-journal">Genet. Epidemiol</span>. <span class="ref-vol">39</span>, 306&#x02013;316. 10.1002/gepi.21899 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4406854/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25847094" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fgepi.21899" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Testing+for+polygenic+effects+in+genome-wide+association+studies&amp;author=W.+Pan&amp;author=Y.+M.+Chen&amp;author=P.+Wei&amp;volume=39&amp;publication_year=2015&amp;pages=306-316&amp;pmid=25847094&amp;doi=10.1002/gepi.21899&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B32"><span class="mixed-citation">Puddey I. B., Beilin L. J. (2006). <span class="ref-title">Alcohol is bad for blood pressure</span>. <span class="ref-journal">Clin. Exp. Pharmacol. Physiol</span>. <span class="ref-vol">33</span>, 847&#x02013;852. 10.1111/j.1440-1681.2006.04452.x [<a href="https://pubmed.ncbi.nlm.nih.gov/16922819" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1440-1681.2006.04452.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Exp.+Pharmacol.+Physiol&amp;title=Alcohol+is+bad+for+blood+pressure&amp;author=I.+B.+Puddey&amp;author=L.+J.+Beilin&amp;volume=33&amp;publication_year=2006&amp;pages=847-852&amp;pmid=16922819&amp;doi=10.1111/j.1440-1681.2006.04452.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B33"><span class="mixed-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M. A., Bender D., et al.. (2007). <span class="ref-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="ref-journal">Am. J. Hum. Genet</span>. <span class="ref-vol">81</span>, 559&#x02013;575. 10.1086/519795 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17701901" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1086%2F519795" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.+A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B34"><span class="mixed-citation">Rudolph A., Chang-Claude J., Schmidt M. K. (2016). <span class="ref-title">Gene-environment interaction and risk of breast cancer</span>. <span class="ref-journal">Br. J. Cancer</span>
<span class="ref-vol">114</span>, 125&#x02013;133. 10.1038/bjc.2015.439 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4815812/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26757262" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fbjc.2015.439" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=Gene-environment+interaction+and+risk+of+breast+cancer&amp;author=A.+Rudolph&amp;author=J.+Chang-Claude&amp;author=M.+K.+Schmidt&amp;volume=114&amp;publication_year=2016&amp;pages=125-133&amp;pmid=26757262&amp;doi=10.1038/bjc.2015.439&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B35"><span class="mixed-citation">Simino J., Sung Y. J., Kume R., Schwander K., Rao D. C. (2013). <span class="ref-title">Gene-alcohol interactions identify several novel blood pressure loci including a promising locus near SLC16A9</span>. <span class="ref-journal">Front. Genet</span>. <span class="ref-vol">4</span>:277. 10.3389/fgene.2013.00277 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3860258/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24376456" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffgene.2013.00277" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Genet&amp;title=Gene-alcohol+interactions+identify+several+novel+blood+pressure+loci+including+a+promising+locus+near+SLC16A9&amp;author=J.+Simino&amp;author=Y.+J.+Sung&amp;author=R.+Kume&amp;author=K.+Schwander&amp;author=D.+C.+Rao&amp;volume=4&amp;publication_year=2013&amp;pages=277&amp;pmid=24376456&amp;doi=10.3389/fgene.2013.00277&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B36"><span class="mixed-citation">Sung Y. J., Winkler T. W., de Las Fuentes L., Bentley A. R., Brown M. R., Kraja A. T., et al.. (2018). <span class="ref-title">A Large-scale multi-ancestry genome-wide study accounting for smoking behavior identifies multiple significant loci for blood pressure</span>. <span class="ref-journal">Am. J. Hum. Genet</span>. <span class="ref-vol">102</span>, 375&#x02013;400. 10.1016/j.ajhg.2018.01.015 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5985266/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29455858" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ajhg.2018.01.015" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+Large-scale+multi-ancestry+genome-wide+study+accounting+for+smoking+behavior+identifies+multiple+significant+loci+for+blood+pressure&amp;author=Y.+J.+Sung&amp;author=T.+W.+Winkler&amp;author=L.+de+Las+Fuentes&amp;author=A.+R.+Bentley&amp;author=M.+R.+Brown&amp;volume=102&amp;publication_year=2018&amp;pages=375-400&amp;pmid=29455858&amp;doi=10.1016/j.ajhg.2018.01.015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B37"><span class="mixed-citation">Tomson J., Lip G. Y. (2006). <span class="ref-title">Alcohol and hypertension: an old relationship revisited</span>. <span class="ref-journal">Alcohol Alcohol</span>. <span class="ref-vol">41</span>, 3&#x02013;4. 10.1093/alcalc/agh223 [<a href="https://pubmed.ncbi.nlm.nih.gov/16239351" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Falcalc%2Fagh223" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alcohol+Alcohol&amp;title=Alcohol+and+hypertension:+an+old+relationship+revisited&amp;author=J.+Tomson&amp;author=G.+Y.+Lip&amp;volume=41&amp;publication_year=2006&amp;pages=3-4&amp;pmid=16239351&amp;doi=10.1093/alcalc/agh223&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B38"><span class="mixed-citation">Wakefield J. (2007). <span class="ref-title">A Bayesian measure of the probability of false discovery in genetic epidemiology studies</span>. <span class="ref-journal">Am. J. Hum. Genet</span>. <span class="ref-vol">81</span>, 208&#x02013;227. 10.1086/519024 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950810/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17668372" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1086%2F519024" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=A+Bayesian+measure+of+the+probability+of+false+discovery+in+genetic+epidemiology+studies&amp;author=J.+Wakefield&amp;volume=81&amp;publication_year=2007&amp;pages=208-227&amp;pmid=17668372&amp;doi=10.1086/519024&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B39"><span class="mixed-citation">Wakefield J. (2009). <span class="ref-title">Bayes factors for genome-wide association studies: comparison with P-values</span>. <span class="ref-journal">Genetic Epidemiol</span>. <span class="ref-vol">33</span>, 79&#x02013;86. 10.1002/gepi.20359 [<a href="https://pubmed.ncbi.nlm.nih.gov/18642345" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fgepi.20359" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetic+Epidemiol&amp;title=Bayes+factors+for+genome-wide+association+studies:+comparison+with+P-values&amp;author=J.+Wakefield&amp;volume=33&amp;publication_year=2009&amp;pages=79-86&amp;pmid=18642345&amp;doi=10.1002/gepi.20359&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B40"><span class="mixed-citation">Wang Y., Sha N., Fang Y. (2009). <span class="ref-title">Analysis of genome-wide association data by large-scale Bayesian logistic regression</span>. <span class="ref-journal">BMC Proc</span>. <span class="ref-vol">3</span>(<span class="ref-iss">Suppl. 7</span>):S16. 10.1186/1753-6561-3-s7-s16 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2795912/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20018005" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1753-6561-3-s7-s16" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Proc&amp;title=Analysis+of+genome-wide+association+data+by+large-scale+Bayesian+logistic+regression&amp;author=Y.+Wang&amp;author=N.+Sha&amp;author=Y.+Fang&amp;volume=3&amp;issue=Suppl.+7&amp;publication_year=2009&amp;pages=S16&amp;pmid=20018005&amp;doi=10.1186/1753-6561-3-s7-s16&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B41"><span class="mixed-citation">WTCCC  (2007). <span class="ref-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">447</span>, 661&#x02013;678. 10.1038/nature05911 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17554300" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature05911" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;doi=10.1038/nature05911&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B42"><span class="mixed-citation">Wu M. C., Lee S., Cai T., Li Y., Boehnke M., Lin X. (2011). <span class="ref-title">Rare-variant association testing for sequencing data with the sequence kernel association test</span>. <span class="ref-journal">Am. J. Hum. Genet</span>. <span class="ref-vol">89</span>, 82&#x02013;93. 10.1016/j.ajhg.2011.05.029 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3135811/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21737059" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.ajhg.2011.05.029" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Rare-variant+association+testing+for+sequencing+data+with+the+sequence+kernel+association+test&amp;author=M.+C.+Wu&amp;author=S.+Lee&amp;author=T.+Cai&amp;author=Y.+Li&amp;author=M.+Boehnke&amp;volume=89&amp;publication_year=2011&amp;pages=82-93&amp;pmid=21737059&amp;doi=10.1016/j.ajhg.2011.05.029&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B43"><span class="mixed-citation">Xin X., He J., Frontini M. G., Ogden L. G., Motsamai O. I., Whelton P. K. (2001). <span class="ref-title">Effects of alcohol reduction on blood pressure: a meta-analysis of randomized controlled trials</span>. <span class="ref-journal">Hypertension</span>
<span class="ref-vol">38</span>, 1112&#x02013;1117. 10.1161/hy1101.093424 [<a href="https://pubmed.ncbi.nlm.nih.gov/11711507" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1161%2Fhy1101.093424" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hypertension&amp;title=Effects+of+alcohol+reduction+on+blood+pressure:+a+meta-analysis+of+randomized+controlled+trials&amp;author=X.+Xin&amp;author=J.+He&amp;author=M.+G.+Frontini&amp;author=L.+G.+Ogden&amp;author=O.+I.+Motsamai&amp;volume=38&amp;publication_year=2001&amp;pages=1112-1117&amp;pmid=11711507&amp;doi=10.1161/hy1101.093424&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li><li id="B44"><span class="mixed-citation">Zhang Y., Biswas S. (2015). <span class="ref-title">An improved version of logistic bayesian LASSO for detecting rare haplotype-environment interactions with application to lung cancer</span>. <span class="ref-journal">Cancer Inform</span>. <span class="ref-vol">14</span>, 11&#x02013;16. 10.4137/CIN.S17290 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4332044/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25733797" ref="reftype=pubmed&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.4137%2FCIN.S17290" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Inform&amp;title=An+improved+version+of+logistic+bayesian+LASSO+for+detecting+rare+haplotype-environment+interactions+with+application+to+lung+cancer&amp;author=Y.+Zhang&amp;author=S.+Biswas&amp;volume=14&amp;publication_year=2015&amp;pages=11-16&amp;pmid=25733797&amp;doi=10.4137/CIN.S17290&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=6339974&amp;issue-id=305165&amp;journal-id=1681&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></li></ul></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Frontiers in Genetics</span> are provided here courtesy of <strong>Frontiers Media SA</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC6339974/pdf/fgene-09-00715.pdf" class="int-view">PDF (3.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/6339974/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/6339974/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="6339974" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6339974%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="h6PLiFJlF9RMAOKwzjLQJ6JKSiaQ47MmOjwLw96VwVkI90VXbqLhCaLLKpxCbYWC">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6339974%2F&amp;text=Genome-Wide%20Gene-Environment%20Interaction%20Analysis%20Using%20Set-Based%20Association%20Tests" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6339974%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6339974/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC6339974/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/30693016/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC6339974/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/30693016/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC6339974/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/6339974/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript" src="/core/mathjax/2.7.9/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC6339974/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
